<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en"><head>

<meta charset="utf-8">
<meta name="generator" content="quarto-1.2.313">

<meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes">


<title>Palaeoecological data-science: a data driven curriculum - 5&nbsp; The neotoma2 package</title>
<style>
code{white-space: pre-wrap;}
span.smallcaps{font-variant: small-caps;}
div.columns{display: flex; gap: min(4vw, 1.5em);}
div.column{flex: auto; overflow-x: auto;}
div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
ul.task-list{list-style: none;}
ul.task-list li input[type="checkbox"] {
  width: 0.8em;
  margin: 0 0.8em 0.2em -1.6em;
  vertical-align: middle;
}
pre > code.sourceCode { white-space: pre; position: relative; }
pre > code.sourceCode > span { display: inline-block; line-height: 1.25; }
pre > code.sourceCode > span:empty { height: 1.2em; }
.sourceCode { overflow: visible; }
code.sourceCode > span { color: inherit; text-decoration: inherit; }
div.sourceCode { margin: 1em 0; }
pre.sourceCode { margin: 0; }
@media screen {
div.sourceCode { overflow: auto; }
}
@media print {
pre > code.sourceCode { white-space: pre-wrap; }
pre > code.sourceCode > span { text-indent: -5em; padding-left: 5em; }
}
pre.numberSource code
  { counter-reset: source-line 0; }
pre.numberSource code > span
  { position: relative; left: -4em; counter-increment: source-line; }
pre.numberSource code > span > a:first-child::before
  { content: counter(source-line);
    position: relative; left: -1em; text-align: right; vertical-align: baseline;
    border: none; display: inline-block;
    -webkit-touch-callout: none; -webkit-user-select: none;
    -khtml-user-select: none; -moz-user-select: none;
    -ms-user-select: none; user-select: none;
    padding: 0 4px; width: 4em;
    color: #aaaaaa;
  }
pre.numberSource { margin-left: 3em; border-left: 1px solid #aaaaaa;  padding-left: 4px; }
div.sourceCode
  {   }
@media screen {
pre > code.sourceCode > span > a:first-child::before { text-decoration: underline; }
}
code span.al { color: #ff0000; font-weight: bold; } /* Alert */
code span.an { color: #60a0b0; font-weight: bold; font-style: italic; } /* Annotation */
code span.at { color: #7d9029; } /* Attribute */
code span.bn { color: #40a070; } /* BaseN */
code span.bu { color: #008000; } /* BuiltIn */
code span.cf { color: #007020; font-weight: bold; } /* ControlFlow */
code span.ch { color: #4070a0; } /* Char */
code span.cn { color: #880000; } /* Constant */
code span.co { color: #60a0b0; font-style: italic; } /* Comment */
code span.cv { color: #60a0b0; font-weight: bold; font-style: italic; } /* CommentVar */
code span.do { color: #ba2121; font-style: italic; } /* Documentation */
code span.dt { color: #902000; } /* DataType */
code span.dv { color: #40a070; } /* DecVal */
code span.er { color: #ff0000; font-weight: bold; } /* Error */
code span.ex { } /* Extension */
code span.fl { color: #40a070; } /* Float */
code span.fu { color: #06287e; } /* Function */
code span.im { color: #008000; font-weight: bold; } /* Import */
code span.in { color: #60a0b0; font-weight: bold; font-style: italic; } /* Information */
code span.kw { color: #007020; font-weight: bold; } /* Keyword */
code span.op { color: #666666; } /* Operator */
code span.ot { color: #007020; } /* Other */
code span.pp { color: #bc7a00; } /* Preprocessor */
code span.sc { color: #4070a0; } /* SpecialChar */
code span.ss { color: #bb6688; } /* SpecialString */
code span.st { color: #4070a0; } /* String */
code span.va { color: #19177c; } /* Variable */
code span.vs { color: #4070a0; } /* VerbatimString */
code span.wa { color: #60a0b0; font-weight: bold; font-style: italic; } /* Warning */
div.csl-bib-body { }
div.csl-entry {
  clear: both;
}
.hanging div.csl-entry {
  margin-left:2em;
  text-indent:-2em;
}
div.csl-left-margin {
  min-width:2em;
  float:left;
}
div.csl-right-inline {
  margin-left:2em;
  padding-left:1em;
}
div.csl-indent {
  margin-left: 2em;
}
</style>


<script src="site_libs/quarto-nav/quarto-nav.js"></script>
<script src="site_libs/quarto-nav/headroom.min.js"></script>
<script src="site_libs/clipboard/clipboard.min.js"></script>
<script src="site_libs/quarto-search/autocomplete.umd.js"></script>
<script src="site_libs/quarto-search/fuse.min.js"></script>
<script src="site_libs/quarto-search/quarto-search.js"></script>
<meta name="quarto:offset" content="./">
<link href="./05-wrangling-visualisation.html" rel="next">
<link href="./03-age-depth-modelling.html" rel="prev">
<script src="site_libs/quarto-html/quarto.js"></script>
<script src="site_libs/quarto-html/popper.min.js"></script>
<script src="site_libs/quarto-html/tippy.umd.min.js"></script>
<script src="site_libs/quarto-html/anchor.min.js"></script>
<link href="site_libs/quarto-html/tippy.css" rel="stylesheet">
<link href="site_libs/quarto-html/quarto-syntax-highlighting.css" rel="stylesheet" id="quarto-text-highlighting-styles">
<script src="site_libs/bootstrap/bootstrap.min.js"></script>
<link href="site_libs/bootstrap/bootstrap-icons.css" rel="stylesheet">
<link href="site_libs/bootstrap/bootstrap.min.css" rel="stylesheet" id="quarto-bootstrap" data-mode="light">
<script id="quarto-search-options" type="application/json">{
  "location": "sidebar",
  "copy-button": false,
  "collapse-after": 3,
  "panel-placement": "start",
  "type": "textbox",
  "limit": 20,
  "language": {
    "search-no-results-text": "No results",
    "search-matching-documents-text": "matching documents",
    "search-copy-link-title": "Copy link to search",
    "search-hide-matches-text": "Hide additional matches",
    "search-more-match-text": "more match in this document",
    "search-more-matches-text": "more matches in this document",
    "search-clear-button-title": "Clear",
    "search-detached-cancel-button-title": "Cancel",
    "search-submit-button-title": "Submit"
  }
}</script>

<script src="site_libs/htmlwidgets-1.6.1/htmlwidgets.js"></script>
<script src="site_libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<link href="site_libs/leaflet-1.3.1/leaflet.css" rel="stylesheet">
<script src="site_libs/leaflet-1.3.1/leaflet.js"></script>
<link href="site_libs/leafletfix-1.0.0/leafletfix.css" rel="stylesheet">
<script src="site_libs/proj4-2.6.2/proj4.min.js"></script>
<script src="site_libs/Proj4Leaflet-1.0.1/proj4leaflet.js"></script>
<link href="site_libs/rstudio_leaflet-1.3.1/rstudio_leaflet.css" rel="stylesheet">
<script src="site_libs/leaflet-binding-2.1.1/leaflet.js"></script>
<script src="site_libs/leaflet-providers-1.9.0/leaflet-providers_1.9.0.js"></script>
<script src="site_libs/leaflet-providers-plugin-2.1.1/leaflet-providers-plugin.js"></script>
<link href="site_libs/leaflet-markercluster-1.0.5/MarkerCluster.css" rel="stylesheet">
<link href="site_libs/leaflet-markercluster-1.0.5/MarkerCluster.Default.css" rel="stylesheet">
<script src="site_libs/leaflet-markercluster-1.0.5/leaflet.markercluster.js"></script>
<script src="site_libs/leaflet-markercluster-1.0.5/leaflet.markercluster.freezable.js"></script>
<script src="site_libs/leaflet-markercluster-1.0.5/leaflet.markercluster.layersupport.js"></script>
<link href="site_libs/datatables-css-0.0.0/datatables-crosstalk.css" rel="stylesheet">
<script src="site_libs/datatables-binding-0.27/datatables.js"></script>
<link href="site_libs/dt-core-1.12.1/css/jquery.dataTables.min.css" rel="stylesheet">
<link href="site_libs/dt-core-1.12.1/css/jquery.dataTables.extra.css" rel="stylesheet">
<script src="site_libs/dt-core-1.12.1/js/jquery.dataTables.min.js"></script>
<link href="site_libs/crosstalk-1.2.0/css/crosstalk.min.css" rel="stylesheet">
<script src="site_libs/crosstalk-1.2.0/js/crosstalk.min.js"></script>


</head>

<body class="nav-sidebar floating">

<div id="quarto-search-results"></div>
  <header id="quarto-header" class="headroom fixed-top">
  <nav class="quarto-secondary-nav" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar" aria-controls="quarto-sidebar" aria-expanded="false" aria-label="Toggle sidebar navigation" onclick="if (window.quartoToggleHeadroom) { window.quartoToggleHeadroom(); }">
    <div class="container-fluid d-flex justify-content-between">
      <h1 class="quarto-secondary-nav-title"><span class="chapter-number">5</span>&nbsp; <span class="chapter-title">The <code>neotoma2</code> package</span></h1>
      <button type="button" class="quarto-btn-toggle btn" aria-label="Show secondary navigation">
        <i class="bi bi-chevron-right"></i>
      </button>
    </div>
  </nav>
</header>
<!-- content -->
<div id="quarto-content" class="quarto-container page-columns page-rows-contents page-layout-article">
<!-- sidebar -->
  <nav id="quarto-sidebar" class="sidebar collapse sidebar-navigation floating overflow-auto">
    <div class="pt-lg-2 mt-2 text-left sidebar-header">
    <div class="sidebar-title mb-0 py-0">
      <a href="./">Palaeoecological data-science: a data driven curriculum</a> 
    </div>
      </div>
      <div class="mt-2 flex-shrink-0 align-items-center">
        <div class="sidebar-search">
        <div id="quarto-search" class="" title="Search"></div>
        </div>
      </div>
    <div class="sidebar-menu-container"> 
    <ul class="list-unstyled mt-1">
        <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./index.html" class="sidebar-item-text sidebar-link"><span class="chapter-number">1</span>&nbsp; <span class="chapter-title">Introduction</span></a>
  </div>
</li>
        <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./01-intro.html" class="sidebar-item-text sidebar-link"><span class="chapter-number">2</span>&nbsp; <span class="chapter-title">Neotoma Explorer</span></a>
  </div>
</li>
        <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./02-intro-to-R.html" class="sidebar-item-text sidebar-link"><span class="chapter-number">3</span>&nbsp; <span class="chapter-title">Intro to R</span></a>
  </div>
</li>
        <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./03-age-depth-modelling.html" class="sidebar-item-text sidebar-link"><span class="chapter-number">4</span>&nbsp; <span class="chapter-title">Age-depth modelling</span></a>
  </div>
</li>
        <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./04-neotoma2.html" class="sidebar-item-text sidebar-link active"><span class="chapter-number">5</span>&nbsp; <span class="chapter-title">The <code>neotoma2</code> package</span></a>
  </div>
</li>
        <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./05-wrangling-visualisation.html" class="sidebar-item-text sidebar-link"><span class="chapter-number">6</span>&nbsp; <span class="chapter-title">Data wrangling and visualisations</span></a>
  </div>
</li>
        <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./06-exploratory-analyses.html" class="sidebar-item-text sidebar-link"><span class="chapter-number">7</span>&nbsp; <span class="chapter-title">Exploratory Analyses</span></a>
  </div>
</li>
        <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./07-dissimilarity-roc.html" class="sidebar-item-text sidebar-link"><span class="chapter-number">8</span>&nbsp; <span class="chapter-title">Dissimilarity and Rates of Change</span></a>
  </div>
</li>
        <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./08-no-analogue.html" class="sidebar-item-text sidebar-link"><span class="chapter-number">9</span>&nbsp; <span class="chapter-title">Novelty and No Analogue Futures</span></a>
  </div>
</li>
        <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./09-ordination-pca.html" class="sidebar-item-text sidebar-link"><span class="chapter-number">10</span>&nbsp; <span class="chapter-title">Ordination: Principal Component Analysis</span></a>
  </div>
</li>
        <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./10-ordination-nmds.html" class="sidebar-item-text sidebar-link"><span class="chapter-number">11</span>&nbsp; <span class="chapter-title">Ordination: Non-metric Dimension Scaling</span></a>
  </div>
</li>
        <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./11-gams.html" class="sidebar-item-text sidebar-link"><span class="chapter-number">12</span>&nbsp; <span class="chapter-title">Generalised Additive Models</span></a>
  </div>
</li>
        <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./12-tapas.html" class="sidebar-item-text sidebar-link"><span class="chapter-number">13</span>&nbsp; <span class="chapter-title">Charcoal Analysis using TAPAS</span></a>
  </div>
</li>
        <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./13-psm.html" class="sidebar-item-text sidebar-link"><span class="chapter-number">14</span>&nbsp; <span class="chapter-title">Proxy System Modelling</span></a>
  </div>
</li>
        <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./15-test-chapter.html" class="sidebar-item-text sidebar-link"><span class="chapter-number">15</span>&nbsp; <span class="chapter-title">Test pad to delete</span></a>
  </div>
</li>
        <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./references.html" class="sidebar-item-text sidebar-link">References</a>
  </div>
</li>
    </ul>
    </div>
</nav>
<!-- margin-sidebar -->
    <div id="quarto-margin-sidebar" class="sidebar margin-sidebar">
        <nav id="TOC" role="doc-toc" class="toc-active">
    <h2 id="toc-title">Table of contents</h2>
   
  <ul>
  <li><a href="#introduction" id="toc-introduction" class="nav-link active" data-scroll-target="#introduction"><span class="toc-section-number">5.1</span>  Introduction</a></li>
  <li><a href="#the-wide-wide-world-of-apis" id="toc-the-wide-wide-world-of-apis" class="nav-link" data-scroll-target="#the-wide-wide-world-of-apis"><span class="toc-section-number">5.2</span>  The Wide, Wide World of APIs</a></li>
  <li><a href="#getting-started-with-neotoma2" id="toc-getting-started-with-neotoma2" class="nav-link" data-scroll-target="#getting-started-with-neotoma2"><span class="toc-section-number">5.3</span>  Getting Started With <code>neotoma2</code></a>
  <ul class="collapse">
  <li><a href="#good-coding-practice-explicitly-naming-packages-and-functions" id="toc-good-coding-practice-explicitly-naming-packages-and-functions" class="nav-link" data-scroll-target="#good-coding-practice-explicitly-naming-packages-and-functions"><span class="toc-section-number">5.3.1</span>  Good coding practice: explicitly naming packages and functions</a></li>
  <li><a href="#getting-help-with-neotoma-apis-r-explorer-tilia" id="toc-getting-help-with-neotoma-apis-r-explorer-tilia" class="nav-link" data-scroll-target="#getting-help-with-neotoma-apis-r-explorer-tilia"><span class="toc-section-number">5.3.2</span>  Getting Help with Neotoma (APIs, R, Explorer, Tilia, …)</a></li>
  </ul></li>
  <li><a href="#neotoma2-site-searches-get_sites" id="toc-neotoma2-site-searches-get_sites" class="nav-link" data-scroll-target="#neotoma2-site-searches-get_sites"><span class="toc-section-number">5.4</span>  Neotoma2: Site Searches: <code>get_sites()</code></a>
  <ul class="collapse">
  <li><a href="#section" id="toc-section" class="nav-link" data-scroll-target="#section"><span class="toc-section-number">5.4.1</span>  </a></li>
  <li><a href="#section-1" id="toc-section-1" class="nav-link" data-scroll-target="#section-1"><span class="toc-section-number">5.4.2</span>  </a></li>
  <li><a href="#section-2" id="toc-section-2" class="nav-link" data-scroll-target="#section-2"><span class="toc-section-number">5.4.3</span>  </a></li>
  <li><a href="#searching-for-datasets" id="toc-searching-for-datasets" class="nav-link" data-scroll-target="#searching-for-datasets"><span class="toc-section-number">5.4.4</span>  Searching for Datasets:</a></li>
  <li><a href="#section-3" id="toc-section-3" class="nav-link" data-scroll-target="#section-3"><span class="toc-section-number">5.4.5</span>  </a></li>
  <li><a href="#filter-records" id="toc-filter-records" class="nav-link" data-scroll-target="#filter-records"><span class="toc-section-number">5.4.6</span>  Filter Records</a></li>
  <li><a href="#section-4" id="toc-section-4" class="nav-link" data-scroll-target="#section-4"><span class="toc-section-number">5.4.7</span>  </a></li>
  <li><a href="#retrieving-sample-data." id="toc-retrieving-sample-data." class="nav-link" data-scroll-target="#retrieving-sample-data."><span class="toc-section-number">5.4.8</span>  Retrieving <code>sample()</code> data.</a></li>
  </ul></li>
  <li><a href="#simple-analytics" id="toc-simple-analytics" class="nav-link" data-scroll-target="#simple-analytics"><span class="toc-section-number">5.5</span>  Simple Analytics</a>
  <ul class="collapse">
  <li><a href="#stratigraphic-plotting-building-a-pollen-diagram" id="toc-stratigraphic-plotting-building-a-pollen-diagram" class="nav-link" data-scroll-target="#stratigraphic-plotting-building-a-pollen-diagram"><span class="toc-section-number">5.5.1</span>  Stratigraphic Plotting: Building a Pollen Diagram</a></li>
  <li><a href="#change-taxon-distributions-across-space-and-time" id="toc-change-taxon-distributions-across-space-and-time" class="nav-link" data-scroll-target="#change-taxon-distributions-across-space-and-time"><span class="toc-section-number">5.5.2</span>  Change Taxon Distributions Across Space and Time</a></li>
  </ul></li>
  <li><a href="#conclusion" id="toc-conclusion" class="nav-link" data-scroll-target="#conclusion"><span class="toc-section-number">5.6</span>  Conclusion</a></li>
  </ul>
</nav>
    </div>
<!-- main -->
<main class="content" id="quarto-document-content">

<header id="title-block-header" class="quarto-title-block default">
<div class="quarto-title">
<h1 class="title"><span id="neotoma2-sec" class="quarto-section-identifier d-none d-lg-block"><span class="chapter-number">5</span>&nbsp; <span class="chapter-title">The <code>neotoma2</code> package</span></span></h1>
</div>



<div class="quarto-title-meta">

    
  
    
  </div>
  

</header>

<section id="introduction" class="level2" data-number="5.1">
<h2 data-number="5.1" class="anchored" data-anchor-id="introduction"><span class="header-section-number">5.1</span> Introduction</h2>
<p>This series of exercises is designed to give you hands-on practice in using APIs and the <code>neotoma2</code> R package <span class="citation" data-cites="goring2015">(<a href="references.html#ref-goring2015" role="doc-biblioref">Goring et al. 2015</a>)</span>, both for practical reasons and for insights into how open-data systems work. The primary purpose of <code>neotoma2</code> is to pass data from the Neotoma Paleoecology Database (Neotoma DB) server into your local R environment. Neotoma relies on Application Programming Interfaces (<a href="https://en.wikipedia.org/wiki/Application_programming_interface">APIs</a>) to communicate with the Neotoma Paleoecology Database, so we’ll begin with an introduction to APIs.</p>
<p>Much of this exercise is based on <a href="https://open.neotomadb.org/EPD_binder/simple_workflow.html">A Simple Workflow</a> and <a href="https://open.neotomadb.org/EPD_binder/complex_workflow.html">A Not so Simple Workflow</a> tutorials that were originally developed for a workshop convened by the European Pollen Database in June 2022. Today we’ll keep the initial analyses relatively simple, so that you can focus on learning the data structures and functions used by <code>neotoma2</code>. A hard-won lesson for a practicing data scientist is how much time and attention goes into data handling!</p>
<p>In this tutorial you will learn how to:</p>
<ul>
<li><p>Use the Neotoma API</p></li>
<li><p>Search for sites using site names and geographic parameters</p></li>
<li><p>Filter results using temporal and spatial parameters</p></li>
<li><p>Obtain sample information for the selected datasets</p></li>
<li><p>Perform basic analysis including the use of climate data from rasters</p></li>
</ul>
<p>This series of exercises is designed to give you hands-on practice in using APIs and the <code>neotoma2</code> R package, both for practical reasons and for insights into how open-data systems work. <code>neotoma2</code>’s primary purpose is to pass data from the Neotoma DB server into your local R computing environment. Neotoma relies on Application Programming Interfaces (<a href="https://en.wikipedia.org/wiki/API">APIs</a>) to communicate with the Neotoma Paleoecology Database, so we’ll begin with an introduction to APIs.</p>
</section>
<section id="the-wide-wide-world-of-apis" class="level2" data-number="5.2">
<h2 data-number="5.2" class="anchored" data-anchor-id="the-wide-wide-world-of-apis"><span class="header-section-number">5.2</span> The Wide, Wide World of APIs</h2>
<p>The Neotoma Paleoecology Database is a relational database, hosted on servers at <a href="https://sites.psu.edu/environmentalinformatics/">Penn State’s Center for Environmental Informatics</a>. For security reasons, direct access to these servers is quite limited, and available only to a few Neotoma and CEI programmers.</p>
<p><strong>APIs</strong> offer public access points into Neotoma that anyone can use. Each API is basically a function: you provide the API with a set of operational parameters, and it returns a set of data or metadata. Each API hence is designed to support one particular task or set of tasks; it offers a narrow window into the larger Neotoma DB. <a href="https://en.wikipedia.org/wiki/Representational_state_transfer">REST-ful APIs</a> follow a particular set of standards that allow them to be read by web browsers (i.e.&nbsp;within the HTTP protocol) and return data objects, typically in HTML, XML, JSON or other human- &amp; machine-readable formats.</p>
<p>The <a href="https://api.neotomadb.org/api-docs/">Neotoma APIs</a> provide a series of functions for retrieving different kinds of data from Neotoma DB. Data objects are returned in <a href="https://en.wikipedia.org/wiki/JSON">JSON</a> format. For this exercise, we strongly recommend adding an extension to your browser that formats the JSON output to make it easier to read, such as <a href="https://addons.mozilla.org/en-US/firefox/addon/jsonview/">JSONView</a> for Firefox, and <a href="https://chrome.google.com/webstore/detail/json-formatter/bcjindcccaagfpapjjmafapmmgkkhgoa?hl=en">JSON Formatter</a> for Chrome.</p>
<p>The APIs for Neotoma can be found here: https://api.neotomadb.org/api-docs/. Look through the lists of different APIs and find the one labeled <code>GET /v2.0/data/sites/{siteid}</code>. Then, click on the Try It Out button at right, enter 666 into the site identifier box, and click Execute. Then scroll down. You should see some example code in curl format (ignore this) and as a URL. If you scroll down further, you’ll see the actual data return, in JSON format. JSON is a structured data format designed to be both human-readable and machine-readable. It looks like a nested series of lists.</p>
<p>Now, let’s go from the API sandbox to direct use of the API URLs in a browser. Copy and paste the below URL into the URL box in your browser:</p>
<p><code>https://api.neotomadb.org/v2.0/data/sites?sitename=%devil%</code></p>
<p>This should open a new web page in your browser with a returned JSON object. For this search, the JSON object should include 16 or more sites with the name ‘devil’ in them (note the use of % as wildcards), including Devil’s Lake, WI. The opening line “status” = “success” means that the API ran successfully. Note that it is possible for an API to run successfully but return no data! For example, try:</p>
<p><code>https://api.neotomadb.org/v2.0/data/sites?sitename=devil</code></p>
<p>Here, “status” = “success” but data=[], i.e.&nbsp;the API successfully reported back to you that no sites in Neotoma have the exact name of ‘devil’.</p>
<p>Once you know what an API is, you will quickly see how widely these are used by modern browsers and websites. For example, try:</p>
<p><code>https://www.google.com/search?q=neotoma&amp;ei=-xE_Y4XHA8GF0PEPi6-p-AU&amp;ved=0ahUKEwjFsPSAksz6AhXBAjQIHYtXCl8Q4dUDCA0&amp;uact=5&amp;oq=neotoma&amp;gs_lcp=Cgdnd3Mtd2l6EAMyDgguEIAEEIsDEKgDEJ4DMggILhCABBCLAzIICC4QgAQQiwMyCAguEIAEEIsDMggILhCABBCLAzIICAAQkQIQiwMyCAguEIAEEIsDMggILhCABBCLAzIICC4QgAQQiwMyCAgAEIAEEIsDOgoIABBHENYEELADOgcIABCwAxBDOg0IABDkAhDWBBCwAxgBOgwILhDIAxCwAxBDGAI6DwguENQCEMgDELADEEMYAjoOCC4QgAQQxwEQ0QMQ1AI6CwgAEIAEELEDEIMBOhEILhCABBCxAxCDARDHARDRAzoOCC4QgAQQsQMQgwEQ1AI6FAguEIAEELEDEIMBENQCEJsDEKgDOg4ILhCABBDHARDRAxCLAzoOCAAQgAQQsQMQgwEQiwM6EQguEIAEENQCEIsDEKgDEKQDOg4IABCRAhCLAxCmAxCoAzoNCAAQQxCLAxCoAxCmAzoFCAAQgAQ6CAguELEDEIMBOgsILhCABBDHARCvAToHCAAQQxCLAzoICC4QgAQQsQM6CAgAEIAEELEDOhEILhCABBCxAxDUAhCjAxCoAzoLCC4QsQMQgwEQiwM6BwgAEIAEEAo6DggAELEDEIMBEJECEIsDOhAIABCABBCxAxCDARAKEIsDOgoIABCABBAKEIsDOgUIABCRAkoECEEYAEoECEYYAVC5BVirCmDFDGgBcAF4AIABjAGIAdcFkgEDMy40mAEAoAEByAERuAEDwAEB2gEGCAEQARgJ2gEGCAIQARgI&amp;sclient=gws-wiz</code></p>
<p>Note that in this demo use of the Google search API, all we did is enter ‘neotoma’ in the standard Google search window and then Google produced and formatted the above API query with a ?search parameter (and who knows what else is lurking in that massive text parameter…)</p>
<p>OK, now your turn:</p>
<p><strong>Exercise Question 1</strong> Use the <em>sites</em> API to retrieve site data for sites of interest. The <em>sites</em> API has a few different parameters, so try out options. Try copying and pasting the URLs into the address line of your browser, and then hitting return. In your homework exercise, provide at least two <em>sites</em> API calls (as URLs) with a comment line for each that explains what the API command is doing.</p>
</section>
<section id="getting-started-with-neotoma2" class="level2" data-number="5.3">
<h2 data-number="5.3" class="anchored" data-anchor-id="getting-started-with-neotoma2"><span class="header-section-number">5.3</span> Getting Started With <code>neotoma2</code></h2>
<p>For this workbook we use several packages, including <code>leaflet</code>, <code>sf</code> and others. We load the packages using the <code>pacman</code> package, which will automatically install the packages if they do not currently exist in your set of packages.</p>
<div class="callout-note callout callout-style-default callout-captioned">
<div class="callout-header d-flex align-content-center">
<div class="callout-icon-container">
<i class="callout-icon"></i>
</div>
<div class="callout-caption-container flex-fill">
Packages required for this section
</div>
</div>
<div class="callout-body-container callout-body">
<p><code>neotoma2</code> is available from GitHub and can be installed in R using the <code>devtools</code> package via:</p>
<div class="cell" data-hash="04-neotoma2_cache/html/packages_8a1af7f785133346b5ac1f285857a88a">
<div class="sourceCode cell-code" id="cb1"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb1-1"><a href="#cb1-1" aria-hidden="true" tabindex="-1"></a><span class="co"># Load up the package</span></span>
<span id="cb1-2"><a href="#cb1-2" aria-hidden="true" tabindex="-1"></a><span class="cf">if</span> (<span class="sc">!</span><span class="fu">require</span>(<span class="st">"pacman"</span>)) <span class="fu">install.packages</span>(<span class="st">"pacman"</span>, <span class="at">repos=</span><span class="st">"http://cran.r-project.org"</span>)</span>
<span id="cb1-3"><a href="#cb1-3" aria-hidden="true" tabindex="-1"></a>pacman<span class="sc">::</span><span class="fu">p_load</span>(dplyr, ggplot2, sf, geojsonsf, leaflet, raster, DT, devtools) <span class="co"># devtools allows us to install from online repositories</span></span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
</div>
</div>
<section id="good-coding-practice-explicitly-naming-packages-and-functions" class="level3" data-number="5.3.1">
<h3 data-number="5.3.1" class="anchored" data-anchor-id="good-coding-practice-explicitly-naming-packages-and-functions"><span class="header-section-number">5.3.1</span> Good coding practice: explicitly naming packages and functions</h3>
<p>Different packages in R are created independently by different teams of developers, and it’s very common for two different packages to use the same function names (the English language is finite…). This can lead to coding errors if you call a function that you know is in one package, but R guesses wrongly that you wanted a function of the same name from an entirely different package. For example, for a function like <code>filter()</code>, which exists in both <code>neotoma2</code> and other packages such as <code>dplyr</code>, you may see an error that looks like:</p>
<div class="sourceCode" id="cb2"><pre class="sourceCode bash code-with-copy"><code class="sourceCode bash"><span id="cb2-1"><a href="#cb2-1" aria-hidden="true" tabindex="-1"></a><span class="ex">Error</span> in UseMethod<span class="er">(</span><span class="st">"filter"</span><span class="kw">)</span> <span class="bu">:</span></span>
<span id="cb2-2"><a href="#cb2-2" aria-hidden="true" tabindex="-1"></a> <span class="ex">no</span> applicable method for <span class="st">'filter'</span> applied to an object of class <span class="st">"sites"</span></span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<p>You can avoid this error by explicitly naming which package has the function that you want to use, through the standard convention of double colons (<code>package.name::function.name</code>). For example, using <code>neotoma2::filter()</code> tells R explicitly that you want to use the <code>filter()</code> function in the <code>neotoma2</code> package, not some other package version.</p>
<p><em>Summary</em>: It’s good coding practice to always explicitly link packages to the functions that you are using! Your future self will thank you.</p>
</section>
<section id="getting-help-with-neotoma-apis-r-explorer-tilia" class="level3" data-number="5.3.2">
<h3 data-number="5.3.2" class="anchored" data-anchor-id="getting-help-with-neotoma-apis-r-explorer-tilia"><span class="header-section-number">5.3.2</span> Getting Help with Neotoma (APIs, R, Explorer, Tilia, …)</h3>
<p>For Geog523, the class Slack workspace is your first and best option. If you’re planning on working with Neotoma more in the future, please join us on <a href="https://join.slack.com/t/neotomadb/shared_invite/zt-cvsv53ep-wjGeCTkq7IhP6eUNA9NxYQ">Slack</a>, where we manage a channel specifically for questions about the R package. You may also wish to join Neotoma’s Google Groups mailing list, and if so <a href="mailto:neotoma-contact@googlegroups.com">contact us</a> to be added.</p>
</section>
</section>
<section id="neotoma2-site-searches-get_sites" class="level2" data-number="5.4">
<h2 data-number="5.4" class="anchored" data-anchor-id="neotoma2-site-searches-get_sites"><span class="header-section-number">5.4</span> Neotoma2: Site Searches: <code>get_sites()</code></h2>
<p>Many users of Neotoma first want to search and explore data at the site level. There are several ways to find sites in <code>neotoma2</code>, but we think of <code>sites</code> primarily as spatial objects. They have names, locations, and are found within geopolitical units. However, sites themselves do not have associated information about taxa, dataset types, or ages. <code>sites</code> instead are simply the container into which we add that information. So, when we search for sites we can search by:</p>
<ul>
<li>siteid</li>
<li>sitename</li>
<li>location</li>
<li>altitude (maximum and minimum)</li>
<li>geopolitical unit</li>
</ul>
<section id="searching-by-site-name-sitenamedevil" class="level4 tabset" data-number="5.4.0.1">
<h4 class="tabset anchored" data-number="5.4.0.1" data-anchor-id="searching-by-site-name-sitenamedevil"><span class="header-section-number">5.4.0.1</span> Searching by Site Name: <code>sitename="%Devil%"</code></h4>
<p>We may know exactly what site we’re looking for (<em>“Devil’s Lake”</em>), or have an approximate guess for the site name (for example, we know it’s something like <em>“Devil Pond”</em>, or <em>“Devil’s Hole”</em>).</p>
<p>We use the general format: <code>get_sites(sitename="XXXXX")</code> for searching by name.</p>
<p>PostgreSQL (and the API) uses the percent sign as a wildcard. So <code>"%Devil%"</code> would pick up “Devils Lake” for us (and would pick up “Devil’s Canopy Cave”). Note that the search query is case insensitive, so <code>"%devil%"</code> will work.</p>
<p>If we want an individual record we can use the siteid, which is a unique identifier for each site: .</p>
<section id="code" class="level5" data-number="5.4.0.1.1">
<h5 data-number="5.4.0.1.1" class="anchored" data-anchor-id="code"><span class="header-section-number">5.4.0.1.1</span> Code</h5>
<div class="cell" data-hash="04-neotoma2_cache/html/sitename_574bd1c9065f9284b0503eaeae1e277c">
<div class="sourceCode cell-code" id="cb3"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb3-1"><a href="#cb3-1" aria-hidden="true" tabindex="-1"></a>devil_sites <span class="ot">&lt;-</span> neotoma2<span class="sc">::</span><span class="fu">get_sites</span>(<span class="at">sitename =</span> <span class="st">"%Devil%"</span>)</span>
<span id="cb3-2"><a href="#cb3-2" aria-hidden="true" tabindex="-1"></a><span class="fu">plotLeaflet</span>(devil_sites)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
</section>
<section id="result" class="level5" data-number="5.4.0.1.2">
<h5 data-number="5.4.0.1.2" class="anchored" data-anchor-id="result"><span class="header-section-number">5.4.0.1.2</span> Result</h5>
<div class="cell" data-hash="04-neotoma2_cache/html/sitenamePlot_880fb6e5c9292d1772da0d947e5e1cae">
<div class="cell-output-display">
<div class="leaflet html-widget html-fill-item-overflow-hidden html-fill-item" id="htmlwidget-b1adbbbdde62da1d9a02" style="width:100%;height:464px;"></div>
<script type="application/json" data-for="htmlwidget-b1adbbbdde62da1d9a02">{"x":{"options":{"crs":{"crsClass":"L.CRS.EPSG3857","code":null,"proj4def":null,"projectedBounds":null,"options":{}}},"calls":[{"method":"addProviderTiles","args":["Stamen.TerrainBackground",null,null,{"errorTileUrl":"","noWrap":false,"detectRetina":false}]},{"method":"addTiles","args":["https://{s}.tile.openstreetmap.org/{z}/{x}/{y}.png",null,null,{"minZoom":0,"maxZoom":18,"tileSize":256,"subdomains":"abc","errorTileUrl":"","tms":false,"noWrap":false,"zoomOffset":0,"zoomReverse":false,"opacity":1,"zIndex":1,"detectRetina":false,"attribution":"&copy; <a href=\"https://openstreetmap.org\">OpenStreetMap<\/a> contributors, <a href=\"https://creativecommons.org/licenses/by-sa/2.0/\">CC-BY-SA<\/a>"}]},{"method":"addCircleMarkers","args":[[48.022914,43.4159,29.36667,30.36667,29.36667,29.86667,38.11667,30.75,48.081514,48.014636,48.022636,43.024608,45.814184,44.98,56.16667,48.0189,35.69722,33.77506,33.77479,42.316004,44.034428,48.056368,47.966236],[-98.962028,-89.73334,-101,-97.75,-82.36667,-82.75,-86.25,-81.5,-98.934766,-98.890688,-98.988878,-77.573886,-93.334446,-123.995754,-133.5,-98.89294,-115.45222,-116.68014,-116.67526,-120.960152,-121.761974,-99.11855,-98.649672],10,null,null,{"interactive":true,"draggable":false,"keyboard":true,"title":"","alt":"","zIndexOffset":0,"opacity":1,"riseOnHover":true,"riseOffset":250,"stroke":true,"color":"#03F","weight":5,"opacity.1":0.5,"fill":true,"fillColor":"#03F","fillOpacity":0.2},{"showCoverageOnHover":true,"zoomToBoundsOnClick":true,"spiderfyOnMaxZoom":true,"removeOutsideVisibleBounds":true,"spiderLegPolylineOptions":{"weight":1.5,"color":"#222","opacity":0.5},"freezeAtZoom":false},null,["<b>Devils Lake, Main Bay<\/b><br><b>Description:<\/b> Very large closed basin, ice-thrust lake, which lies in a preglacial valley. Water level has varied dramatically during the historical period.<br><a href=http://apps.neotomadb.org/explorer/?siteids=665>Explorer Link<\/a>","<b>Devils Lake<\/b><br><b>Description:<\/b> Lake is located in a pre-glacial gorge of the Wisconsin river cut through the Baraboo Hills, a bedrock high. The valley is dammed at both ends by terminal moraines deposited by the Green Bay lobe, which flowed around the Baraboo Hills. Lake is fed by groundwater. Surrounding vegetation: white-black-bur oak.<br><a href=http://apps.neotomadb.org/explorer/?siteids=666>Explorer Link<\/a>","<b>Devil's Mouth [41VV188]<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=3686>Explorer Link<\/a>","<b>Devil's Hollow [41TV38]<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=4228>Explorer Link<\/a>","<b>Devil's Den<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=5092>Explorer Link<\/a>","<b>Devil's Eye Spring<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=5189>Explorer Link<\/a>","<b>Devil's Staircase Pit Cave<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=5323>Explorer Link<\/a>","<b>Devil's Walking Stick [9Cam177]<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=5853>Explorer Link<\/a>","<b>Devils Lake, Creel Bay<\/b><br><b>Description:<\/b> Very large closed basin, ice-thrust lake, which lies in a preglacial valley. Water level has varied dramatically during the historical period. Creel Bay has a maximum depth of about 15 m.<br><a href=http://apps.neotomadb.org/explorer/?siteids=9717>Explorer Link<\/a>","<b>Devils Lake, Mission Bay<\/b><br><b>Description:<\/b> Very large closed basin, ice-thrust lake, which lies in a preglacial valley. Water level has varied dramatically during the historical period. MIssion Bay has a maximum depth of about 13 m.<br><a href=http://apps.neotomadb.org/explorer/?siteids=9732>Explorer Link<\/a>","<b>Devils Lake<\/b><br><b>Description:<\/b> Very large closed basin, ice-thrust lake, which lies in a preglacial valley. Water level has varied dramatically during the historical period. MIssion Bay has a maximum depth of about 13 m.<br><a href=http://apps.neotomadb.org/explorer/?siteids=9961>Explorer Link<\/a>","<b>Devil's Bathtub<\/b><br><b>Description:<\/b> Kettle pond located in the till plains of the Erie-Ontario lowland. Surrounding vegetation: Quercus and Acer saccharum, with nearby plantations of Pinus resinosa.<br><a href=http://apps.neotomadb.org/explorer/?siteids=10280>Explorer Link<\/a>","<b>Devils Lake<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=11726>Explorer Link<\/a>","<b>Devil's Lake<\/b><br><b>Description:<\/b> Devils Lake is an eutrophic lake located in Lincoln County on the central Oregon coast. The lake was formed by natural accumulation of. beachs and impounding water at the outlet of the D River.<br><a href=http://apps.neotomadb.org/explorer/?siteids=14224>Explorer Link<\/a>","<b>Devil's Canopy Cave [AKPET221]<\/b><br><b>Description:<\/b> Silty deposit in a cave on Prince of Wales Island, Petersburg quadrangle, southeastern Alaska. <br><a href=http://apps.neotomadb.org/explorer/?siteids=18518>Explorer Link<\/a>","<b>Devils Lake<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=23349>Explorer Link<\/a>","<b>Devil Peak Fauna<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=23437>Explorer Link<\/a>","<b>SJ-10, Devil's Slide<\/b><br><b>Description:<\/b> San Jacinto Mountains, Peninsular Range. Lower montane with Pinus lambertiana, Abies concolor, Pinus jeffreyii, Quercus vaccinifolia, upper Calocedrus decurrens.<br><a href=http://apps.neotomadb.org/explorer/?siteids=26972>Explorer Link<\/a>","<b>SJ-11, Devil's Slide<\/b><br><b>Description:<\/b> San Jacinto Mountains, Peninsular Range. Lower montane, southern aspect with Quercus vaccinifolia, Abies concolor, Pinus jeffreyii, Chrysolepis sempervirens, Leptodactylon pungens.<br><a href=http://apps.neotomadb.org/explorer/?siteids=26973>Explorer Link<\/a>","<b>Devil Lake<\/b><br><b>Description:<\/b> Juniperus occidentalis, Pinus ponderosa, Artemisia tridentata, Poaceae<br><a href=http://apps.neotomadb.org/explorer/?siteids=27576>Explorer Link<\/a>","<b>Devils Lake<\/b><br><b>Description:<\/b> Abies lasiocarpa, Pinus contorta<br><a href=http://apps.neotomadb.org/explorer/?siteids=27605>Explorer Link<\/a>","<b>Devils<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=28909>Explorer Link<\/a>","<b>E. Devils<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=28911>Explorer Link<\/a>"],null,null,{"interactive":false,"permanent":false,"direction":"auto","opacity":1,"offset":[0,0],"textsize":"10px","textOnly":false,"className":"","sticky":true},null]}],"limits":{"lat":[29.36667,56.16667],"lng":[-133.5,-77.573886]}},"evals":[],"jsHooks":[]}</script>
</div>
</div>
</section>
</section>
<section id="section" class="level3" data-number="5.4.1">
<h3 data-number="5.4.1" class="anchored" data-anchor-id="section"><span class="header-section-number">5.4.1</span> </h3>
<p><strong>Exercise Question 2</strong> How many sites have the name ‘clear’ in them? Show both your code and provide the total count.</p>
<section id="searching-by-location-locc" class="level4 tabset" data-number="5.4.1.1">
<h4 class="tabset anchored" data-number="5.4.1.1" data-anchor-id="searching-by-location-locc"><span class="header-section-number">5.4.1.1</span> Searching by Location: <code>loc=c()</code></h4>
<p>The <code>neotoma</code> package used a bounding box for locations, structured as a vector of latitude and longitude values: <code>c(xmin, ymin, xmax, ymax)</code>. The <code>neotoma2</code> R package supports both this simple bounding box and also more complex spatial objects, using the <a href="https://r-spatial.github.io/sf/"><code>sf</code> package</a>. Using the <code>sf</code> package allows us to more easily work with raster and polygon data in R, and to select sites using more complex spatial objects. The <code>loc</code> parameter works with the simple vector, <a href="https://arthur-e.github.io/Wicket/sandbox-gmaps3.html">WKT</a>, <a href="http://geojson.io/#map=2/20.0/0.0">geoJSON</a> objects and native <code>sf</code> objects in R. <strong>Note however</strong> that the <code>neotoma2</code> package is a wrapper for a simple API call using a URL (see APIs above), and URL strings have a maximum limit of 1028 characters, so the API currently cannot accept very long/complex spatial objects.</p>
<p>As an example of different ways that you can search by location, let’s say you wanted to search for all sites in the state of Michigan. Here are three spatial representations of Michigan: 1) a geoJSON list with five elements, 2) WKT, and 3) bounding box representation. And, as a fourth variant, we’ve transformed the <code>mich$geoJSON</code> element to an object for the <code>sf</code> package. Any of these four spatial representations work with the <code>neotoma2</code> package.</p>
<div class="cell" data-hash="04-neotoma2_cache/html/boundingBox_a478b19e0c52d207e87c20311a4070b6">
<div class="sourceCode cell-code" id="cb4"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb4-1"><a href="#cb4-1" aria-hidden="true" tabindex="-1"></a>mich <span class="ot">&lt;-</span> <span class="fu">list</span>(<span class="at">geoJSON =</span> <span class="st">'{"type": "Polygon",</span></span>
<span id="cb4-2"><a href="#cb4-2" aria-hidden="true" tabindex="-1"></a><span class="st">        "coordinates": [[</span></span>
<span id="cb4-3"><a href="#cb4-3" aria-hidden="true" tabindex="-1"></a><span class="st">            [-86.95, 41.55],</span></span>
<span id="cb4-4"><a href="#cb4-4" aria-hidden="true" tabindex="-1"></a><span class="st">            [-82.43, 41.55],</span></span>
<span id="cb4-5"><a href="#cb4-5" aria-hidden="true" tabindex="-1"></a><span class="st">            [-82.43, 45.88],</span></span>
<span id="cb4-6"><a href="#cb4-6" aria-hidden="true" tabindex="-1"></a><span class="st">            [-86.95, 45.88],</span></span>
<span id="cb4-7"><a href="#cb4-7" aria-hidden="true" tabindex="-1"></a><span class="st">            [-86.95, 41.55]</span></span>
<span id="cb4-8"><a href="#cb4-8" aria-hidden="true" tabindex="-1"></a><span class="st">            ]]}'</span>,</span>
<span id="cb4-9"><a href="#cb4-9" aria-hidden="true" tabindex="-1"></a>        <span class="at">WKT =</span> <span class="st">'POLYGON ((-86.95 41.55,</span></span>
<span id="cb4-10"><a href="#cb4-10" aria-hidden="true" tabindex="-1"></a><span class="st">                         -82.43 41.55,</span></span>
<span id="cb4-11"><a href="#cb4-11" aria-hidden="true" tabindex="-1"></a><span class="st">                         -82.43 45.88,</span></span>
<span id="cb4-12"><a href="#cb4-12" aria-hidden="true" tabindex="-1"></a><span class="st">                         -86.95 45.88,</span></span>
<span id="cb4-13"><a href="#cb4-13" aria-hidden="true" tabindex="-1"></a><span class="st">                         -86.95 41.55))'</span>,</span>
<span id="cb4-14"><a href="#cb4-14" aria-hidden="true" tabindex="-1"></a>        <span class="at">bbox =</span> <span class="fu">c</span>(<span class="sc">-</span><span class="fl">86.95</span>, <span class="fl">41.55</span>, <span class="sc">-</span><span class="fl">82.43</span>, <span class="fl">45.88</span>))</span>
<span id="cb4-15"><a href="#cb4-15" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb4-16"><a href="#cb4-16" aria-hidden="true" tabindex="-1"></a>mich<span class="sc">$</span>sf <span class="ot">&lt;-</span> geojsonsf<span class="sc">::</span><span class="fu">geojson_sf</span>(mich<span class="sc">$</span>geoJSON)[[<span class="dv">1</span>]]</span>
<span id="cb4-17"><a href="#cb4-17" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb4-18"><a href="#cb4-18" aria-hidden="true" tabindex="-1"></a>mich_sites <span class="ot">&lt;-</span> neotoma2<span class="sc">::</span><span class="fu">get_sites</span>(<span class="at">loc =</span> mich<span class="sc">$</span>geoJSON, <span class="at">all_data =</span> <span class="cn">TRUE</span>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
<p>You can always simply <code>plot()</code> the <code>sites</code> objects, but this won’t show any geographic context. The <code>leaflet::plotLeaflet()</code> function returns a <code>leaflet()</code> map, and allows you to further customize it, or add additional spatial data (like our original bounding polygon, <code>mich$sf</code>, which works directly with the R <code>leaflet</code> package):</p>
<section id="code-1" class="level5" data-number="5.4.1.1.1">
<h5 data-number="5.4.1.1.1" class="anchored" data-anchor-id="code-1"><span class="header-section-number">5.4.1.1.1</span> Code</h5>
<div class="cell" data-hash="04-neotoma2_cache/html/plotL_b520c8a1f008022080e03619f64e8b82">
<div class="sourceCode cell-code" id="cb5"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb5-1"><a href="#cb5-1" aria-hidden="true" tabindex="-1"></a>neotoma2<span class="sc">::</span><span class="fu">plotLeaflet</span>(mich_sites) <span class="sc">%&gt;%</span></span>
<span id="cb5-2"><a href="#cb5-2" aria-hidden="true" tabindex="-1"></a>  leaflet<span class="sc">::</span><span class="fu">addPolygons</span>(<span class="at">map =</span> .,</span>
<span id="cb5-3"><a href="#cb5-3" aria-hidden="true" tabindex="-1"></a>                       <span class="at">data =</span> mich<span class="sc">$</span>sf,</span>
<span id="cb5-4"><a href="#cb5-4" aria-hidden="true" tabindex="-1"></a>                       <span class="at">color =</span> <span class="st">"green"</span>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
</section>
<section id="result-1" class="level5" data-number="5.4.1.1.2">
<h5 data-number="5.4.1.1.2" class="anchored" data-anchor-id="result-1"><span class="header-section-number">5.4.1.1.2</span> Result</h5>
<div class="cell" data-hash="04-neotoma2_cache/html/plotLeaf_9ad2239e07e28f1e97f352fd584faa74">
<div class="cell-output-display">
<div class="leaflet html-widget html-fill-item-overflow-hidden html-fill-item" id="htmlwidget-da5b3f82c20870f83b2c" style="width:100%;height:464px;"></div>
<script type="application/json" data-for="htmlwidget-da5b3f82c20870f83b2c">{"x":{"options":{"crs":{"crsClass":"L.CRS.EPSG3857","code":null,"proj4def":null,"projectedBounds":null,"options":{}}},"calls":[{"method":"addProviderTiles","args":["Stamen.TerrainBackground",null,null,{"errorTileUrl":"","noWrap":false,"detectRetina":false}]},{"method":"addTiles","args":["https://{s}.tile.openstreetmap.org/{z}/{x}/{y}.png",null,null,{"minZoom":0,"maxZoom":18,"tileSize":256,"subdomains":"abc","errorTileUrl":"","tms":false,"noWrap":false,"zoomOffset":0,"zoomReverse":false,"opacity":1,"zIndex":1,"detectRetina":false,"attribution":"&copy; <a href=\"https://openstreetmap.org\">OpenStreetMap<\/a> contributors, <a href=\"https://creativecommons.org/licenses/by-sa/2.0/\">CC-BY-SA<\/a>"}]},{"method":"addCircleMarkers","args":[[42.08806,43.12389,41.63794,44.7,43.48,41.92,42.33,44.88333,41.67,44.78417,45.6,45.06667,43.25,45.62,45.3,45.28,45.293028,45.18,45.05,44.98,45.08,44.93,44.75,44.63667,44.82,44.73,44.68,44.43,44.2,44.32,44.43,44.25,44.42,43.98,43.85,44,43.892714,44,43.98,43.7,43.677906,43.77,43.62,43.47,43.73,43.05,43.35,43.17,43.27,43.23,43.22,43.38,43.12,42.83,42.8,42.88,42.8,42.58,42.55,42.45,42.58,42.37,42.43,42.45,42.33,42.58,42.23,42.32,42.18,42.18,42.08,41.95,41.8,41.95,45.58,43.17,43.41667,42.4,41.58306,43.38333,45.81667,45.8,43,45.75,42.61667,43.11667,43.25,43.25,43.5,43.86667,43.36667,44.86667,42.2,42.5,43.25,42.61667,41.75,43.26667,43.75,45.68333,43.38333,43.36667,43.25,43.61667,41.86667,41.73333,45.86667,45.86667,45.86667,43.05,44.86667,42.11667,43.25,45.75,43.5,44.61667,44.36667,43.86667,45.86667,43,43,43.5,42.61667,42.61667,43.67639,43.61667,43.25,43.25,41.75,42.5,43.5,42.61667,42.5,42.61667,43.36667,42.11667,42.11667,42.25,42.25,42.25,41.86667,43.36667,43.25,43.86667,42.5,42.61667,42.25,41.80833,42.75,42.25,42.11667,42.96667,43.61667,44.25,42.36667,41.75,42.5,42.11667,43.25,42.11667,42,42.61667,42.75,43.36667,43.25,41.75,42.2,42.11667,41.86667,42.11667,41.86667,42.36667,42.83333,43.86667,44.11667,43.25,42.11667,41.86667,42.61667,43.11667,42.86667,43,42.86667,42.61667,41.86667,42,43.28333,42.25,42.86667,42.5,42.75,43,43.5,43.44167,43.25,42.86667,42.5,42.61667,43.61667,41.86667,42.75,43.36667,43.25,45.40889,45.0248,45.711714,43.263084,42.663308,42.06176,44.225546,41.900378,42.508146,44.42611,41.62264,42.08975,44.662242,42.08806,41.614226,42.289622,43.354304,44.02141,42.8594,42.550928,43.251548,41.757238,44.559244,44.987468,45.233824,45.269172,44.806006,44.956924,43.990492,43.593596,43.862978,43.85714,42.188992,42.189886,42.291996,41.709336,41.650122,41.777876,41.647994,41.62694,43.217426,42.08806,45.107052,42.08333,42.3,42.32,42.12,42.03,42.35,42.22,43.09639,42.66,42.09,42.18,42,43.05028,41.78,41.76,41.74,42.08333,41.70757,43.22835,42.21176,42.21592,45.06292,42.72456,41.551,42.61823,44.63943,42.32383,41.65016,43.85268,42.12122,45.41455,45.16506,41.90887,43.83822,41.57788,44.923236,45.008684,44.708582,45.126492,44.838888,44.739828,44.69277,44.947418,44.903598,42.42304,41.58504,44.43081,42.51691,42.58198,41.72093,43.49985,41.5534,42.74496,43.23106,41.56235,41.93461,43.92943,42.426,43.21856,42.08118,41.58028,42.383,41.67863,42.82731,43.15032,42.39865,43.9911,42.61644,42.276,42.4783,42.47148,42.18669,43.70497,43.62,41.67158,41.62711,45.130618,45.028548,44.641044,44.599476],[-82.6375,-83.24111,-86.545192,-84.95833,-85,-82.76,-83.63,-85.11667,-86.53,-84.5925,-84.31667,-84.78333,-84.92,-84.01667,-84.73,-83.95,-83.96693,-85.28,-84.7,-84.3,-84.31667,-83.8,-85.72,-85.15611,-84.48,-84.1,-83.4,-86,-85.97,-85.33,-84.7,-84.48,-83.97,-86.32,-85.83,-85.42,-84.94226,-84.18,-83.17,-85.87,-85.394954,-84.95,-84.73,-84.83,-83.93,-83.82,-86.08,-85.85,-85.35,-85.05,-83.47,-83.25,-85.12,-85.47,-84.88,-84.45,-83.8,-86.13,-85.42,-84.78,-84.7,-83.98,-84.02,-83.75,-83.61667,-83.3,-86.2,-85.52,-85.12,-84.42,-86.18,-85.7,-84.98,-84.28,-84.67,-84.83,-84.88333,-85.38333,-85.25,-83.98333,-84.58333,-84.75,-83.25,-86.75,-83.36667,-85.36667,-83.86667,-83.86667,-83.86667,-83.86667,-85.75,-86,-85.41667,-82.86667,-83.86667,-82.75,-86.25,-86.11667,-83.86667,-86.73333,-84.05,-84,-83.86667,-84.36667,-86.25,-83.48333,-84.75,-84.75,-84.75,-86.05,-85.75,-83.61667,-83.75,-84.75,-83.86667,-83.5,-86.11667,-83.25,-84.75,-82.61667,-83.25,-83.61667,-85.75,-85.86667,-84.34583,-84.25,-84.75,-84.5,-84.25,-83.75,-86.25,-83.25,-83.5,-83.36667,-85.75,-83.61667,-83.86667,-83.86667,-83.75,-86.11667,-85.25,-82.75,-83.75,-85.25,-84.25,-83.86667,-83.61667,-86.48333,-83.36667,-83.75,-86.25,-83.96667,-83,-83.5,-84.36667,-84.11667,-83.61667,-86.25,-84.86667,-86.11667,-85.86667,-83.5,-84.25,-85.61667,-86,-85.5,-85.41667,-85.36667,-84.61667,-84,-86.25,-84.75,-83.33333,-86.36667,-84,-85.25,-85.75,-86.25,-84.61667,-83.61667,-83.75,-83.5,-83.75,-84.86667,-85.11667,-86.5,-83.19167,-85.36667,-85,-84.11667,-84.25,-83.25,-84.86667,-84.83333,-84.5,-85.61667,-86,-84,-82.75,-85.75,-82.86667,-84.75,-83.86667,-86.82778,-83.616,-85.560956,-85.329328,-85.484808,-86.286074,-85.354406,-85.1083,-85.270892,-83.40998,-85.212974,-82.64544,-85.748768,-82.6375,-86.848486,-85.637356,-85.35784,-85.28417,-85.2144,-85.499636,-85.060706,-86.522242,-86.185328,-84.179618,-83.963046,-84.633798,-84.931208,-84.743334,-84.90423,-82.873406,-85.833676,-85.979354,-85.95376,-86.102936,-85.947184,-85.119754,-85.051948,-85.374006,-84.767808,-84.065,-83.46062,-82.6375,-83.638284,-82.625,-82.68,-82.84,-82.97,-82.92,-82.5,-82.8,-85.23306,-82.43,-82.47,-82.72,-82.83333,-82.59583,-82.44,-82.47,-82.44,-82.58333,-85.55972,-85.98783,-83.52098,-83.58492,-83.45699,-83.08541,-85.28682,-83.50589,-85.78652,-85.46734,-85.08176,-84.77215,-84.32304,-84.80371,-84.9776,-85.89437,-85.07388,-85.66354,-85.884822,-86.121768,-86.131692,-86.02572,-86.06419,-86.062164,-86.18553,-85.901532,-85.95366,-83.9967,-84.93076,-83.92049,-83.79582,-83.84599,-85.00769,-86.02363,-85.44424,-83.31057,-86.28844,-85.39243,-85.33089,-86.43465,-84.93952,-85.468,-86.21029,-84.92309,-86.04986,-86.3695,-83.74694,-85.47564,-84.05681,-86.10689,-85.20169,-84.21804,-86.06684,-85.5257,-85.57347,-86.42622,-85.28224,-85.04989,-85.02365,-84.781304,-84.636414,-83.88304,-83.769296],10,null,null,{"interactive":true,"draggable":false,"keyboard":true,"title":"","alt":"","zIndexOffset":0,"opacity":1,"riseOnHover":true,"riseOffset":250,"stroke":true,"color":"#03F","weight":5,"opacity.1":0.5,"fill":true,"fillColor":"#03F","fillOpacity":0.2},{"showCoverageOnHover":true,"zoomToBoundsOnClick":true,"spiderfyOnMaxZoom":true,"removeOutsideVisibleBounds":true,"spiderLegPolylineOptions":{"weight":1.5,"color":"#222","opacity":0.5},"freezeAtZoom":false},null,["<b>Bondi Section<\/b><br><b>Description:<\/b> Gravel pit. Physiography: shoreface sands,Arkona/Whittlesey trans.<br><a href=http://apps.neotomadb.org/explorer/?siteids=294>Explorer Link<\/a>","<b>Chippewa Bog<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=360>Explorer Link<\/a>","<b>Clear Lake<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=479>Explorer Link<\/a>","<b>Cub Lake<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=518>Explorer Link<\/a>","<b>Demont Lake<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=661>Explorer Link<\/a>","<b>Lake Erie<\/b><br><b>Description:<\/b> Lake Erie.<br><a href=http://apps.neotomadb.org/explorer/?siteids=785>Explorer Link<\/a>","<b>Frains Lake<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=821>Explorer Link<\/a>","<b>Green Lake<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=952>Explorer Link<\/a>","<b>Hudson Lake<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=1100>Explorer Link<\/a>","<b>Jones Lake<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=1360>Explorer Link<\/a>","<b>Lake Sixteen<\/b><br><b>Description:<\/b> Small lake in peatland. Physiography: low relief with sand ridges. Surrounding vegetation: Pinus/Quercus/Betula/Populus.<br><a href=http://apps.neotomadb.org/explorer/?siteids=1540>Explorer Link<\/a>","<b>Lake 27<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=1541>Explorer Link<\/a>","<b>Crystal Lake<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=2207>Explorer Link<\/a>","<b>Lancaster Lake<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=2655>Explorer Link<\/a>","<b>Carpenter Lake<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=2656>Explorer Link<\/a>","<b>Weber Lake<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=2657>Explorer Link<\/a>","<b>Lost Lake<\/b><br><b>Description:<\/b> Lake is of glacial origin. Terrestrial Plant Community: oak, pine, birch, and willow.<br><a href=http://apps.neotomadb.org/explorer/?siteids=2658>Explorer Link<\/a>","<b>Esau Lake<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=2659>Explorer Link<\/a>","<b>One Lake<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=2660>Explorer Link<\/a>","<b>Gaylanta Lake<\/b><br><b>Description:<\/b> Kettle lake with complex morphometry. Many houses around the lake. Highland around this kettle feature is end moraine. Upland vegetation northern hardwoods-pine forest.<br><a href=http://apps.neotomadb.org/explorer/?siteids=2661>Explorer Link<\/a>","<b>Jackson Lake<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=2662>Explorer Link<\/a>","<b>Beaver Lake<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=2663>Explorer Link<\/a>","<b>South Twin Lake<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=2664>Explorer Link<\/a>","<b>East Lake<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=2665>Explorer Link<\/a>","<b>Shupac Lake<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=2666>Explorer Link<\/a>","<b>South Perry Lake<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=2667>Explorer Link<\/a>","<b>Brownlee Lake<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=2668>Explorer Link<\/a>","<b>Healy Lake<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=2669>Explorer Link<\/a>","<b>Crystal Lake<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=2670>Explorer Link<\/a>","<b>Round Lake<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=2671>Explorer Link<\/a>","<b>Higgins Lake<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=2672>Explorer Link<\/a>","<b>Rifle Lake<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=2673>Explorer Link<\/a>","<b>Hackert Lake [Crystal Lake]<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=2674>Explorer Link<\/a>","<b>North Lake<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=2675>Explorer Link<\/a>","<b>Wells Lake<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=2676>Explorer Link<\/a>","<b>Budd Lake<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=2677>Explorer Link<\/a>","<b>Clear Lake<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=2678>Explorer Link<\/a>","<b>Bently Lake<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=2679>Explorer Link<\/a>","<b>Rush Lake<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=2680>Explorer Link<\/a>","<b>Woodland Lake<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=2681>Explorer Link<\/a>","<b>Clear Lake<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=2682>Explorer Link<\/a>","<b>Littlefield Lake<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=2683>Explorer Link<\/a>","<b>Salsuesha Lake<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=2684>Explorer Link<\/a>","<b>Topico Marsh<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=2685>Explorer Link<\/a>","<b>Half Moon Lake<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=2686>Explorer Link<\/a>","<b>Watcheer Pond<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=2687>Explorer Link<\/a>","<b>Duck Lake<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=2688>Explorer Link<\/a>","<b>Crockery Lake<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=2689>Explorer Link<\/a>","<b>Lincoln Lake<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=2690>Explorer Link<\/a>","<b>Holland Lake<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=2691>Explorer Link<\/a>","<b>Otter Lake<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=2692>Explorer Link<\/a>","<b>Shay Lake<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=2693>Explorer Link<\/a>","<b>Long Lake<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=2694>Explorer Link<\/a>","<b>Campau Lake<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=2695>Explorer Link<\/a>","<b>Mullenkin Lake<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=2696>Explorer Link<\/a>","<b>Round Lake<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=2697>Explorer Link<\/a>","<b>Bryam Lake<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=2698>Explorer Link<\/a>","<b>Hutchin's Lake<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=2699>Explorer Link<\/a>","<b>Guernsey Lake<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=2700>Explorer Link<\/a>","<b>Narrow Lake<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=2701>Explorer Link<\/a>","<b>Taylor Lake<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=2702>Explorer Link<\/a>","<b>Blind Lake<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=2703>Explorer Link<\/a>","<b>Pickerel Lake<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=2704>Explorer Link<\/a>","<b>Whitmore Lake<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=2705>Explorer Link<\/a>","<b>Murray Lake<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=2706>Explorer Link<\/a>","<b>Sodon Lake<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=2707>Explorer Link<\/a>","<b>Rush Lake<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=2708>Explorer Link<\/a>","<b>Crooked Lake<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=2709>Explorer Link<\/a>","<b>Pleasant Lake<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=2710>Explorer Link<\/a>","<b>Lyon Lake<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=2711>Explorer Link<\/a>","<b>Lee Lake<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=2712>Explorer Link<\/a>","<b>Hayck Lake<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=2713>Explorer Link<\/a>","<b>Vandercook Lake<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=2714>Explorer Link<\/a>","<b>Round Lake<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=2715>Explorer Link<\/a>","<b>Douglas Lake<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=2716>Explorer Link<\/a>","<b>McCracken's Pond<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=2783>Explorer Link<\/a>","<b>Vestaburg Bog<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=2814>Explorer Link<\/a>","<b>Wintergreen Lake<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=2864>Explorer Link<\/a>","<b>Pretty Lake<\/b><br><b>Description:<\/b> Kettle lake. Physiography: broad (Dekalb)lowland of moderate relief. Surrounding vegetation: cultivation/Quercus/Carya/Acer/Populus.<br><a href=http://apps.neotomadb.org/explorer/?siteids=2954>Explorer Link<\/a>","<b>Schultz<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=3571>Explorer Link<\/a>","<b>Juntunen [20MK1]<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=3641>Explorer Link<\/a>","<b>Ft. Michilimackinac<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=3642>Explorer Link<\/a>","<b>Younge<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=3650>Explorer Link<\/a>","<b>Ogontz Bay [USFS328]<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=4235>Explorer Link<\/a>","<b>Walled Lake Ursus<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=4346>Explorer Link<\/a>","<b>Greenville Ursus<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=4421>Explorer Link<\/a>","<b>Fosters [20SA74]<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=4526>Explorer Link<\/a>","<b>Schmidt [20SA192]<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=4527>Explorer Link<\/a>","<b>Kantzler [20BY30]<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=4528>Explorer Link<\/a>","<b>Lalone [20ARL]<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=4529>Explorer Link<\/a>","<b>Stadelmeyer [20SA195]<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=4530>Explorer Link<\/a>","<b>Sleeping Bear Dune<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=5330>Explorer Link<\/a>","<b>Coville Farm<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=5363>Explorer Link<\/a>","<b>Wolf<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=5364>Explorer Link<\/a>","<b>Bridgeport [20SA620]<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=5365>Explorer Link<\/a>","<b>Riviere Au Vase [Graham/Greene School/Mud Creek] [20MB3]<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=5366>Explorer Link<\/a>","<b>Moccasin Bluff [20BE-B8]<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=5367>Explorer Link<\/a>","<b>Spring Creek [20MU3]<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=5368>Explorer Link<\/a>","<b>Butterfield<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=5369>Explorer Link<\/a>","<b>Burnt Bluff Cave [B-95]<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=5370>Explorer Link<\/a>","<b>Feeheley [20SA128]<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=5375>Explorer Link<\/a>","<b>Mahoney [20SA193]<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=5531>Explorer Link<\/a>","<b>Younge [20SA209]<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=5532>Explorer Link<\/a>","<b>Naugle [20MD30]<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=5533>Explorer Link<\/a>","<b>Berrien Springs Jail House<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=5534>Explorer Link<\/a>","<b>Morin [20MO40]<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=5535>Explorer Link<\/a>","<b>Sposito [SIS-29]<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=5536>Explorer Link<\/a>","<b>Norge Village<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=5537>Explorer Link<\/a>","<b>Beyer<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=5538>Explorer Link<\/a>","<b>Spoonville [20OT1]<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=5539>Explorer Link<\/a>","<b>Fisher Lake [20LU21]<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=5540>Explorer Link<\/a>","<b>20WA176<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=5541>Explorer Link<\/a>","<b>Birch Run Road [20SA393]<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=5542>Explorer Link<\/a>","<b>Mission of St. Ignace<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=5543>Explorer Link<\/a>","<b>Trombly House [20BY70]<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=5544>Explorer Link<\/a>","<b>Upper Buff Creek [20AA128]<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=5545>Explorer Link<\/a>","<b>Point Arcadia [20MT120]<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=5546>Explorer Link<\/a>","<b>Caseville Airport [20HU164]<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=5547>Explorer Link<\/a>","<b>Gros Cap [MK6/7/111]<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=5548>Explorer Link<\/a>","<b>Veitch [20SC77]<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=5549>Explorer Link<\/a>","<b>20LP98<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=5550>Explorer Link<\/a>","<b>Jahr [20TU6]<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=5551>Explorer Link<\/a>","<b>DeBoer [20AE62]<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=5552>Explorer Link<\/a>","<b>46th Street Site<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=5554>Explorer Link<\/a>","<b>Bailer Site<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=5555>Explorer Link<\/a>","<b>Midland Site<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=5556>Explorer Link<\/a>","<b>Shaffer<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=5557>Explorer Link<\/a>","<b>Ruskovic Site<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=5558>Explorer Link<\/a>","<b>Cadmus Road<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=5559>Explorer Link<\/a>","<b>Four Lakes Drive Site<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=5560>Explorer Link<\/a>","<b>Rothbury Mastodon Site<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=5561>Explorer Link<\/a>","<b>Troy Mastodon Site<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=5562>Explorer Link<\/a>","<b>New Hudson Site<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=5563>Explorer Link<\/a>","<b>Groleau-White Lake Site<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=5564>Explorer Link<\/a>","<b>Jolman Site<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=5565>Explorer Link<\/a>","<b>Sakstrup Site<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=5566>Explorer Link<\/a>","<b>Pleasant Lake Site<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=5567>Explorer Link<\/a>","<b>Kuhl Site<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=5568>Explorer Link<\/a>","<b>Killin Gravel Pit<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=5569>Explorer Link<\/a>","<b>Shine Site<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=5570>Explorer Link<\/a>","<b>Zeller Site<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=5571>Explorer Link<\/a>","<b>Bryce Site<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=5572>Explorer Link<\/a>","<b>Maurer Site<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=5573>Explorer Link<\/a>","<b>Evart Site<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=5574>Explorer Link<\/a>","<b>Mead Site<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=5575>Explorer Link<\/a>","<b>Adams Site<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=5576>Explorer Link<\/a>","<b>Ann Arbor Platygonus Site<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=5577>Explorer Link<\/a>","<b>Prillwitz Site<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=5578>Explorer Link<\/a>","<b>Flanders Site<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=5579>Explorer Link<\/a>","<b>Ann Arbor Alces Site<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=5580>Explorer Link<\/a>","<b>Watervliet Site<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=5581>Explorer Link<\/a>","<b>Lennon [Flint] Site<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=5582>Explorer Link<\/a>","<b>Sweeney Site<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=5583>Explorer Link<\/a>","<b>Henry Site<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=5584>Explorer Link<\/a>","<b>Betz Site<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=5585>Explorer Link<\/a>","<b>Seneca Township Site<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=5586>Explorer Link<\/a>","<b>American Aggregates Gravel Pit<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=5587>Explorer Link<\/a>","<b>Carmichael Site<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=5588>Explorer Link<\/a>","<b>Zamarron Site<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=5589>Explorer Link<\/a>","<b>Hartford Proboscidean<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=5590>Explorer Link<\/a>","<b>Powers Site<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=5591>Explorer Link<\/a>","<b>Green Site<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=5592>Explorer Link<\/a>","<b>Harper<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=5593>Explorer Link<\/a>","<b>Croton Muskox<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=5594>Explorer Link<\/a>","<b>Moreland Muskox<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=5595>Explorer Link<\/a>","<b>White Pigeon Muskox<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=5596>Explorer Link<\/a>","<b>Scotts Muskox Spec. 2146 [KMP-2146]<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=5597>Explorer Link<\/a>","<b>Scotts Muskox Spec. 34124 [UMMP-34124]<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=5598>Explorer Link<\/a>","<b>Hillsdale Muskox<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=5599>Explorer Link<\/a>","<b>Manchester Muskox<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=5600>Explorer Link<\/a>","<b>Berrien Muskox<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=5601>Explorer Link<\/a>","<b>Heisler Site<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=5602>Explorer Link<\/a>","<b>Shelton Site [20OK394]<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=5603>Explorer Link<\/a>","<b>Ludington Castoroides<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=5604>Explorer Link<\/a>","<b>Van Horn Mammoth<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=5605>Explorer Link<\/a>","<b>Turk Lake<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=5606>Explorer Link<\/a>","<b>Paw Paw Muskox<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=5607>Explorer Link<\/a>","<b>Dowagiac River Site<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=5608>Explorer Link<\/a>","<b>I-96 Site<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=5609>Explorer Link<\/a>","<b>Taylor Site<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=5610>Explorer Link<\/a>","<b>Ray Road Site<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=5611>Explorer Link<\/a>","<b>Johnson Site<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=5612>Explorer Link<\/a>","<b>Cole Site<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=5613>Explorer Link<\/a>","<b>Parker Site<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=5614>Explorer Link<\/a>","<b>Bronson<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=5615>Explorer Link<\/a>","<b>Berndt Site<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=5616>Explorer Link<\/a>","<b>Rappuhn Site<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=5617>Explorer Link<\/a>","<b>Vanmiddlesworth Site<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=5618>Explorer Link<\/a>","<b>Crandell Site<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=5619>Explorer Link<\/a>","<b>Sheathelm Site<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=5620>Explorer Link<\/a>","<b>Gullich Road Site<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=5621>Explorer Link<\/a>","<b>Van Sickle Site<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=5622>Explorer Link<\/a>","<b>Winn Mastodon<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=5623>Explorer Link<\/a>","<b>Thaller Site<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=5624>Explorer Link<\/a>","<b>Smith Site<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=5625>Explorer Link<\/a>","<b>Grandville<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=5706>Explorer Link<\/a>","<b>Schwerdt [20AE127]<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=5707>Explorer Link<\/a>","<b>Fowlerville Caribou<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=5711>Explorer Link<\/a>","<b>Minden City Caribou<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=5712>Explorer Link<\/a>","<b>Glacial Lake Mogodore Mammuthus<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=5713>Explorer Link<\/a>","<b>Holcombe Beach [20MB30]<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=5721>Explorer Link<\/a>","<b>Pitt Farm<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=5762>Explorer Link<\/a>","<b>20SA1034<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=6299>Explorer Link<\/a>","<b>Rock Island<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=6370>Explorer Link<\/a>","<b>Irwin Smith Bog<\/b><br><b>Description:<\/b> Ombrotrophic peatland located near Lake Huron. Surrounding vegetation: Sphagnum mosses, various graminoids (mainly Carex), low-growing ericaceous shrubs (Kalmia polifolia, Chamaedaphne calyculata), and scattered trees (Larix laricina, Picea mariana, Pinus strobus) characterize the hummock-hollow topography of the bog surface. Regional vegetation before 19th century land clearance comprised a mosaic of mixed conifer/hardwood forests, pine forests, and conifer swamps.<br><a href=http://apps.neotomadb.org/explorer/?siteids=8559>Explorer Link<\/a>","<b>Barney's Lake<\/b><br><b>Description:<\/b> Closed basin lake on Beaver Island in Lake Michigan. Possible glacial kettle lake or may occupy a former tunnel valley or glacial channel (https://gsa.confex.com/gsa/2005NC/finalprogram/abstract_87295.htm). Topography is characterized by dunes, swales, and till. Apple orchards are abundant and some wetlands are present. There are many seeps around the lake with cold temps (9-14 degrees C), all associated with chara. Alignment of Barney's Lake and Egg Lake in an elongate depression suggests fracture or joint-related karstification with the possibility of bedrock. Sources for groundwater seeps. For Holocene history of this lake see:. . . Kapp, R.D., Bushouse, S., and Foster, B., 1969, A Contribution to the Geology and Forest History of Beaver IL, Michigan, Proc. 12th Conf. Grt. Lakes Res., p. 225-236.<br><a href=http://apps.neotomadb.org/explorer/?siteids=8580>Explorer Link<\/a>","<b>Bass Lake<\/b><br><b>Description:<\/b> A lake with public access and moderate build-up of houses around the shore. Additional concern regarding a trailer park that is built right up to the water's edge. Lake stage is 1-2 feet above normal and drowned trees indicate that this condition is recent (last few years). Wetlands are dominated by cattails and act as recharge areas.<br><a href=http://apps.neotomadb.org/explorer/?siteids=8582>Explorer Link<\/a>","<b>Bassett Lake<\/b><br><b>Description:<\/b> Public access lake with very little development and woods around it. There are several seeps along the base of the highlands. The wetlands around the lake are cattail/tamerack/nuphar dominated. The lake has an extensive marl and chara bench. The hypolimnion is probably anoxic during the summer.<br><a href=http://apps.neotomadb.org/explorer/?siteids=9734>Explorer Link<\/a>","<b>Black Lake<\/b><br><b>Description:<\/b> Terrestrial Plant Community: oak and maple. The rest of the area is heavily wooded. Marl benches are present, although small.<br><a href=http://apps.neotomadb.org/explorer/?siteids=9777>Explorer Link<\/a>","<b>Berry Lake [Frog Lake]<\/b><br><b>Description:<\/b> Kettle lake in glacial outwash. The water is very brown from tannins from a strand of tamaracks in adjacent wetland. Upland vegetation: oak with some pine and birch. Tamaracks and heath located in adjacent wetland area.<br><a href=http://apps.neotomadb.org/explorer/?siteids=9787>Explorer Link<\/a>","<b>Cary Lake<\/b><br><b>Description:<\/b> Glacial origin lake apparently within a glacial meltwater channel. Lake is very clean and deep with houses along one side and wetland and woods along the other side. It has an extensive carbonate bench with a steep drop-off to deeper water.<br><a href=http://apps.neotomadb.org/explorer/?siteids=9896>Explorer Link<\/a>","<b>Clear Lake<\/b><br><b>Description:<\/b> Glacial origin lake, kettle or meltwater channel. There are some houses along one side and the rest of the lake is surrounded by woods and wetlands.<br><a href=http://apps.neotomadb.org/explorer/?siteids=9905>Explorer Link<\/a>","<b>Au Sable River<\/b><br><b>Description:<\/b> The site is on the south bank of the Au Sable river in the SW1/4, SW 1/4,SE1/4, Sec.36, T. 24 N., R. 8 E<br><a href=http://apps.neotomadb.org/explorer/?siteids=9946>Explorer Link<\/a>","<b>Appleman Lake<\/b><br><b>Description:<\/b> Appleman Lake is a kettle pond situated on glacial till and outwash from the Lake Michigan Lobe.<br><a href=http://apps.neotomadb.org/explorer/?siteids=10003>Explorer Link<\/a>","<b>Leamington Site<\/b><br><b>Description:<\/b> The Bondi site is a gravel pit located on a broad hill NW of Leamington, Essex County, Ontario. The hill was an island in a succession of Erie basin gracial lakes.<br><a href=http://apps.neotomadb.org/explorer/?siteids=10067>Explorer Link<\/a>","<b>Ellis Lake<\/b><br><b>Description:<\/b> Kettle lake surrounded by wetlands.<br><a href=http://apps.neotomadb.org/explorer/?siteids=10104>Explorer Link<\/a>","<b>Glacial Lake Arkona-Bondi Site<\/b><br><b>Description:<\/b> The deposits that produced the bone here were part of a proglacial subaquatic fan that was deposited in glacial Lake Maumee. This deposit was subsequently only modified by lake currents. The aggregate excavation that uncovered this bone at the Bondi site took place within this fan's surface. Site is located between Leamington and Colchester in Essex County, Ontario.<br><a href=http://apps.neotomadb.org/explorer/?siteids=10125>Explorer Link<\/a>","<b>Pinhook Bog<\/b><br><b>Description:<\/b> Bog occupies a depression in an area of kame and kettle topography on the Valparaiso moraine. Surrounding vegetation: Oak-dominated woodland and agricultural land. Prior to European settlement, the vegetation is assumed to have been oak-hickory forest and prairie.<br><a href=http://apps.neotomadb.org/explorer/?siteids=10129>Explorer Link<\/a>","<b>Route 458 Bog<\/b><br><b>Description:<\/b> A small treed bog which occupies a depression on the Newberry moraine. Surrounding vegetation: Northern hardwoods forest of Fagus grandifolia, Acer saccharum, and Tsuga canadensis. A carpet of Sphagnum covers the bog. Carex oligosperma is ubiquitous.<br><a href=http://apps.neotomadb.org/explorer/?siteids=10166>Explorer Link<\/a>","<b>Cowden Lake<\/b><br><b>Description:<\/b> Lake located in the forest tension zone, an ecotone between the mixed coniferous-deciduous forest and the eastern broadleaf forest in the Lower Peninsula of Michigan.<br><a href=http://apps.neotomadb.org/explorer/?siteids=10239>Explorer Link<\/a>","<b>Hicks Lake<\/b><br><b>Description:<\/b> Lake located in the forest tension zone, an ecotone between the mixed coniferous-deciduous forest and the eastern broadleaf forest in the Lower Peninsula of Michigan.<br><a href=http://apps.neotomadb.org/explorer/?siteids=10242>Explorer Link<\/a>","<b>Morrison Lake<\/b><br><b>Description:<\/b> Lake located in the forest tension zone, an ecotone between the mixed coniferous-deciduous forest and the eastern broadleaf forest in the Lower Peninsula of Michigan.<br><a href=http://apps.neotomadb.org/explorer/?siteids=10251>Explorer Link<\/a>","<b>Fish Lake<\/b><br><b>Description:<\/b> Kettle lake is surrounded by wetlands and forest. The wetland that fringes the lake is wide and partly submerged with a large marl bench.<br><a href=http://apps.neotomadb.org/explorer/?siteids=10264>Explorer Link<\/a>","<b>Halfmoon Lake<\/b><br><b>Description:<\/b> Glacial kettle lake.<br><a href=http://apps.neotomadb.org/explorer/?siteids=10423>Explorer Link<\/a>","<b>Spicer Lake<\/b><br><b>Description:<\/b> Spicer Lake is located in New Carlisle, St. Joseph County, northern Indiana. It is a kettle lake formed by stagnant ice embedded in the glacial till left behind when the Lake Michigan Lobe of the Laurentide Ice Sheet retreated into Indiana ca. 15,000 years ago. Much of the kettle has filled in, leaving 5 ha of open water, surrounded by swamp and wetland forests.<br><a href=http://apps.neotomadb.org/explorer/?siteids=13350>Explorer Link<\/a>","<b>Upper Herring Lake<\/b><br><b>Description:<\/b> Lake is an embayment of Lake Michigan and is separated from the main lake by a barrier of sand and a Lower Herring Lake. Also blocked by a sand barrier from the main lake. These types of lakes, independent now, but once embayments of Lake Michigan are common on the East side Lake Michigan. Area surrounding is wooded with some orchards and berry farms.<br><a href=http://apps.neotomadb.org/explorer/?siteids=13457>Explorer Link<\/a>","<b>Lake Fifteen<\/b><br><b>Description:<\/b> Kettle Lake in till, with obvious supply from carbonate and dolomite sources. Extensive marl benches with Chara and molluscs. The lake is very steep-sided with groundwater sapping along the shore very evident.<br><a href=http://apps.neotomadb.org/explorer/?siteids=13466>Explorer Link<\/a>","<b>Lake Ferdelman<\/b><br><b>Description:<\/b> Kettle lake.<br><a href=http://apps.neotomadb.org/explorer/?siteids=13467>Explorer Link<\/a>","<b>Silver Lake<\/b><br><b>Description:<\/b> Small kettle lake surrounded by pine, hemlock, and mixed hardwoods. . . Terrestrial Plant Community: hemlock, oak, pine, and birch.<br><a href=http://apps.neotomadb.org/explorer/?siteids=13577>Explorer Link<\/a>","<b>Indian Lake<\/b><br><b>Description:<\/b> Kettle lake. . Terrestrial Plant Community: hemlock, pine, oak, willow, and birch.<br><a href=http://apps.neotomadb.org/explorer/?siteids=13578>Explorer Link<\/a>","<b>Lake Manuka<\/b><br><b>Description:<\/b> Lake is of glacial origin. Terrestrial Plant Community: oak with some pine and birch and scirpus and heath located in wetland area.<br><a href=http://apps.neotomadb.org/explorer/?siteids=13579>Explorer Link<\/a>","<b>Lily Lake<\/b><br><b>Description:<\/b> Small kettle lake with public access located in the sand plains. . . Terrestrial Plant Community: mostly oak with pine, birch, and cherry.<br><a href=http://apps.neotomadb.org/explorer/?siteids=13581>Explorer Link<\/a>","<b>Minden Bog<\/b><br><b>Description:<\/b> Ombrotrophic, raised bog. Central portion dominated by low-growing shrubs (Chamaedaphne calyculata, Kalmia polifolia), widely scattered trees (Larix laricina, Betula papyrifera) and a dense groundcover of Sphagnum. Upland vegetation dominated by Betula papyrifera, Populus tremuloides, Alnus incana and Aronia prunifolia. Peat in the center of the bog is at least 7.5 m deep. Northern half of the bog has been mined for peat.<br><a href=http://apps.neotomadb.org/explorer/?siteids=13582>Explorer Link<\/a>","<b>North Lake<\/b><br><b>Description:<\/b> Small, marl-rich lake of a glacial origin with public access landing near town of Baldwin. . . Terrestrial Plant Community: pine, oak, maple, cherry, and wetlands with tamarack, bullrushes, and cattails.<br><a href=http://apps.neotomadb.org/explorer/?siteids=13584>Explorer Link<\/a>","<b>Reed Lake<\/b><br><b>Description:<\/b> Lake is of glacial origin. . Terrestrial Plant Community: mostly oak with some maple and pine. Wetlands are dominated by bullrush.<br><a href=http://apps.neotomadb.org/explorer/?siteids=13585>Explorer Link<\/a>","<b>Threemile Lake<\/b><br><b>Description:<\/b> Lake of glacial origin. . Terrestrial Plant Community: oak, maple, pine, and birch. Wetlands dominated by heath and cattails grading into nuphar beds.<br><a href=http://apps.neotomadb.org/explorer/?siteids=13586>Explorer Link<\/a>","<b>Shafer Lake<\/b><br><b>Description:<\/b> Lake is of glacial origin. . Terrestrial Plant Community: oak, maple, alder, cherry, and willow. Wetlands are nuphar dominated.<br><a href=http://apps.neotomadb.org/explorer/?siteids=13587>Explorer Link<\/a>","<b>School Section Lake<\/b><br><b>Description:<\/b> Lake is of glacial origin. . Terrestrial Plant Community: oak, maple, and willows.<br><a href=http://apps.neotomadb.org/explorer/?siteids=13588>Explorer Link<\/a>","<b>Lime Lake<\/b><br><b>Description:<\/b> Lake is of glacial origin. . Terrestrial Plant Community: mulberry, willow, cottonwood, and elm.<br><a href=http://apps.neotomadb.org/explorer/?siteids=13592>Explorer Link<\/a>","<b>Loon Lake<\/b><br><b>Description:<\/b> Lake is of glacial origin. . Terrestrial Plant Community: cottonwoods, willow, maple, nuphar, nymphea, and typha.<br><a href=http://apps.neotomadb.org/explorer/?siteids=13593>Explorer Link<\/a>","<b>Lee Lake<\/b><br><b>Description:<\/b> Lake is of glacial origin. . Terrestrial Plant Community: willlow and tamaracks.<br><a href=http://apps.neotomadb.org/explorer/?siteids=13607>Explorer Link<\/a>","<b>Mud Lake<\/b><br><b>Description:<\/b> Glacial Kettle Lake.<br><a href=http://apps.neotomadb.org/explorer/?siteids=13623>Explorer Link<\/a>","<b>The Winameg Mastodon Site<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=16019>Explorer Link<\/a>","<b>Otter Lake<\/b><br><b>Description:<\/b> Otter Lake is a medium sized kettle lake located in southeastern lower Michigan, and consisting of two sub-basins that are separated by a sill. The most common species in the wetlands surrounding Otter Lake are Larix laricina, Thuja occidentalis, Salix discolor, and a variety of Carex species. Dominant forest trees in the area are Fagus grandifolia, Acer rubrum, Cornus foemina, Ulmus americana, Quercus bicolor, and Juglans nigra in mesic soiles, while Quercus alba and Carya cordiformis are common in drier soils. <br><a href=http://apps.neotomadb.org/explorer/?siteids=16313>Explorer Link<\/a>","<b>Bondi [AaHp-VP]<\/b><br><b>Description:<\/b> Proglacial subaquatic fan with overlying bar deposits located northwest of Leamington, Essex County, Ontario.The Bondi site is on one of these fans, which provided a large terrestrial area that supported plants and animals. <br><a href=http://apps.neotomadb.org/explorer/?siteids=16740>Explorer Link<\/a>","<b>Herron Peatland<\/b><br><b>Description:<\/b> Sphagnum dominated peatland.<br><a href=http://apps.neotomadb.org/explorer/?siteids=16891>Explorer Link<\/a>","<b>Verbeke Mastodon [AaHp-VP]<\/b><br><b>Description:<\/b> Roger Verbeke's farm, Con. 4, west of. (BGS 228) Hwy. 77. Mersea Township, near Leamington, Essex County, Ontario.<br><a href=http://apps.neotomadb.org/explorer/?siteids=16921>Explorer Link<\/a>","<b>Lake St. Claire<\/b><br><b>Description:<\/b> Delorme Site Codes; Land, 15; Physical, 5; Sediment, S; Aquatic, P; Distance From Shore: 0.3<br><a href=http://apps.neotomadb.org/explorer/?siteids=17281>Explorer Link<\/a>","<b>Lake St. Claire/Pike Creek<\/b><br><b>Description:<\/b> Delorme Site Codes; Land, 15; Physical, 2; Sediment, H; Aquatic, P; Distance From Shore: 3<br><a href=http://apps.neotomadb.org/explorer/?siteids=17326>Explorer Link<\/a>","<b>Cedar River<\/b><br><b>Description:<\/b> Delorme Site Codes; Land, 15; Physical, 4; Sediment, H; Aquatic, P; Distance From Shore: 0.3<br><a href=http://apps.neotomadb.org/explorer/?siteids=17331>Explorer Link<\/a>","<b>LDDsite5519<\/b><br><b>Description:<\/b> Delorme Site Codes; Land, 15; Physical, 11; Sediment, S; Aquatic, P; Distance From Shore: 0.3<br><a href=http://apps.neotomadb.org/explorer/?siteids=17338>Explorer Link<\/a>","<b>Lake St. Clair<\/b><br><b>Description:<\/b> Delorme Site Codes; Land, 15; Physical, 1; Sediment, H; Aquatic, A<br><a href=http://apps.neotomadb.org/explorer/?siteids=17390>Explorer Link<\/a>","<b>LDDsite5897<\/b><br><b>Description:<\/b> Delorme Site Codes; Land, 15; Physical, 13; Sediment, S; Aquatic, P; Distance From Shore: 0.3<br><a href=http://apps.neotomadb.org/explorer/?siteids=17393>Explorer Link<\/a>","<b>Belding Peccary<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=17498>Explorer Link<\/a>","<b>LDDsite5885<\/b><br><b>Description:<\/b> Delorme Site Codes; Land, 15; Physical, 11; Sediment, S; Aquatic, A; Distance From Shore: 0.3<br><a href=http://apps.neotomadb.org/explorer/?siteids=18113>Explorer Link<\/a>","<b>LDDsite5894<\/b><br><b>Description:<\/b> Delorme Site Codes; Land, 15; Physical, 13; Sediment, S; Aquatic, P; Distance From Shore: 0.3<br><a href=http://apps.neotomadb.org/explorer/?siteids=18123>Explorer Link<\/a>","<b>Belle River<\/b><br><b>Description:<\/b> Delorme Site Codes; Land, 15; Physical, 13; Sediment, H; Aquatic, A; Distance From Shore: 0.3<br><a href=http://apps.neotomadb.org/explorer/?siteids=18126>Explorer Link<\/a>","<b>Bruner-Colasanti [AaHq-8]<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=21274>Explorer Link<\/a>","<b>Mill Creek Site<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=21757>Explorer Link<\/a>","<b>Lake Erie<\/b><br><b>Description:<\/b> Delorme Site Codes; Land, 15; Physical, 1; Sediment, S; Aquatic, A; Distance From Shore: 14650<br><a href=http://apps.neotomadb.org/explorer/?siteids=22262>Explorer Link<\/a>","<b>Lake Erie<\/b><br><b>Description:<\/b> Delorme Site Codes; Land, 15; Physical, 1; Sediment, S; Aquatic, A; Distance From Shore: 13000<br><a href=http://apps.neotomadb.org/explorer/?siteids=22273>Explorer Link<\/a>","<b>Lake Erie<\/b><br><b>Description:<\/b> Delorme Site Codes; Land, 15; Physical, 1; Sediment, S; Aquatic, A; Distance From Shore: 15600<br><a href=http://apps.neotomadb.org/explorer/?siteids=22274>Explorer Link<\/a>","<b>Robson Road [AaHp-20]<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=22399>Explorer Link<\/a>","<b>Hershberger<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=22593>Explorer Link<\/a>","<b>Moorland<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=22599>Explorer Link<\/a>","<b>Belleville Lake<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=22743>Explorer Link<\/a>","<b>Ford Lake<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=22864>Explorer Link<\/a>","<b>Lake Besser<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=22953>Explorer Link<\/a>","<b>Stony Creek Lake<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=23172>Explorer Link<\/a>","<b>Blackman Lake<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=23288>Explorer Link<\/a>","<b>Bogie Lake<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=23297>Explorer Link<\/a>","<b>Bridge Lake<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=23300>Explorer Link<\/a>","<b>Campbell Lake<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=23312>Explorer Link<\/a>","<b>Cheeseboro Lake<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=23323>Explorer Link<\/a>","<b>Clark Co. Lake<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=23324>Explorer Link<\/a>","<b>Clark Lake<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=23325>Explorer Link<\/a>","<b>Crooked Lake<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=23333>Explorer Link<\/a>","<b>Deer Lake<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=23346>Explorer Link<\/a>","<b>Donnell Lake<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=23352>Explorer Link<\/a>","<b>Eight Point Lake<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=23360>Explorer Link<\/a>","<b>Fish Lake<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=23371>Explorer Link<\/a>","<b>Bass Lake<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=24000>Explorer Link<\/a>","<b>Florence<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=24001>Explorer Link<\/a>","<b>Loon<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=24002>Explorer Link<\/a>","<b>Manitou<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=24003>Explorer Link<\/a>","<b>North Bar<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=24004>Explorer Link<\/a>","<b>Otter<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=24005>Explorer Link<\/a>","<b>Round<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=24006>Explorer Link<\/a>","<b>Shell<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=24007>Explorer Link<\/a>","<b>Tucker<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=24008>Explorer Link<\/a>","<b>Hi-Land Lake<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=24405>Explorer Link<\/a>","<b>Johnson Lake<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=24418>Explorer Link<\/a>","<b>Lake Au Sable<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=24429>Explorer Link<\/a>","<b>Lake Brighton<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=24502>Explorer Link<\/a>","<b>Lake Chemung<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=24506>Explorer Link<\/a>","<b>Little Otter<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=24559>Explorer Link<\/a>","<b>Martin Lake<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=24587>Explorer Link<\/a>","<b>Messick Lake<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=24598>Explorer Link<\/a>","<b>Mill Lake<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=24601>Explorer Link<\/a>","<b>Muskegon Lake<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=24612>Explorer Link<\/a>","<b>Olin Lake<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=24629>Explorer Link<\/a>","<b>Palmer Lake<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=24636>Explorer Link<\/a>","<b>Pere Marquette Lake<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=24641>Explorer Link<\/a>","<b>Pine Lake<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=24645>Explorer Link<\/a>","<b>Pine Lake (Ke)<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=24646>Explorer Link<\/a>","<b>Round Lake (Va)<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=24671>Explorer Link<\/a>","<b>Round Lake (St)<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=24674>Explorer Link<\/a>","<b>Saddle Lake<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=24679>Explorer Link<\/a>","<b>South Chain Lake<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=24699>Explorer Link<\/a>","<b>Squaw Lake<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=24711>Explorer Link<\/a>","<b>Stoner Lake<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=24715>Explorer Link<\/a>","<b>Sullivan Lake<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=24720>Explorer Link<\/a>","<b>Tallman Lake<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=24725>Explorer Link<\/a>","<b>Thornapple Lake<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=24727>Explorer Link<\/a>","<b>Tims Lake<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=24730>Explorer Link<\/a>","<b>Upper Scott<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=24754>Explorer Link<\/a>","<b>Warner Lake<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=24765>Explorer Link<\/a>","<b>West Lake<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=24768>Explorer Link<\/a>","<b>Wyckoff Lake<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=24779>Explorer Link<\/a>","<b>Blue Lake (Me)<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=24794>Explorer Link<\/a>","<b>Crooked Lake (St)<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=24798>Explorer Link<\/a>","<b>Fox Lake (St)<\/b><br><b>Description:<\/b> NA<br><a href=http://apps.neotomadb.org/explorer/?siteids=24802>Explorer Link<\/a>","<b>Huffman Lake<\/b><br><b>Description:<\/b> Huffman Lake is situated on an outwash plain between the inner and outer Port Huron moraine within the snowbelt. Surrounding vegetation: Second-growth forest dominated by Acer saccharum and Populus. Wetland forest communities composed primarily of Thuja occidentalis and Abies balsamea are located near the shoreline and along the stream draining the lake.<br><a href=http://apps.neotomadb.org/explorer/?siteids=28059>Explorer Link<\/a>","<b>Clifford Lake<\/b><br><b>Description:<\/b> Clifford Lake is situated on the Port Huron moraine. Surrounding vegetation: Forest communities are dominated by sugar maple, although much of the surrounding landscape is occupied by abandoned agricultural land. Pre-European settlement forest communities were dominated by sugar maple (Acer saccharum), American beech (Fagus grandifolia), and eastern hemlock (Tsuga canadensis).<br><a href=http://apps.neotomadb.org/explorer/?siteids=28100>Explorer Link<\/a>","<b>O'Brien Lake<\/b><br><b>Description:<\/b> O'Brien Lake is located on a broad outwash plain. Surrounding vegetation: A forest community dominated by jack pine (Pinus banksiana) and red pine (P. resinosa). This forest is similar to the pre-European settlement vegetation. A wetland forest dominated by Tsuga occidentalis lines the outlet stream.<br><a href=http://apps.neotomadb.org/explorer/?siteids=28101>Explorer Link<\/a>","<b>Horseshoe Lake<\/b><br><b>Description:<\/b> Horseshoe Lake lies within a valley on the Port Huron Moraine. Surrounding vegetation: Second-growth mixed pine-oak forest community. Pinus resinosa, P. strobus, Quercus ellipsoidalis, and Populus. Pre-European settlement vegetation included abundant Tsuga canadensis and Fagus grandifolia.<br><a href=http://apps.neotomadb.org/explorer/?siteids=28113>Explorer Link<\/a>"],null,null,{"interactive":false,"permanent":false,"direction":"auto","opacity":1,"offset":[0,0],"textsize":"10px","textOnly":false,"className":"","sticky":true},null]},{"method":"addPolygons","args":[[[[{"lng":[-86.95,-82.43,-82.43,-86.95,-86.95],"lat":[41.55,41.55,45.88,45.88,41.55]}]]],null,null,{"interactive":true,"className":"","stroke":true,"color":"green","weight":5,"opacity":0.5,"fill":true,"fillColor":"green","fillOpacity":0.2,"smoothFactor":1,"noClip":false},null,null,null,{"interactive":false,"permanent":false,"direction":"auto","opacity":1,"offset":[0,0],"textsize":"10px","textOnly":false,"className":"","sticky":true},null]}],"limits":{"lat":[41.55,45.88],"lng":[-86.95,-82.43]}},"evals":[],"jsHooks":[]}</script>
</div>
</div>
</section>
</section>
</section>
<section id="section-1" class="level3" data-number="5.4.2">
<h3 data-number="5.4.2" class="anchored" data-anchor-id="section-1"><span class="header-section-number">5.4.2</span> </h3>
<p><strong>Exercise Question 3</strong> Which state has more sites in Neotoma, Minnesota or Wisconsin? How many in each state? Provide both code and answer.</p>
<section id="helper-functions-for-site-searches" class="level4 tabset" data-number="5.4.2.1">
<h4 class="tabset anchored" data-number="5.4.2.1" data-anchor-id="helper-functions-for-site-searches"><span class="header-section-number">5.4.2.1</span> Helper Functions for Site Searches</h4>
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="images/neotomaUML_as.svg" class="img-fluid figure-img"></p>
<p></p><figcaption class="figure-caption">Neotoma R Package Universal Modeling Language (UML) diagram.</figcaption><p></p>
</figure>
</div>
<p>If we look at the <a href="https://en.wikipedia.org/wiki/Unified_Modeling_Language">UML diagram</a> for the objects in the <code>neotoma2</code> R package, we can see that there are a set of functions that can operate on <code>sites</code>. As we add to <code>sites</code> objects, using <code>get_datasets()</code> or <code>get_downloads()</code>, we are able to use more of these helper functions. We can use functions like <code>summary()</code> to get a more complete sense of the types of data in this set of sites.</p>
<p>The following code gives the summary table. We do some R magic here to change the way the data is displayed (turning it into a <code>datatable()</code> object using the <code>DT</code> package), but the main function is the <code>summary()</code> call.</p>
<section id="code-2" class="level5" data-number="5.4.2.1.1">
<h5 data-number="5.4.2.1.1" class="anchored" data-anchor-id="code-2"><span class="header-section-number">5.4.2.1.1</span> Code</h5>
<div class="cell" data-hash="04-neotoma2_cache/html/summary_sites_9bee10bc720113debbae1b26ec3daaf6">
<div class="sourceCode cell-code" id="cb6"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb6-1"><a href="#cb6-1" aria-hidden="true" tabindex="-1"></a>neotoma2<span class="sc">::</span><span class="fu">summary</span>(mich_sites)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
</section>
<section id="result-2" class="level5" data-number="5.4.2.1.2">
<h5 data-number="5.4.2.1.2" class="anchored" data-anchor-id="result-2"><span class="header-section-number">5.4.2.1.2</span> Result</h5>
<div class="cell" data-hash="04-neotoma2_cache/html/summarySitesTable_15d6952720d962f3c74c85b3d0ce68d1">
<div class="cell-output-display">

<div class="datatables html-widget html-fill-item-overflow-hidden html-fill-item" id="htmlwidget-fcceab85ec28682848d0" style="width:100%;height:auto;"></div>
<script type="application/json" data-for="htmlwidget-fcceab85ec28682848d0">{"x":{"filter":"none","vertical":false,"data":[[294,294,360,360,479,479,518,518,661,661,785,785,821,821,821,952,952,1100,1100,1360,1540,1540,1541,1541,1541,2207,2655,2656,2657,2658,2658,2658,2659,2660,2661,2661,2661,2662,2663,2664,2665,2666,2667,2668,2669,2670,2671,2672,2673,2674,2675,2676,2677,2678,2679,2680,2681,2682,2683,2684,2685,2686,2687,2688,2689,2690,2691,2692,2693,2694,2695,2696,2697,2698,2699,2700,2701,2702,2703,2704,2705,2706,2707,2708,2709,2710,2711,2712,2713,2714,2715,2716,2783,2814,2814,2864,2864,2954,2954,3571,3571,3641,3641,3642,3642,3650,4235,4346,4421,4526,4527,4528,4529,4530,5330,5330,5363,5363,5364,5365,5365,5366,5367,5368,5368,5369,5370,5370,5375,5375,5531,5532,5533,5533,5534,5535,5535,5536,5537,5537,5538,5539,5539,5540,5541,5542,5543,5544,5544,5545,5546,5547,5548,5549,5550,5551,5552,5552,5554,5554,5555,5555,5556,5557,5558,5559,5560,5561,5562,5563,5564,5564,5565,5566,5567,5567,5568,5569,5570,5571,5572,5573,5574,5575,5575,5576,5577,5577,5578,5578,5579,5580,5581,5582,5582,5583,5583,5584,5585,5586,5587,5588,5589,5590,5591,5591,5592,5593,5593,5594,5595,5596,5597,5597,5598,5598,5599,5600,5601,5602,5602,5603,5603,5604,5605,5605,5606,5607,5608,5609,5610,5611,5612,5613,5614,5615,5616,5617,5617,5618,5619,5620,5620,5621,5622,5623,5624,5624,5625,5706,5706,5706,5707,5707,5711,5712,5713,5721,5762,6299,6370,8559,8559,8559,8559,8559,8580,8580,8582,8582,9734,9734,9734,9734,9777,9777,9787,9787,9896,9896,9905,9905,9946,9946,10003,10003,10003,10067,10067,10104,10104,10125,10125,10129,10129,10166,10239,10239,10242,10242,10251,10251,10264,10264,10423,10423,13350,13350,13457,13457,13466,13466,13467,13467,13577,13577,13578,13578,13579,13579,13581,13581,13582,13582,13582,13582,13582,13582,13582,13582,13582,13582,13582,13582,13582,13582,13582,13582,13582,13582,13582,13582,13582,13582,13582,13582,13582,13582,13582,13582,13582,13582,13582,13582,13582,13582,13582,13582,13582,13582,13582,13582,13582,13582,13582,13582,13582,13582,13584,13584,13585,13585,13586,13586,13587,13587,13588,13588,13592,13592,13593,13593,13607,13607,13623,13623,16019,16313,16313,16740,16740,16891,16891,16891,16891,16891,16891,16891,16891,16891,16891,16921,16921,17281,17281,17326,17326,17331,17331,17338,17338,17390,17390,17390,17390,17393,17393,17498,18113,18113,18123,18123,18126,18126,21274,21274,21757,21757,22262,22262,22273,22273,22274,22274,22399,22399,22593,22593,22599,22599,22743,22743,22864,22864,22953,22953,23172,23172,23288,23288,23288,23297,23297,23297,23300,23300,23300,23312,23312,23312,23323,23323,23323,23324,23325,23325,23325,23333,23333,23333,23333,23333,23346,23346,23346,23352,23352,23352,23360,23360,23360,23371,23371,23371,23371,23371,24000,24000,24000,24000,24000,24001,24001,24001,24001,24001,24002,24002,24002,24002,24003,24003,24003,24003,24003,24004,24004,24004,24004,24005,24005,24006,24006,24007,24007,24007,24007,24007,24008,24008,24405,24405,24405,24418,24418,24418,24429,24429,24502,24502,24502,24506,24506,24506,24559,24559,24559,24587,24587,24587,24598,24598,24598,24601,24601,24601,24612,24612,24612,24629,24629,24629,24636,24636,24636,24641,24641,24641,24645,24645,24645,24646,24646,24646,24671,24671,24671,24674,24674,24674,24679,24679,24679,24699,24699,24699,24711,24711,24711,24715,24715,24715,24720,24720,24720,24725,24725,24725,24727,24727,24727,24730,24730,24730,24754,24754,24754,24765,24765,24765,24768,24768,24768,24779,24779,24779,24794,24798,24802,28059,28059,28100,28100,28101,28101,28113,28113],["Bondi Section","Bondi Section","Chippewa Bog","Chippewa Bog","Clear Lake","Clear Lake","Cub Lake","Cub Lake","Demont Lake","Demont Lake","Lake Erie","Lake Erie","Frains Lake","Frains Lake","Frains Lake","Green Lake","Green Lake","Hudson Lake","Hudson Lake","Jones Lake","Lake Sixteen","Lake Sixteen","Lake 27","Lake 27","Lake 27","Crystal Lake","Lancaster Lake","Carpenter Lake","Weber Lake","Lost Lake","Lost Lake","Lost Lake","Esau Lake","One Lake","Gaylanta Lake","Gaylanta Lake","Gaylanta Lake","Jackson Lake","Beaver Lake","South Twin Lake","East Lake","Shupac Lake","South Perry Lake","Brownlee Lake","Healy Lake","Crystal Lake","Round Lake","Higgins Lake","Rifle Lake","Hackert Lake [Crystal Lake]","North Lake","Wells Lake","Budd Lake","Clear Lake","Bently Lake","Rush Lake","Woodland Lake","Clear Lake","Littlefield Lake","Salsuesha Lake","Topico Marsh","Half Moon Lake","Watcheer Pond","Duck Lake","Crockery Lake","Lincoln Lake","Holland Lake","Otter Lake","Shay Lake","Long Lake","Campau Lake","Mullenkin Lake","Round Lake","Bryam Lake","Hutchin's Lake","Guernsey Lake","Narrow Lake","Taylor Lake","Blind Lake","Pickerel Lake","Whitmore Lake","Murray Lake","Sodon Lake","Rush Lake","Crooked Lake","Pleasant Lake","Lyon Lake","Lee Lake","Hayck Lake","Vandercook Lake","Round Lake","Douglas Lake","McCracken's Pond","Vestaburg Bog","Vestaburg Bog","Wintergreen Lake","Wintergreen Lake","Pretty Lake","Pretty Lake","Schultz","Schultz","Juntunen [20MK1]","Juntunen [20MK1]","Ft. Michilimackinac","Ft. Michilimackinac","Younge","Ogontz Bay [USFS328]","Walled Lake Ursus","Greenville Ursus","Fosters [20SA74]","Schmidt [20SA192]","Kantzler [20BY30]","Lalone [20ARL]","Stadelmeyer [20SA195]","Sleeping Bear Dune","Sleeping Bear Dune","Coville Farm","Coville Farm","Wolf","Bridgeport [20SA620]","Bridgeport [20SA620]","Riviere Au Vase [Graham/Greene School/Mud Creek] [20MB3]","Moccasin Bluff [20BE-B8]","Spring Creek [20MU3]","Spring Creek [20MU3]","Butterfield","Burnt Bluff Cave [B-95]","Burnt Bluff Cave [B-95]","Feeheley [20SA128]","Feeheley [20SA128]","Mahoney [20SA193]","Younge [20SA209]","Naugle [20MD30]","Naugle [20MD30]","Berrien Springs Jail House","Morin [20MO40]","Morin [20MO40]","Sposito [SIS-29]","Norge Village","Norge Village","Beyer","Spoonville [20OT1]","Spoonville [20OT1]","Fisher Lake [20LU21]","20WA176","Birch Run Road [20SA393]","Mission of St. Ignace","Trombly House [20BY70]","Trombly House [20BY70]","Upper Buff Creek [20AA128]","Point Arcadia [20MT120]","Caseville Airport [20HU164]","Gros Cap [MK6/7/111]","Veitch [20SC77]","20LP98","Jahr [20TU6]","DeBoer [20AE62]","DeBoer [20AE62]","46th Street Site","46th Street Site","Bailer Site","Bailer Site","Midland Site","Shaffer","Ruskovic Site","Cadmus Road","Four Lakes Drive Site","Rothbury Mastodon Site","Troy Mastodon Site","New Hudson Site","Groleau-White Lake Site","Groleau-White Lake Site","Jolman Site","Sakstrup Site","Pleasant Lake Site","Pleasant Lake Site","Kuhl Site","Killin Gravel Pit","Shine Site","Zeller Site","Bryce Site","Maurer Site","Evart Site","Mead Site","Mead Site","Adams Site","Ann Arbor Platygonus Site","Ann Arbor Platygonus Site","Prillwitz Site","Prillwitz Site","Flanders Site","Ann Arbor Alces Site","Watervliet Site","Lennon [Flint] Site","Lennon [Flint] Site","Sweeney Site","Sweeney Site","Henry Site","Betz Site","Seneca Township Site","American Aggregates Gravel Pit","Carmichael Site","Zamarron Site","Hartford Proboscidean","Powers Site","Powers Site","Green Site","Harper","Harper","Croton Muskox","Moreland Muskox","White Pigeon Muskox","Scotts Muskox Spec. 2146 [KMP-2146]","Scotts Muskox Spec. 2146 [KMP-2146]","Scotts Muskox Spec. 34124 [UMMP-34124]","Scotts Muskox Spec. 34124 [UMMP-34124]","Hillsdale Muskox","Manchester Muskox","Berrien Muskox","Heisler Site","Heisler Site","Shelton Site [20OK394]","Shelton Site [20OK394]","Ludington Castoroides","Van Horn Mammoth","Van Horn Mammoth","Turk Lake","Paw Paw Muskox","Dowagiac River Site","I-96 Site","Taylor Site","Ray Road Site","Johnson Site","Cole Site","Parker Site","Bronson","Berndt Site","Rappuhn Site","Rappuhn Site","Vanmiddlesworth Site","Crandell Site","Sheathelm Site","Sheathelm Site","Gullich Road Site","Van Sickle Site","Winn Mastodon","Thaller Site","Thaller Site","Smith Site","Grandville","Grandville","Grandville","Schwerdt [20AE127]","Schwerdt [20AE127]","Fowlerville Caribou","Minden City Caribou","Glacial Lake Mogodore Mammuthus","Holcombe Beach [20MB30]","Pitt Farm","20SA1034","Rock Island","Irwin Smith Bog","Irwin Smith Bog","Irwin Smith Bog","Irwin Smith Bog","Irwin Smith Bog","Barney's Lake","Barney's Lake","Bass Lake","Bass Lake","Bassett Lake","Bassett Lake","Bassett Lake","Bassett Lake","Black Lake","Black Lake","Berry Lake [Frog Lake]","Berry Lake [Frog Lake]","Cary Lake","Cary Lake","Clear Lake","Clear Lake","Au Sable River","Au Sable River","Appleman Lake","Appleman Lake","Appleman Lake","Leamington Site","Leamington Site","Ellis Lake","Ellis Lake","Glacial Lake Arkona-Bondi Site","Glacial Lake Arkona-Bondi Site","Pinhook Bog","Pinhook Bog","Route 458 Bog","Cowden Lake","Cowden Lake","Hicks Lake","Hicks Lake","Morrison Lake","Morrison Lake","Fish Lake","Fish Lake","Halfmoon Lake","Halfmoon Lake","Spicer Lake","Spicer Lake","Upper Herring Lake","Upper Herring Lake","Lake Fifteen","Lake Fifteen","Lake Ferdelman","Lake Ferdelman","Silver Lake","Silver Lake","Indian Lake","Indian Lake","Lake Manuka","Lake Manuka","Lily Lake","Lily Lake","Minden Bog","Minden Bog","Minden Bog","Minden Bog","Minden Bog","Minden Bog","Minden Bog","Minden Bog","Minden Bog","Minden Bog","Minden Bog","Minden Bog","Minden Bog","Minden Bog","Minden Bog","Minden Bog","Minden Bog","Minden Bog","Minden Bog","Minden Bog","Minden Bog","Minden Bog","Minden Bog","Minden Bog","Minden Bog","Minden Bog","Minden Bog","Minden Bog","Minden Bog","Minden Bog","Minden Bog","Minden Bog","Minden Bog","Minden Bog","Minden Bog","Minden Bog","Minden Bog","Minden Bog","Minden Bog","Minden Bog","Minden Bog","Minden Bog","Minden Bog","Minden Bog","Minden Bog","Minden Bog","North Lake","North Lake","Reed Lake","Reed Lake","Threemile Lake","Threemile Lake","Shafer Lake","Shafer Lake","School Section Lake","School Section Lake","Lime Lake","Lime Lake","Loon Lake","Loon Lake","Lee Lake","Lee Lake","Mud Lake","Mud Lake","The Winameg Mastodon Site","Otter Lake","Otter Lake","Bondi [AaHp-VP]","Bondi [AaHp-VP]","Herron Peatland","Herron Peatland","Herron Peatland","Herron Peatland","Herron Peatland","Herron Peatland","Herron Peatland","Herron Peatland","Herron Peatland","Herron Peatland","Verbeke Mastodon [AaHp-VP]","Verbeke Mastodon [AaHp-VP]","Lake St. Claire","Lake St. Claire","Lake St. Claire/Pike Creek","Lake St. Claire/Pike Creek","Cedar River","Cedar River","LDDsite5519","LDDsite5519","Lake St. Clair","Lake St. Clair","Lake St. Clair","Lake St. Clair","LDDsite5897","LDDsite5897","Belding Peccary","LDDsite5885","LDDsite5885","LDDsite5894","LDDsite5894","Belle River","Belle River","Bruner-Colasanti [AaHq-8]","Bruner-Colasanti [AaHq-8]","Mill Creek Site","Mill Creek Site","Lake Erie","Lake Erie","Lake Erie","Lake Erie","Lake Erie","Lake Erie","Robson Road [AaHp-20]","Robson Road [AaHp-20]","Hershberger","Hershberger","Moorland","Moorland","Belleville Lake","Belleville Lake","Ford Lake","Ford Lake","Lake Besser","Lake Besser","Stony Creek Lake","Stony Creek Lake","Blackman Lake","Blackman Lake","Blackman Lake","Bogie Lake","Bogie Lake","Bogie Lake","Bridge Lake","Bridge Lake","Bridge Lake","Campbell Lake","Campbell Lake","Campbell Lake","Cheeseboro Lake","Cheeseboro Lake","Cheeseboro Lake","Clark Co. Lake","Clark Lake","Clark Lake","Clark Lake","Crooked Lake","Crooked Lake","Crooked Lake","Crooked Lake","Crooked Lake","Deer Lake","Deer Lake","Deer Lake","Donnell Lake","Donnell Lake","Donnell Lake","Eight Point Lake","Eight Point Lake","Eight Point Lake","Fish Lake","Fish Lake","Fish Lake","Fish Lake","Fish Lake","Bass Lake","Bass Lake","Bass Lake","Bass Lake","Bass Lake","Florence","Florence","Florence","Florence","Florence","Loon","Loon","Loon","Loon","Manitou","Manitou","Manitou","Manitou","Manitou","North Bar","North Bar","North Bar","North Bar","Otter","Otter","Round","Round","Shell","Shell","Shell","Shell","Shell","Tucker","Tucker","Hi-Land Lake","Hi-Land Lake","Hi-Land Lake","Johnson Lake","Johnson Lake","Johnson Lake","Lake Au Sable","Lake Au Sable","Lake Brighton","Lake Brighton","Lake Brighton","Lake Chemung","Lake Chemung","Lake Chemung","Little Otter","Little Otter","Little Otter","Martin Lake","Martin Lake","Martin Lake","Messick Lake","Messick Lake","Messick Lake","Mill Lake","Mill Lake","Mill Lake","Muskegon Lake","Muskegon Lake","Muskegon Lake","Olin Lake","Olin Lake","Olin Lake","Palmer Lake","Palmer Lake","Palmer Lake","Pere Marquette Lake","Pere Marquette Lake","Pere Marquette Lake","Pine Lake","Pine Lake","Pine Lake","Pine Lake (Ke)","Pine Lake (Ke)","Pine Lake (Ke)","Round Lake (Va)","Round Lake (Va)","Round Lake (Va)","Round Lake (St)","Round Lake (St)","Round Lake (St)","Saddle Lake","Saddle Lake","Saddle Lake","South Chain Lake","South Chain Lake","South Chain Lake","Squaw Lake","Squaw Lake","Squaw Lake","Stoner Lake","Stoner Lake","Stoner Lake","Sullivan Lake","Sullivan Lake","Sullivan Lake","Tallman Lake","Tallman Lake","Tallman Lake","Thornapple Lake","Thornapple Lake","Thornapple Lake","Tims Lake","Tims Lake","Tims Lake","Upper Scott","Upper Scott","Upper Scott","Warner Lake","Warner Lake","Warner Lake","West Lake","West Lake","West Lake","Wyckoff Lake","Wyckoff Lake","Wyckoff Lake","Blue Lake (Me)","Crooked Lake (St)","Fox Lake (St)","Huffman Lake","Huffman Lake","Clifford Lake","Clifford Lake","O'Brien Lake","O'Brien Lake","Horseshoe Lake","Horseshoe Lake"],["BONDI","BONDI","CHIPPEWA","CHIPPEWA","CLEARLK","CLEARLK","CUB2","CUB2","DEMONTLK","DEMONTLK","ERIE","ERIE","FRAINSLK","FRAINSLK","TW126","GREENLK","GREENLK","HUDSONLK","HUDSONLK","TW085","LAKE16","LAKE16","LAKE27","LAKE27","TW078","ROK013","TW073","TW074","TW075","NANODE375","NANODE375","TW076","TW077","TW079","NANODE372","NANODE372","TW080","TW081","TW082","TW083","TW084","TW086","TW087","TW088","TW089","TW090","TW091","TW092","TW093","TW094","TW095","TW096","TW097","TW098","TW099","TW100","TW101","TW102","TW103","TW104","TW105","TW106","TW107","TW108","TW109","TW110","TW111","TW112","TW113","TW114","TW115","TW116","TW117","TW118","TW119","TW120","TW121","TW122","TW123","TW124","TW125","TW127","TW128","TW129","TW130","TW131","TW132","TW133","TW134","TW135","TW136","TW138","TWH001","VESTABRG","VESTA-II","WINTERGR","WINTERGR","PRETTYLK","PRETTYLK","SCHULTZ","SCHULTZ","JUNTUNEN","JUNTUNEN","FTMICHIL","FTMICHIL","YOUNGE1","OGONTZ","WLURSUS","GRNURSUS","FOSTERS","SCHMDTMI","KANTZLER","LALONE","STDLMEYR","SLPBERDN","SLPBERDN","COVILLE","COVILLE","WOLF","BRDGPORT","BRDGPORT","RVAUVASE","MOCCBLUF","SPRCRKMI","SPRCRKMI","BUTTRFLD","BURNTBLF","BURNTBLF","FEEHELEY","FEEHELEY","MAHONEY","YOUNGE2","NAUGLE","NAUGLE","BERRSPJH","MORIN","MORIN","SPOSITO","NORGEVLG","NORGEVLG","BEYER","SPOONVIL","SPOONVIL","FISHERLK","MIWA176","BIRCHRUN","STIGNACE","TROMBBT","TROMBTU","UPBLUFCR","PTARCF2","CASVILAP","GROSCAP","VEITCH","MILP98","JAHR","DEBOER","DEBOER","FORTYSIX","FORTYSIX","BAILER","BAILER","MIDLNDMS","SHAFERMI","RUSHOVIC","CADMUSRD","FOURLKDR","ROTHBURY","TROYMAST","HUDSONMS","GROLWHIT","GROLWHIT","JOLMAN","SAKSTRUP","PLESMIWA","PLESMIWA","KUHL","KILLINGP","SHINE","ZELLER","BRYCE","MAURER","EVART","MEAD","MEAD","ADAMS","ANNARBPL","ANNARBPL","PRILWITZ","PRILWITZ","FLANDERS","ANNARBAL","WTRLITMM","LENNON","LENNON","SWEENEY","SWEENEY","HENRY","BETZ","SENECAMM","AMAGGRGP","CARMICHL","ZAMARRON","HRTFRDPR","POWERS","POWERS","GREEN","HARPER","HARPER","CROTONMX","MORLNDMX","WHTPGNMX","KMP2146","KMP2146","UMMP34124","UMMP34124","HLSDALMX","MANCHMX","BERIENMX","HEISLRMS","HEISLRMS","SHELTNMS","SHELTNMS","LUDINGCS","VANHRNMM","VANHRNMM","TURKLAKE","PAWPAWMX","DOWAGIAC","I-96","TAYLORMI","RAYROAD","JOHNSON","COLE","PARKER","BRONSON","BERNDT","RAPPUHN","RAPPUHN","VANMIDDL","CRANDALL","SHEATHLM","SHEATHLM","GULICHRD","VANSICKL","WINNMAST","THALLER","THALLER","SMITH","GRANDVMS","GRANDVMS","GRANDVMS","SCHWERDT","SCHWERDT","FOWLERCB","MINCTYCB","GLMOGOMM","HOLCOMBE","PITTFARM","MISA1034","ROCKISL","IRWINSMT","IRWINSMT","IRWINSMT","IRWINSMT","IRWINSMT","NANODE369","NANODE369","NANODE388","NANODE388","NANODE394","NANODE394","NANODE395","NANODE395","NANODE390","NANODE390","NANODE380","NANODE380","NANODE440","NANODE440","NANODE393","NANODE393","AUSABLERIV","AUSABLERIV","APPLEMAN","APPLEMAN","APPLEMAN","LEAMINGTON","LEAMINGTON","NANODE377","NANODE377","BONDIARK","BONDIARK","PINHOOK","PINHOOK","ROUTE 458","COWDEN","COWDEN","HICKSMI","HICKSMI","MORRIMI","MORRIMI","NANODE396","NANODE396","NANODE387","NANODE387","SPIC06A1","SPIC06A1","NANODE370","NANODE370","NANODE373","NANODE373","NANODE374","NANODE374","NANODE376","NANODE376","NANODE378","NANODE378","NANODE379","NANODE379","NANODE382","NANODE382","MI01","MI02","MI03","MI04","MI05","MI06","MI07","MI08","MI09","MI10","MI11","MI12","MI13","MI14","MI15","MI16","MI17","MI18","MI19","MI20","MI21","MI22","MI23","MI25","MI26","MI27","MI28","MI29","MINDEN","MINDEN","MINDEN","MINDENA","MINDENB","MINDENC","MINDEND","MINDENE","MINDENF","MINDENG","MINDENH","MINDENI","MINDENJ","MINDENK","MINDENL","MINDENM","MINDENN","MINDENO","NANODE383","NANODE383","NANODE385","NANODE385","NANODE389","NANODE389","NANODE391","NANODE391","NANODE392","NANODE392","NANODE401","NANODE401","NANODE402","NANODE402","NANODE439","NANODE439","NANODE486","NANODE486","FM4960","OTTERMI","OTTERMI","FM3621","FM3621","AL01","AL02","AL03","AL04","AL05","AL06","AL07","AL08","AL09","AL10","FM3620","FM3620","LDD5515","LDD5515","LDD5516","LDD5516","LDD5517","LDD5517","LDD5519","LDD5519","LDD5533","LDD5533","LDD5536","LDD5536","LDD5897","LDD5897","FM4935","LDD5885","LDD5885","LDD5894","LDD5894","LDD5896","LDD5896","FM3622","FM3622","FM2974","FM2974","LDD5438","LDD5438","LDD5443","LDD5443","LDD5444","LDD5444","FM3619","FM3619","HERSHBMA","HERSHBMA","MOORL","MOORL","NLA0463","NLA0463WC","NLA1487","NLA1487WC","NLA0330","NLA0330WC","NLA1802","NLA1802WC","NLA3035","NLA3035","NLA3035WC","NLA0010","NLA0010","NLA0010WC","NLA1690","NLA1690","NLA1690WC","NLA0619","NLA0619","NLA0619WC","NLA0107","NLA0107","NLA0107WC","NLA1706","NLA1167","NLA1167","NLA1167WC","NLA1131","NLA1131","NLA1334","NLA1334","NLA1334WC","NLA1434","NLA1434","NLA1434WC","NLA1707","NLA1707","NLA1707WC","NLA0426","NLA0426","NLA0426WC","NLA0935","NLA0935","NLA0987","NLA0987","NLA0987WC","BASS2005","BASS_SS05","BASS_SS09","BASS_SS14","SLBASS-WC","FLOR2008","FLOR_SS05","FLOR_SS09","FLOR_SS14","SLFLOR-WC","LOON_SS05","LOON_SS09","LOON_SS14","SLLOON-WC","MANT2005","MANT_SS05","MANT_SS09","MANT_SS14","SLMANT-WC","NBAR_SS05","NBAR_SS09","NBAR_SS14","SLNBAR-WC","OTTR_SS08","SLOTTR-WC","ROND_SS05","SLROND-WC","SHEL2005","SHLL_SS05","SHLL_SS09","SHLL_SS14","SLSHLL-WC","SLTUCK-WC","TUCK_SS05","NLA0399","NLA0399","NLA0399WC","NLA0811","NLA0811","NLA0811WC","NLA1162","NLA1162WC","NLA1782D","NLA1782D","NLA1782WC","NLA1014D","NLA1014D","NLA1014WC","NLA555D","NLA555D","NLA555WC","NLA874D","NLA874D","NLA874WC","NLA0043D","NLA0043D","NLA0043WC","NLA1034D","NLA1034D","NLA1034WC","NLA0938D","NLA0938D","NLA0938WC","NLA646D","NLA646D","NLA646WC","NLA0363D","NLA0363D","NLA0363WC","NLA1626D","NLA1626D","NLA1626WC","NLA0875D","NLA0875D","NLA0875WC","NLA0234D","NLA0234D","NLA0234WC","NLA1003D","NLA1003D","NLA1003WC","NLA1835D","NLA1835D","NLA1835WC","NLA1771D","NLA1771D","NLA1771WC","NLA2987D","NLA2987D","NLA2987WC","NLA0502D","NLA0502D","NLA0502WC","NLA1258D","NLA1258D","NLA1258WC","NLA7007D","NLA7007D","NLA7007WC","NLA0170D","NLA0170D","NLA0170WC","NLA1130D","NLA1130D","NLA1130WC","NLA1679D","NLA1679D","NLA1679WC","NLA1515D","NLA1515D","NLA1515WC","NLA1643D","NLA1643D","NLA1643WC","NLA1387D","NLA1387D","NLA1387WC","NLA1370D","NLA1370D","NLA1370WC","NLA1450WC","NLA1131WC","NLA55WC","HUFFMI","HUFFMI","CLIFFMI","CLIFFMI","OBRIENMI","OBRIENMI","HORSEMI","HORSEMI"],[0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0],[1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1],["pollen","geochronologic","pollen","geochronologic","pollen","geochronologic","pollen","geochronologic","pollen","geochronologic","pollen","geochronologic","pollen","geochronologic","pollen surface sample","pollen","geochronologic","pollen","geochronologic","pollen surface sample","pollen","geochronologic","pollen","geochronologic","pollen surface sample","pollen surface sample","pollen surface sample","pollen surface sample","pollen surface sample","water chemistry","ostracode surface sample","pollen surface sample","pollen surface sample","pollen surface sample","water chemistry","ostracode surface sample","pollen surface sample","pollen surface sample","pollen surface sample","pollen surface sample","pollen surface sample","pollen surface sample","pollen surface sample","pollen surface sample","pollen surface sample","pollen surface sample","pollen surface sample","pollen surface sample","pollen surface sample","pollen surface sample","pollen surface sample","pollen surface sample","pollen surface sample","pollen surface sample","pollen surface sample","pollen surface sample","pollen surface sample","pollen surface sample","pollen surface sample","pollen surface sample","pollen surface sample","pollen surface sample","pollen surface sample","pollen surface sample","pollen surface sample","pollen surface sample","pollen surface sample","pollen surface sample","pollen surface sample","pollen surface sample","pollen surface sample","pollen surface sample","pollen surface sample","pollen surface sample","pollen surface sample","pollen surface sample","pollen surface sample","pollen surface sample","pollen surface sample","pollen surface sample","pollen surface sample","pollen surface sample","pollen surface sample","pollen surface sample","pollen surface sample","pollen surface sample","pollen surface sample","pollen surface sample","pollen surface sample","pollen surface sample","pollen surface sample","pollen surface sample","pollen surface sample","pollen","geochronologic","pollen","geochronologic","pollen","geochronologic","vertebrate fauna","geochronologic","vertebrate fauna","geochronologic","vertebrate fauna","geochronologic","vertebrate fauna","vertebrate fauna","vertebrate fauna","vertebrate fauna","vertebrate fauna","vertebrate fauna","vertebrate fauna","vertebrate fauna","vertebrate fauna","vertebrate fauna","geochronologic","vertebrate fauna","geochronologic","vertebrate fauna","vertebrate fauna","geochronologic","vertebrate fauna","vertebrate fauna","vertebrate fauna","geochronologic","vertebrate fauna","vertebrate fauna","geochronologic","vertebrate fauna","geochronologic","vertebrate fauna","vertebrate fauna","vertebrate fauna","geochronologic","vertebrate fauna","vertebrate fauna","geochronologic","vertebrate fauna","vertebrate fauna","geochronologic","vertebrate fauna","vertebrate fauna","geochronologic","vertebrate fauna","vertebrate fauna","vertebrate fauna","vertebrate fauna","vertebrate fauna","vertebrate fauna","vertebrate fauna","vertebrate fauna","vertebrate fauna","vertebrate fauna","vertebrate fauna","vertebrate fauna","vertebrate fauna","vertebrate fauna","geochronologic","vertebrate fauna","geochronologic","vertebrate fauna","geochronologic","vertebrate fauna","vertebrate fauna","vertebrate fauna","vertebrate fauna","vertebrate fauna","vertebrate fauna","vertebrate fauna","vertebrate fauna","vertebrate fauna","geochronologic","vertebrate fauna","vertebrate fauna","vertebrate fauna","geochronologic","vertebrate fauna","vertebrate fauna","vertebrate fauna","vertebrate fauna","vertebrate fauna","vertebrate fauna","vertebrate fauna","vertebrate fauna","geochronologic","vertebrate fauna","vertebrate fauna","geochronologic","vertebrate fauna","geochronologic","vertebrate fauna","vertebrate fauna","vertebrate fauna","vertebrate fauna","geochronologic","vertebrate fauna","geochronologic","vertebrate fauna","vertebrate fauna","vertebrate fauna","vertebrate fauna","vertebrate fauna","vertebrate fauna","vertebrate fauna","vertebrate fauna","geochronologic","vertebrate fauna","vertebrate fauna","geochronologic","vertebrate fauna","vertebrate fauna","vertebrate fauna","vertebrate fauna","geochronologic","vertebrate fauna","geochronologic","vertebrate fauna","vertebrate fauna","vertebrate fauna","vertebrate fauna","geochronologic","vertebrate fauna","geochronologic","vertebrate fauna","vertebrate fauna","geochronologic","vertebrate fauna","vertebrate fauna","vertebrate fauna","vertebrate fauna","vertebrate fauna","vertebrate fauna","vertebrate fauna","vertebrate fauna","vertebrate fauna","vertebrate fauna","vertebrate fauna","vertebrate fauna","geochronologic","vertebrate fauna","vertebrate fauna","vertebrate fauna","geochronologic","vertebrate fauna","vertebrate fauna","vertebrate fauna","vertebrate fauna","geochronologic","vertebrate fauna","vertebrate fauna","geochronologic","vertebrate fauna","vertebrate fauna","geochronologic","vertebrate fauna","vertebrate fauna","vertebrate fauna","vertebrate fauna","vertebrate fauna","vertebrate fauna","vertebrate fauna","geochronologic","plant macrofossil","pollen","stable isotope","biomarker","water chemistry","ostracode surface sample","water chemistry","ostracode surface sample","water chemistry","ostracode surface sample","water chemistry","ostracode surface sample","water chemistry","ostracode surface sample","water chemistry","ostracode surface sample","water chemistry","ostracode surface sample","water chemistry","ostracode surface sample","geochronologic","insect","geochronologic","pollen","charcoal","geochronologic","insect","water chemistry","ostracode surface sample","geochronologic","vertebrate fauna","geochronologic","pollen","pollen","geochronologic","pollen","geochronologic","pollen","geochronologic","pollen","water chemistry","ostracode surface sample","water chemistry","ostracode surface sample","geochronologic","pollen","water chemistry","ostracode surface sample","water chemistry","ostracode surface sample","water chemistry","ostracode surface sample","water chemistry","ostracode surface sample","water chemistry","ostracode surface sample","water chemistry","ostracode surface sample","water chemistry","ostracode surface sample","testate amoebae surface sample","testate amoebae surface sample","testate amoebae surface sample","testate amoebae surface sample","testate amoebae surface sample","testate amoebae surface sample","testate amoebae surface sample","testate amoebae surface sample","testate amoebae surface sample","testate amoebae surface sample","testate amoebae surface sample","testate amoebae surface sample","testate amoebae surface sample","testate amoebae surface sample","testate amoebae surface sample","testate amoebae surface sample","testate amoebae surface sample","testate amoebae surface sample","testate amoebae surface sample","testate amoebae surface sample","testate amoebae surface sample","testate amoebae surface sample","testate amoebae surface sample","testate amoebae surface sample","testate amoebae surface sample","testate amoebae surface sample","testate amoebae surface sample","testate amoebae surface sample","geochronologic","testate amoebae","pollen","testate amoebae surface sample","testate amoebae surface sample","testate amoebae surface sample","testate amoebae surface sample","testate amoebae surface sample","testate amoebae surface sample","testate amoebae surface sample","testate amoebae surface sample","testate amoebae surface sample","testate amoebae surface sample","testate amoebae surface sample","testate amoebae surface sample","testate amoebae surface sample","testate amoebae surface sample","testate amoebae surface sample","water chemistry","ostracode surface sample","water chemistry","ostracode surface sample","water chemistry","ostracode surface sample","water chemistry","ostracode surface sample","water chemistry","ostracode surface sample","water chemistry","ostracode surface sample","water chemistry","ostracode surface sample","water chemistry","ostracode surface sample","water chemistry","ostracode surface sample","vertebrate fauna","geochronologic","pollen","geochronologic","vertebrate fauna","testate amoebae surface sample","testate amoebae surface sample","testate amoebae surface sample","testate amoebae surface sample","testate amoebae surface sample","testate amoebae surface sample","testate amoebae surface sample","testate amoebae surface sample","testate amoebae surface sample","testate amoebae surface sample","geochronologic","vertebrate fauna","water chemistry","ostracode surface sample","water chemistry","ostracode surface sample","water chemistry","ostracode surface sample","water chemistry","ostracode surface sample","water chemistry","ostracode surface sample","water chemistry","ostracode surface sample","water chemistry","ostracode surface sample","vertebrate fauna","water chemistry","ostracode surface sample","water chemistry","ostracode surface sample","water chemistry","ostracode surface sample","geochronologic","vertebrate fauna","geochronologic","vertebrate fauna","water chemistry","ostracode surface sample","water chemistry","ostracode surface sample","water chemistry","ostracode surface sample","geochronologic","vertebrate fauna","geochronologic","vertebrate fauna","geochronologic","vertebrate fauna","diatom surface sample","water chemistry","diatom surface sample","water chemistry","diatom","water chemistry","diatom","water chemistry","diatom","diatom","water chemistry","diatom","diatom","water chemistry","diatom","diatom","water chemistry","diatom","diatom","water chemistry","diatom","diatom","water chemistry","diatom","diatom","diatom","water chemistry","diatom","diatom","diatom","diatom","water chemistry","diatom","diatom","water chemistry","diatom","diatom","water chemistry","diatom","diatom","water chemistry","diatom","diatom","diatom","diatom","water chemistry","diatom","diatom surface sample","diatom surface sample","diatom surface sample","water chemistry","diatom","diatom surface sample","diatom surface sample","diatom surface sample","water chemistry","diatom surface sample","diatom surface sample","diatom surface sample","water chemistry","diatom","diatom surface sample","diatom surface sample","diatom surface sample","water chemistry","diatom surface sample","diatom surface sample","diatom surface sample","water chemistry","diatom surface sample","water chemistry","diatom surface sample","water chemistry","diatom","diatom surface sample","diatom surface sample","diatom surface sample","water chemistry","water chemistry","diatom surface sample","diatom","diatom","water chemistry","diatom","diatom","water chemistry","diatom","water chemistry","diatom surface sample","diatom bottom sample","water chemistry","diatom surface sample","diatom bottom sample","water chemistry","diatom surface sample","diatom bottom sample","water chemistry","diatom surface sample","diatom bottom sample","water chemistry","diatom surface sample","diatom bottom sample","water chemistry","diatom surface sample","diatom bottom sample","water chemistry","diatom surface sample","diatom bottom sample","water chemistry","diatom surface sample","diatom bottom sample","water chemistry","diatom surface sample","diatom bottom sample","water chemistry","diatom surface sample","diatom bottom sample","water chemistry","diatom surface sample","diatom bottom sample","water chemistry","diatom surface sample","diatom bottom sample","water chemistry","diatom surface sample","diatom bottom sample","water chemistry","diatom surface sample","diatom bottom sample","water chemistry","diatom surface sample","diatom bottom sample","water chemistry","diatom surface sample","diatom bottom sample","water chemistry","diatom surface sample","diatom bottom sample","water chemistry","diatom surface sample","diatom bottom sample","water chemistry","diatom surface sample","diatom bottom sample","water chemistry","diatom surface sample","diatom bottom sample","water chemistry","diatom surface sample","diatom bottom sample","water chemistry","diatom surface sample","diatom bottom sample","water chemistry","diatom surface sample","diatom bottom sample","water chemistry","diatom surface sample","diatom bottom sample","water chemistry","diatom surface sample","diatom bottom sample","water chemistry","diatom surface sample","diatom bottom sample","water chemistry","water chemistry","water chemistry","water chemistry","geochronologic","pollen","geochronologic","pollen","geochronologic","pollen","geochronologic","pollen"]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th>siteid<\/th>\n      <th>sitename<\/th>\n      <th>collectionunit<\/th>\n      <th>chronolgies<\/th>\n      <th>datasets<\/th>\n      <th>types<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"scrollX":"100%","dom":"t","columnDefs":[{"className":"dt-right","targets":[0,3,4]}],"order":[],"autoWidth":false,"orderClasses":false},"selection":{"mode":"multiple","selected":null,"target":"row","selectable":null}},"evals":[],"jsHooks":[]}</script>
</div>
</div>
</section>
</section>
</section>
<section id="section-2" class="level3" data-number="5.4.3">
<h3 data-number="5.4.3" class="anchored" data-anchor-id="section-2"><span class="header-section-number">5.4.3</span> </h3>
<p>We can see that there are no chronologies associated with the <code>site</code> objects. This is because, at present, we have not pulled in the <code>dataset</code> information we need. All we know from <code>get_sites()</code> are the kinds of datasets we have.</p>
</section>
<section id="searching-for-datasets" class="level3 tabset" data-number="5.4.4">
<h3 class="tabset anchored" data-number="5.4.4" data-anchor-id="searching-for-datasets"><span class="header-section-number">5.4.4</span> Searching for Datasets:</h3>
<p>Now that we know how to search for sites, we can start to search for datasets. As we’ve discussed before, in the Neotoma data model, each site can contain one or more collection units, each of which can contain one or more datasets. Similarly, a <code>sites</code> object contains <code>collectionunits</code> which contain <code>datasets</code>. From the table above, we can see that some of the sites we’ve looked at contain pollen data. However, so far we have only downloaded the <code>sites</code> data object and not any of the actual pollen data, it’s just that (for convenience) the <code>sites</code> API returns some information about datasets, to make it easier to navigate the records.</p>
<p>With a <code>sites</code> object we can directly call <code>get_datasets()</code>, to pull in more metadata about the datasets. At any time we can use <code>datasets()</code> to get more information about any datasets that a <code>sites</code> object may contain. Compare the output of <code>datasets(mich_sites)</code> to the output of a similar call using the following:</p>
<section id="code-3" class="level4" data-number="5.4.4.1">
<h4 data-number="5.4.4.1" class="anchored" data-anchor-id="code-3"><span class="header-section-number">5.4.4.1</span> Code</h4>
<div class="cell" data-hash="04-neotoma2_cache/html/datasetsFromSites_1c3ef2a33177dfd5377f83d6c4fb636b">
<div class="sourceCode cell-code" id="cb7"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb7-1"><a href="#cb7-1" aria-hidden="true" tabindex="-1"></a>mich_datasets <span class="ot">&lt;-</span> neotoma2<span class="sc">::</span><span class="fu">get_datasets</span>(mich_sites, <span class="at">all_data =</span> <span class="cn">TRUE</span>)</span>
<span id="cb7-2"><a href="#cb7-2" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb7-3"><a href="#cb7-3" aria-hidden="true" tabindex="-1"></a><span class="fu">datasets</span>(mich_datasets)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
</section>
<section id="result-3" class="level4" data-number="5.4.4.2">
<h4 data-number="5.4.4.2" class="anchored" data-anchor-id="result-3"><span class="header-section-number">5.4.4.2</span> Result</h4>
<div class="cell" data-hash="04-neotoma2_cache/html/datasetsFromSitesResult_449640ec8cc3de4ffdb5701c356a8aa8">
<div class="cell-output-display">

<div class="datatables html-widget html-fill-item-overflow-hidden html-fill-item" id="htmlwidget-464540f073c0ea0115fe" style="width:100%;height:auto;"></div>
<script type="application/json" data-for="htmlwidget-464540f073c0ea0115fe">{"x":{"filter":"none","vertical":false,"data":[["1","2","3","4","5","6","7","8","9","10","11","12","13","14","15","16","17","18","19","20","21","22","23","24","25","26","27","28","29","30","31","32","33","34","35","36","37","38","39","40","41","42","43","44","45","46","47","48","49","50","51","52","53","54","55","56","57","58","59","60","61","62","63","64","65","66","67","68","69","70","71","72","73","74","75","76","77","78","79","80","81","82","83","84","85","86","87","88","89","90","91","92","93","94","95","96","97","98","99","100","101","102","103","104","105","106","107","108","109","110","111","112","113","114","115","116","117","118","119","120","121","122","123","124","125","126","127","128","129","130","131","132","133","134","135","136","137","138","139","140","141","142","143","144","145","146","147","148","149","150","151","152","153","154","155","156","157","158","159","160","161","162","163","164","165","166","167","168","169","170","171","172","173","174","175","176","177","178","179","180","181","182","183","184","185","186","187","188","189","190","191","192","193","194","195","196","197","198","199","200","201","202","203","204","205","206","207","208","209","210","211","212","213","214","215","216","217","218","219","220","221","222","223","224","225","226","227","228","229","230","231","232","233","234","235","236","237","238","239","240","241","242","243","244","245","246","247","248","249","250","251","252","253","254","255","256","257","258","259","260","261","262","263","264","265","266","267","268","269","270","271","272","273","274","275","276","277","278","279","280","281","282","283","284","285","286","287","288","289","290","291","292","293","294","295","296","297","298","299","300","301","302","303","304","305","306","307","308","309","310","311","312","313","314","315","316","317","318","319","320","321","322","323","324","325","326","327","328","329","330","331","332","333","334","335","336","337","338","339","340","341","342","343","344","345","346","347","348","349","350","351","352","353","354","355","356","357","358","359","360","361","362","363","364","365","366","367","368","369","370","371","372","373","374","375","376","377","378","379","380","381","382","383","384","385","386","387","388","389","390","391","392","393","394","395","396","397","398","399","400","401","402","403","404","405","406","407","408","409","410","411","412","413","414","415","416","417","418","419","420","421","422","423","424","425","426","427","428","429","430","431","432","433","434","435","436","437","438","439","440","441","442","443","444","445","446","447","448","449","450","451","452","453","454","455","456","457","458","459","460","461","462","463","464","465","466","467","468","469","470","471","472","473","474","475","476","477","478","479","480","481","482","483","484","485","486","487","488","489","490","491","492","493","494","495","496","497","498","499","500","501","502","503","504","505","506","507","508","509","510","511","512","513","514","515","516","517","518","519","520","521","522","523","524","525","526","527","528","529","530","531","532","533","534","535","536","537","538","539","540","541","542","543","544","545","546","547","548","549","550","551","552","553","554","555","556","557","558","559","560","561","562","563","564","565","566","567","568","569","570","571","572","573","574","575","576","577","578","579","580","581","582","583","584","585","586","587","588","589","590","591","592","593","594","595","596","597","598","599","600","601","602","603","604","605","606","607","608","609","610","611","612","613","614","615","616","617"],[7943,300,7996,369,490,8006,8035,532,679,8055,8099,805,8158,982,8195,1133,3804,1580,8293,2283,2750,2751,2752,2754,2755,2757,2758,2759,2760,2761,2762,2763,2764,2765,2766,2767,2768,2769,2770,2771,2772,2773,2774,2775,2776,2777,2778,2779,2780,2781,2782,2783,2784,2785,2786,2787,2788,2789,2790,2791,2792,2793,2794,2795,2796,2797,2798,2799,2800,2801,2802,2803,2804,2805,2806,2807,2808,2809,2810,2811,2878,2961,8754,3563,8841,4608,9303,4681,9334,4682,9335,4690,5349,5493,5568,5687,5688,5689,5690,5691,10097,6639,10100,6672,6673,10101,6674,6675,6676,6677,10102,6678,6679,10103,6684,10104,6848,6849,6850,10132,6851,10133,6852,6853,6854,10134,6855,10135,6856,6857,6858,6859,6860,6863,6864,6865,6866,6867,6868,6869,10136,6870,10137,6872,6873,10138,6874,6875,6876,6877,6878,6879,6880,6881,6882,10139,6883,6884,10140,6885,6886,6887,6888,6889,6890,6891,6892,6893,10141,6894,10142,6895,10143,6896,6897,6898,6899,10144,6900,6901,10145,6902,6903,6904,6905,6906,6907,6908,6909,10146,6910,10147,6911,6912,6913,6914,10148,6915,10149,6916,6917,6918,6919,6920,10150,6921,10151,6922,10152,6923,6924,6925,6926,6927,6928,6929,6930,6931,6932,6933,6934,10153,6935,6936,6937,6938,10154,6939,6940,6941,10155,6942,6943,7032,39237,10184,7033,10185,7037,7038,7039,7047,7090,7653,7725,13045,13046,41042,41041,13047,13082,13083,13087,13086,14457,14456,14474,14473,14687,14688,14706,14705,14801,14800,45817,14956,14957,15122,15121,15193,15192,15236,15235,15244,15243,15313,15486,15487,15491,15492,15511,15510,15541,15542,15905,15906,19936,19937,20197,20198,20217,20218,20219,20220,20500,20501,20502,20503,20505,20504,20508,20509,20517,20516,20519,20518,20521,20520,20523,20522,20525,20524,20533,20532,20535,20534,20563,20562,20602,20603,24717,25305,25306,26626,26627,27513,27512,28769,28768,28862,28861,28874,28873,28889,28890,29011,29012,29223,30677,30676,30696,30697,30703,30702,36407,36406,37486,37487,38538,38539,38561,38560,38562,38563,38798,38799,39246,39247,39262,39261,40146,43538,43542,43548,52054,52055,52140,52139,52142,52141,52144,52143,8122,847,3805,8294,1581,3803,2753,20498,20499,2756,15619,15620,8733,3831,6861,6862,14400,14401,14406,14407,20511,20512,20510,27292,27293,27294,27295,27296,27297,27298,27299,27300,27301,27302,27303,27304,27305,27306,27307,27308,27309,27310,27311,27312,27313,27314,27315,27316,27317,27318,27319,27320,27321,27322,27323,27324,27325,27326,27327,27328,27329,27330,27331,27332,27333,27334,27075,27076,27077,27078,27079,27080,27081,27082,27083,27084,29005,29006,32290,32289,39466,41969,39593,42030,39686,42099,39915,42434,40078,40103,42935,40174,40118,42942,40177,40121,42945,40133,40189,42957,40201,40145,42968,40147,40202,42969,40211,40156,40212,40157,42978,40243,40325,42986,40249,40331,42992,40340,40258,43000,40352,40270,40271,40353,43012,41800,52233,53202,53208,54229,41801,52984,53203,53209,54230,41802,53204,53210,54231,41803,52421,53205,53211,54232,41804,53206,53212,54233,41805,54234,41806,54235,41807,52899,53207,53213,54236,41808,54237,43048,50268,54741,43062,54759,50286,43073,50298,43194,50375,54812,43198,54816,50379,43257,51456,54911,43286,51485,54940,43307,51494,54949,43311,51498,54953,43324,51511,54966,43342,51529,54984,43349,51536,54991,43355,51542,54997,43359,55001,51546,43360,51547,55002,43386,51573,55028,43389,51576,55031,43395,51582,55037,43419,55073,51606,43432,51619,55086,43436,51623,55090,43441,55095,51628,43446,55100,51633,43448,55102,51635,43451,51639,55105,43478,55132,51666,43490,55144,51678,43494,55148,51682,43505,51693,55159],["North American Pollen Database","North American Pollen Database","North American Pollen Database","North American Pollen Database","North American Pollen Database","North American Pollen Database","North American Pollen Database","North American Pollen Database","North American Pollen Database","North American Pollen Database","North American Pollen Database","North American Pollen Database","North American Pollen Database","North American Pollen Database","North American Pollen Database","North American Pollen Database","North American Pollen Database","North American Pollen Database","North American Pollen Database","North American Pollen Database","North American Pollen Database","North American Pollen Database","North American Pollen Database","North American Pollen Database","North American Pollen Database","North American Pollen Database","North American Pollen Database","North American Pollen Database","North American Pollen Database","North American Pollen Database","North American Pollen Database","North American Pollen Database","North American Pollen Database","North American Pollen Database","North American Pollen Database","North American Pollen Database","North American Pollen Database","North American Pollen Database","North American Pollen Database","North American Pollen Database","North American Pollen Database","North American Pollen Database","North American Pollen Database","North American Pollen Database","North American Pollen Database","North American Pollen Database","North American Pollen Database","North American Pollen Database","North American Pollen Database","North American Pollen Database","North American Pollen Database","North American Pollen Database","North American Pollen Database","North American Pollen Database","North American Pollen Database","North American Pollen Database","North American Pollen Database","North American Pollen Database","North American Pollen Database","North American Pollen Database","North American Pollen Database","North American Pollen Database","North American Pollen Database","North American Pollen Database","North American Pollen Database","North American Pollen Database","North American Pollen Database","North American Pollen Database","North American Pollen Database","North American Pollen Database","North American Pollen Database","North American Pollen Database","North American Pollen Database","North American Pollen Database","North American Pollen Database","North American Pollen Database","North American Pollen Database","North American Pollen Database","North American Pollen Database","North American Pollen Database","North American Pollen Database","North American Pollen Database","North American Pollen Database","North American Pollen Database","North American Pollen Database","FAUNMAP","FAUNMAP","FAUNMAP","FAUNMAP","FAUNMAP","FAUNMAP","FAUNMAP","FAUNMAP","FAUNMAP","FAUNMAP","FAUNMAP","FAUNMAP","FAUNMAP","FAUNMAP","FAUNMAP","FAUNMAP","FAUNMAP","FAUNMAP","FAUNMAP","FAUNMAP","FAUNMAP","FAUNMAP","FAUNMAP","FAUNMAP","FAUNMAP","FAUNMAP","FAUNMAP","FAUNMAP","FAUNMAP","FAUNMAP","FAUNMAP","FAUNMAP","FAUNMAP","FAUNMAP","FAUNMAP","FAUNMAP","FAUNMAP","FAUNMAP","FAUNMAP","FAUNMAP","FAUNMAP","FAUNMAP","FAUNMAP","FAUNMAP","FAUNMAP","FAUNMAP","FAUNMAP","FAUNMAP","FAUNMAP","FAUNMAP","FAUNMAP","FAUNMAP","FAUNMAP","FAUNMAP","FAUNMAP","FAUNMAP","FAUNMAP","FAUNMAP","FAUNMAP","FAUNMAP","FAUNMAP","FAUNMAP","FAUNMAP","FAUNMAP","FAUNMAP","FAUNMAP","FAUNMAP","FAUNMAP","FAUNMAP","FAUNMAP","FAUNMAP","FAUNMAP","FAUNMAP","FAUNMAP","FAUNMAP","FAUNMAP","FAUNMAP","FAUNMAP","FAUNMAP","FAUNMAP","FAUNMAP","FAUNMAP","FAUNMAP","FAUNMAP","FAUNMAP","FAUNMAP","FAUNMAP","FAUNMAP","FAUNMAP","FAUNMAP","FAUNMAP","FAUNMAP","FAUNMAP","FAUNMAP","FAUNMAP","FAUNMAP","FAUNMAP","FAUNMAP","FAUNMAP","FAUNMAP","FAUNMAP","FAUNMAP","FAUNMAP","FAUNMAP","FAUNMAP","FAUNMAP","FAUNMAP","FAUNMAP","FAUNMAP","FAUNMAP","FAUNMAP","FAUNMAP","FAUNMAP","FAUNMAP","FAUNMAP","FAUNMAP","FAUNMAP","FAUNMAP","FAUNMAP","FAUNMAP","FAUNMAP","FAUNMAP","FAUNMAP","FAUNMAP","FAUNMAP","FAUNMAP","FAUNMAP","FAUNMAP","FAUNMAP","FAUNMAP","FAUNMAP","FAUNMAP","FAUNMAP","FAUNMAP","FAUNMAP","FAUNMAP","FAUNMAP","FAUNMAP","FAUNMAP","FAUNMAP","FAUNMAP","FAUNMAP","FAUNMAP","FAUNMAP","FAUNMAP","FAUNMAP","FAUNMAP","FAUNMAP","FAUNMAP","FAUNMAP","FAUNMAP","FAUNMAP","FAUNMAP","FAUNMAP","FAUNMAP","FAUNMAP","FAUNMAP","FAUNMAP","FAUNMAP","FAUNMAP","North American Pollen Database","North American Pollen Database","Neotoma Biomarker Database","Neotoma Biomarker Database","North American Pollen Database","North American Non-Marine Ostracode Database Project (NANODe)","North American Non-Marine Ostracode Database Project (NANODe)","North American Non-Marine Ostracode Database Project (NANODe)","North American Non-Marine Ostracode Database Project (NANODe)","North American Non-Marine Ostracode Database Project (NANODe)","North American Non-Marine Ostracode Database Project (NANODe)","North American Non-Marine Ostracode Database Project (NANODe)","North American Non-Marine Ostracode Database Project (NANODe)","North American Non-Marine Ostracode Database Project (NANODe)","North American Non-Marine Ostracode Database Project (NANODe)","North American Non-Marine Ostracode Database Project (NANODe)","North American Non-Marine Ostracode Database Project (NANODe)","NDSU Insect Database","NDSU Insect Database","Neotoma Charcoal Data","North American Pollen Database","North American Pollen Database","NDSU Insect Database","NDSU Insect Database","North American Non-Marine Ostracode Database Project (NANODe)","North American Non-Marine Ostracode Database Project (NANODe)","FAUNMAP","FAUNMAP","North American Pollen Database","North American Pollen Database","North American Pollen Database","North American Pollen Database","North American Pollen Database","North American Pollen Database","North American Pollen Database","North American Pollen Database","North American Pollen Database","North American Non-Marine Ostracode Database Project (NANODe)","North American Non-Marine Ostracode Database Project (NANODe)","North American Non-Marine Ostracode Database Project (NANODe)","North American Non-Marine Ostracode Database Project (NANODe)","North American Pollen Database","North American Pollen Database","North American Non-Marine Ostracode Database Project (NANODe)","North American Non-Marine Ostracode Database Project (NANODe)","North American Non-Marine Ostracode Database Project (NANODe)","North American Non-Marine Ostracode Database Project (NANODe)","North American Non-Marine Ostracode Database Project (NANODe)","North American Non-Marine Ostracode Database Project (NANODe)","North American Non-Marine Ostracode Database Project (NANODe)","North American Non-Marine Ostracode Database Project (NANODe)","North American Non-Marine Ostracode Database Project (NANODe)","North American Non-Marine Ostracode Database Project (NANODe)","North American Non-Marine Ostracode Database Project (NANODe)","North American Non-Marine Ostracode Database Project (NANODe)","North American Non-Marine Ostracode Database Project (NANODe)","North American Non-Marine Ostracode Database Project (NANODe)","North American Non-Marine Ostracode Database Project (NANODe)","North American Non-Marine Ostracode Database Project (NANODe)","North American Non-Marine Ostracode Database Project (NANODe)","North American Non-Marine Ostracode Database Project (NANODe)","North American Non-Marine Ostracode Database Project (NANODe)","North American Non-Marine Ostracode Database Project (NANODe)","North American Non-Marine Ostracode Database Project (NANODe)","North American Non-Marine Ostracode Database Project (NANODe)","North American Non-Marine Ostracode Database Project (NANODe)","North American Non-Marine Ostracode Database Project (NANODe)","North American Non-Marine Ostracode Database Project (NANODe)","North American Non-Marine Ostracode Database Project (NANODe)","North American Non-Marine Ostracode Database Project (NANODe)","North American Non-Marine Ostracode Database Project (NANODe)","North American Non-Marine Ostracode Database Project (NANODe)","North American Non-Marine Ostracode Database Project (NANODe)","North American Non-Marine Ostracode Database Project (NANODe)","North American Non-Marine Ostracode Database Project (NANODe)","FAUNMAP","North American Pollen Database","North American Pollen Database","FAUNMAP","FAUNMAP","FAUNMAP","FAUNMAP","Canadian Museum of Nature-Delorme Ostracoda-Surface Samples","Canadian Museum of Nature-Delorme Ostracoda-Surface Samples","Canadian Museum of Nature-Delorme Ostracoda-Surface Samples","Canadian Museum of Nature-Delorme Ostracoda-Surface Samples","Canadian Museum of Nature-Delorme Ostracoda-Surface Samples","Canadian Museum of Nature-Delorme Ostracoda-Surface Samples","Canadian Museum of Nature-Delorme Ostracoda-Surface Samples","Canadian Museum of Nature-Delorme Ostracoda-Surface Samples","Canadian Museum of Nature-Delorme Ostracoda-Surface Samples","Canadian Museum of Nature-Delorme Ostracoda-Surface Samples","FAUNMAP","Canadian Museum of Nature-Delorme Ostracoda-Surface Samples","Canadian Museum of Nature-Delorme Ostracoda-Surface Samples","Canadian Museum of Nature-Delorme Ostracoda-Surface Samples","Canadian Museum of Nature-Delorme Ostracoda-Surface Samples","Canadian Museum of Nature-Delorme Ostracoda-Surface Samples","Canadian Museum of Nature-Delorme Ostracoda-Surface Samples","FAUNMAP","FAUNMAP","FAUNMAP","FAUNMAP","Canadian Museum of Nature-Delorme Ostracoda-Surface Samples","Canadian Museum of Nature-Delorme Ostracoda-Surface Samples","Canadian Museum of Nature-Delorme Ostracoda-Surface Samples","Canadian Museum of Nature-Delorme Ostracoda-Surface Samples","Canadian Museum of Nature-Delorme Ostracoda-Surface Samples","Canadian Museum of Nature-Delorme Ostracoda-Surface Samples","FAUNMAP","FAUNMAP","FAUNMAP","FAUNMAP","FAUNMAP","FAUNMAP","Academy of Natural Sciences of Drexel University","Academy of Natural Sciences of Drexel University","Academy of Natural Sciences of Drexel University","Academy of Natural Sciences of Drexel University","North American Pollen Database","North American Pollen Database","North American Pollen Database","North American Pollen Database","North American Pollen Database","North American Pollen Database","North American Pollen Database","North American Pollen Database","North American Pollen Database","North American Pollen Database","North American Pollen Database","North American Pollen Database","North American Pollen Database","North American Pollen Database","North American Pollen Database","North American Non-Marine Ostracode Database Project (NANODe)","North American Non-Marine Ostracode Database Project (NANODe)","North American Pollen Database","North American Non-Marine Ostracode Database Project (NANODe)","North American Non-Marine Ostracode Database Project (NANODe)","North American Pollen Database","North American Pollen Database","FAUNMAP","FAUNMAP","North American Non-Marine Ostracode Database Project (NANODe)","North American Non-Marine Ostracode Database Project (NANODe)","North American Non-Marine Ostracode Database Project (NANODe)","North American Non-Marine Ostracode Database Project (NANODe)","Neotoma Testate Amoebae Database","North American Pollen Database","Neotoma Testate Amoebae Database","Neotoma Testate Amoebae Database","Neotoma Testate Amoebae Database","Neotoma Testate Amoebae Database","Neotoma Testate Amoebae Database","Neotoma Testate Amoebae Database","Neotoma Testate Amoebae Database","Neotoma Testate Amoebae Database","Neotoma Testate Amoebae Database","Neotoma Testate Amoebae Database","Neotoma Testate Amoebae Database","Neotoma Testate Amoebae Database","Neotoma Testate Amoebae Database","Neotoma Testate Amoebae Database","Neotoma Testate Amoebae Database","Neotoma Testate Amoebae Database","Neotoma Testate Amoebae Database","Neotoma Testate Amoebae Database","Neotoma Testate Amoebae Database","Neotoma Testate Amoebae Database","Neotoma Testate Amoebae Database","Neotoma Testate Amoebae Database","Neotoma Testate Amoebae Database","Neotoma Testate Amoebae Database","Neotoma Testate Amoebae Database","Neotoma Testate Amoebae Database","Neotoma Testate Amoebae Database","Neotoma Testate Amoebae Database","Neotoma Testate Amoebae Database","Neotoma Testate Amoebae Database","Neotoma Testate Amoebae Database","Neotoma Testate Amoebae Database","Neotoma Testate Amoebae Database","Neotoma Testate Amoebae Database","Neotoma Testate Amoebae Database","Neotoma Testate Amoebae Database","Neotoma Testate Amoebae Database","Neotoma Testate Amoebae Database","Neotoma Testate Amoebae Database","Neotoma Testate Amoebae Database","Neotoma Testate Amoebae Database","Neotoma Testate Amoebae Database","Neotoma Testate Amoebae Database","Neotoma Testate Amoebae Database","Neotoma Testate Amoebae Database","Neotoma Testate Amoebae Database","Neotoma Testate Amoebae Database","Neotoma Testate Amoebae Database","Neotoma Testate Amoebae Database","Neotoma Testate Amoebae Database","Neotoma Testate Amoebae Database","Neotoma Testate Amoebae Database","Neotoma Testate Amoebae Database","Neotoma Testate Amoebae Database","Canadian Museum of Nature-Delorme Ostracoda-Surface Samples","Canadian Museum of Nature-Delorme Ostracoda-Surface Samples","Canadian Museum of Nature-Delorme Ostracoda-Surface Samples","Canadian Museum of Nature-Delorme Ostracoda-Surface Samples","Academy of Natural Sciences of Drexel University","Academy of Natural Sciences of Drexel University","Academy of Natural Sciences of Drexel University","Academy of Natural Sciences of Drexel University","Academy of Natural Sciences of Drexel University","Academy of Natural Sciences of Drexel University","Academy of Natural Sciences of Drexel University","Academy of Natural Sciences of Drexel University","Academy of Natural Sciences of Drexel University","Academy of Natural Sciences of Drexel University","Academy of Natural Sciences of Drexel University","Academy of Natural Sciences of Drexel University","Academy of Natural Sciences of Drexel University","Academy of Natural Sciences of Drexel University","Academy of Natural Sciences of Drexel University","Academy of Natural Sciences of Drexel University","Academy of Natural Sciences of Drexel University","Academy of Natural Sciences of Drexel University","Academy of Natural Sciences of Drexel University","Academy of Natural Sciences of Drexel University","Academy of Natural Sciences of Drexel University","Academy of Natural Sciences of Drexel University","Academy of Natural Sciences of Drexel University","Academy of Natural Sciences of Drexel University","Academy of Natural Sciences of Drexel University","Academy of Natural Sciences of Drexel University","Academy of Natural Sciences of Drexel University","Academy of Natural Sciences of Drexel University","Academy of Natural Sciences of Drexel University","Academy of Natural Sciences of Drexel University","Academy of Natural Sciences of Drexel University","Academy of Natural Sciences of Drexel University","Academy of Natural Sciences of Drexel University","Academy of Natural Sciences of Drexel University","Academy of Natural Sciences of Drexel University","Academy of Natural Sciences of Drexel University","Academy of Natural Sciences of Drexel University","Academy of Natural Sciences of Drexel University","Academy of Natural Sciences of Drexel University","Academy of Natural Sciences of Drexel University","Academy of Natural Sciences of Drexel University","Academy of Natural Sciences of Drexel University","Academy of Natural Sciences of Drexel University","Academy of Natural Sciences of Drexel University","Academy of Natural Sciences of Drexel University","Academy of Natural Sciences of Drexel University","St. Croix Watershed Research Station of the Science Museum of Minnesota","St. Croix Watershed Research Station of the Science Museum of Minnesota","St. Croix Watershed Research Station of the Science Museum of Minnesota","St. Croix Watershed Research Station of the Science Museum of Minnesota","Academy of Natural Sciences of Drexel University","St. Croix Watershed Research Station of the Science Museum of Minnesota","St. Croix Watershed Research Station of the Science Museum of Minnesota","St. Croix Watershed Research Station of the Science Museum of Minnesota","St. Croix Watershed Research Station of the Science Museum of Minnesota","Academy of Natural Sciences of Drexel University","St. Croix Watershed Research Station of the Science Museum of Minnesota","St. Croix Watershed Research Station of the Science Museum of Minnesota","St. Croix Watershed Research Station of the Science Museum of Minnesota","Academy of Natural Sciences of Drexel University","St. Croix Watershed Research Station of the Science Museum of Minnesota","St. Croix Watershed Research Station of the Science Museum of Minnesota","St. Croix Watershed Research Station of the Science Museum of Minnesota","St. Croix Watershed Research Station of the Science Museum of Minnesota","Academy of Natural Sciences of Drexel University","St. Croix Watershed Research Station of the Science Museum of Minnesota","St. Croix Watershed Research Station of the Science Museum of Minnesota","St. Croix Watershed Research Station of the Science Museum of Minnesota","Academy of Natural Sciences of Drexel University","St. Croix Watershed Research Station of the Science Museum of Minnesota","Academy of Natural Sciences of Drexel University","St. Croix Watershed Research Station of the Science Museum of Minnesota","Academy of Natural Sciences of Drexel University","St. Croix Watershed Research Station of the Science Museum of Minnesota","St. Croix Watershed Research Station of the Science Museum of Minnesota","St. Croix Watershed Research Station of the Science Museum of Minnesota","St. Croix Watershed Research Station of the Science Museum of Minnesota","Academy of Natural Sciences of Drexel University","St. Croix Watershed Research Station of the Science Museum of Minnesota","Academy of Natural Sciences of Drexel University","Academy of Natural Sciences of Drexel University","Academy of Natural Sciences of Drexel University","Academy of Natural Sciences of Drexel University","Academy of Natural Sciences of Drexel University","Academy of Natural Sciences of Drexel University","Academy of Natural Sciences of Drexel University","Academy of Natural Sciences of Drexel University","Academy of Natural Sciences of Drexel University","Academy of Natural Sciences of Drexel University","Academy of Natural Sciences of Drexel University","Academy of Natural Sciences of Drexel University","Academy of Natural Sciences of Drexel University","Academy of Natural Sciences of Drexel University","Academy of Natural Sciences of Drexel University","Academy of Natural Sciences of Drexel University","Academy of Natural Sciences of Drexel University","Academy of Natural Sciences of Drexel University","Academy of Natural Sciences of Drexel University","Academy of Natural Sciences of Drexel University","Academy of Natural Sciences of Drexel University","Academy of Natural Sciences of Drexel University","Academy of Natural Sciences of Drexel University","Academy of Natural Sciences of Drexel University","Academy of Natural Sciences of Drexel University","Academy of Natural Sciences of Drexel University","Academy of Natural Sciences of Drexel University","Academy of Natural Sciences of Drexel University","Academy of Natural Sciences of Drexel University","Academy of Natural Sciences of Drexel University","Academy of Natural Sciences of Drexel University","Academy of Natural Sciences of Drexel University","Academy of Natural Sciences of Drexel University","Academy of Natural Sciences of Drexel University","Academy of Natural Sciences of Drexel University","Academy of Natural Sciences of Drexel University","Academy of Natural Sciences of Drexel University","Academy of Natural Sciences of Drexel University","Academy of Natural Sciences of Drexel University","Academy of Natural Sciences of Drexel University","Academy of Natural Sciences of Drexel University","Academy of Natural Sciences of Drexel University","Academy of Natural Sciences of Drexel University","Academy of Natural Sciences of Drexel University","Academy of Natural Sciences of Drexel University","Academy of Natural Sciences of Drexel University","Academy of Natural Sciences of Drexel University","Academy of Natural Sciences of Drexel University","Academy of Natural Sciences of Drexel University","Academy of Natural Sciences of Drexel University","Academy of Natural Sciences of Drexel University","Academy of Natural Sciences of Drexel University","Academy of Natural Sciences of Drexel University","Academy of Natural Sciences of Drexel University","Academy of Natural Sciences of Drexel University","Academy of Natural Sciences of Drexel University","Academy of Natural Sciences of Drexel University","Academy of Natural Sciences of Drexel University","Academy of Natural Sciences of Drexel University","Academy of Natural Sciences of Drexel University","Academy of Natural Sciences of Drexel University","Academy of Natural Sciences of Drexel University","Academy of Natural Sciences of Drexel University","Academy of Natural Sciences of Drexel University","Academy of Natural Sciences of Drexel University","Academy of Natural Sciences of Drexel University","Academy of Natural Sciences of Drexel University","Academy of Natural Sciences of Drexel University","Academy of Natural Sciences of Drexel University","Academy of Natural Sciences of Drexel University","Academy of Natural Sciences of Drexel University","Academy of Natural Sciences of Drexel University","Academy of Natural Sciences of Drexel University","Academy of Natural Sciences of Drexel University","Academy of Natural Sciences of Drexel University","Academy of Natural Sciences of Drexel University","Academy of Natural Sciences of Drexel University","Academy of Natural Sciences of Drexel University","Academy of Natural Sciences of Drexel University","Academy of Natural Sciences of Drexel University","Academy of Natural Sciences of Drexel University","Academy of Natural Sciences of Drexel University","Academy of Natural Sciences of Drexel University","Academy of Natural Sciences of Drexel University","Academy of Natural Sciences of Drexel University","Academy of Natural Sciences of Drexel University"],["geochronologic","pollen","geochronologic","pollen","pollen","geochronologic","geochronologic","pollen","pollen","geochronologic","geochronologic","pollen","geochronologic","pollen","geochronologic","pollen","pollen surface sample","pollen","geochronologic","pollen surface sample","pollen surface sample","pollen surface sample","pollen surface sample","pollen surface sample","pollen surface sample","pollen surface sample","pollen surface sample","pollen surface sample","pollen surface sample","pollen surface sample","pollen surface sample","pollen surface sample","pollen surface sample","pollen surface sample","pollen surface sample","pollen surface sample","pollen surface sample","pollen surface sample","pollen surface sample","pollen surface sample","pollen surface sample","pollen surface sample","pollen surface sample","pollen surface sample","pollen surface sample","pollen surface sample","pollen surface sample","pollen surface sample","pollen surface sample","pollen surface sample","pollen surface sample","pollen surface sample","pollen surface sample","pollen surface sample","pollen surface sample","pollen surface sample","pollen surface sample","pollen surface sample","pollen surface sample","pollen surface sample","pollen surface sample","pollen surface sample","pollen surface sample","pollen surface sample","pollen surface sample","pollen surface sample","pollen surface sample","pollen surface sample","pollen surface sample","pollen surface sample","pollen surface sample","pollen surface sample","pollen surface sample","pollen surface sample","pollen surface sample","pollen surface sample","pollen surface sample","pollen surface sample","pollen surface sample","pollen surface sample","pollen surface sample","pollen","geochronologic","pollen","geochronologic","vertebrate fauna","geochronologic","vertebrate fauna","geochronologic","vertebrate fauna","geochronologic","vertebrate fauna","vertebrate fauna","vertebrate fauna","vertebrate fauna","vertebrate fauna","vertebrate fauna","vertebrate fauna","vertebrate fauna","vertebrate fauna","geochronologic","vertebrate fauna","geochronologic","vertebrate fauna","vertebrate fauna","geochronologic","vertebrate fauna","vertebrate fauna","vertebrate fauna","vertebrate fauna","geochronologic","vertebrate fauna","vertebrate fauna","geochronologic","vertebrate fauna","geochronologic","vertebrate fauna","vertebrate fauna","vertebrate fauna","geochronologic","vertebrate fauna","geochronologic","vertebrate fauna","vertebrate fauna","vertebrate fauna","geochronologic","vertebrate fauna","geochronologic","vertebrate fauna","vertebrate fauna","vertebrate fauna","vertebrate fauna","vertebrate fauna","vertebrate fauna","vertebrate fauna","vertebrate fauna","vertebrate fauna","vertebrate fauna","vertebrate fauna","vertebrate fauna","geochronologic","vertebrate fauna","geochronologic","vertebrate fauna","vertebrate fauna","geochronologic","vertebrate fauna","vertebrate fauna","vertebrate fauna","vertebrate fauna","vertebrate fauna","vertebrate fauna","vertebrate fauna","vertebrate fauna","vertebrate fauna","geochronologic","vertebrate fauna","vertebrate fauna","geochronologic","vertebrate fauna","vertebrate fauna","vertebrate fauna","vertebrate fauna","vertebrate fauna","vertebrate fauna","vertebrate fauna","vertebrate fauna","vertebrate fauna","geochronologic","vertebrate fauna","geochronologic","vertebrate fauna","geochronologic","vertebrate fauna","vertebrate fauna","vertebrate fauna","vertebrate fauna","geochronologic","vertebrate fauna","vertebrate fauna","geochronologic","vertebrate fauna","vertebrate fauna","vertebrate fauna","vertebrate fauna","vertebrate fauna","vertebrate fauna","vertebrate fauna","vertebrate fauna","geochronologic","vertebrate fauna","geochronologic","vertebrate fauna","vertebrate fauna","vertebrate fauna","vertebrate fauna","geochronologic","vertebrate fauna","geochronologic","vertebrate fauna","vertebrate fauna","vertebrate fauna","vertebrate fauna","vertebrate fauna","geochronologic","vertebrate fauna","geochronologic","vertebrate fauna","geochronologic","vertebrate fauna","vertebrate fauna","vertebrate fauna","vertebrate fauna","vertebrate fauna","vertebrate fauna","vertebrate fauna","vertebrate fauna","vertebrate fauna","vertebrate fauna","vertebrate fauna","vertebrate fauna","geochronologic","vertebrate fauna","vertebrate fauna","vertebrate fauna","vertebrate fauna","geochronologic","vertebrate fauna","vertebrate fauna","vertebrate fauna","geochronologic","vertebrate fauna","vertebrate fauna","vertebrate fauna","vertebrate fauna","geochronologic","vertebrate fauna","geochronologic","vertebrate fauna","vertebrate fauna","vertebrate fauna","vertebrate fauna","vertebrate fauna","vertebrate fauna","vertebrate fauna","geochronologic","plant macrofossil","biomarker","stable isotope","pollen","water chemistry","ostracode surface sample","ostracode surface sample","water chemistry","ostracode surface sample","water chemistry","ostracode surface sample","water chemistry","water chemistry","ostracode surface sample","ostracode surface sample","water chemistry","insect","geochronologic","charcoal","geochronologic","pollen","insect","geochronologic","ostracode surface sample","water chemistry","vertebrate fauna","geochronologic","pollen","geochronologic","pollen","geochronologic","pollen","geochronologic","pollen","pollen","geochronologic","water chemistry","ostracode surface sample","water chemistry","ostracode surface sample","geochronologic","pollen","water chemistry","ostracode surface sample","water chemistry","ostracode surface sample","water chemistry","ostracode surface sample","water chemistry","ostracode surface sample","water chemistry","ostracode surface sample","ostracode surface sample","water chemistry","water chemistry","ostracode surface sample","ostracode surface sample","water chemistry","ostracode surface sample","water chemistry","ostracode surface sample","water chemistry","ostracode surface sample","water chemistry","ostracode surface sample","water chemistry","ostracode surface sample","water chemistry","ostracode surface sample","water chemistry","ostracode surface sample","water chemistry","water chemistry","ostracode surface sample","vertebrate fauna","geochronologic","pollen","geochronologic","vertebrate fauna","vertebrate fauna","geochronologic","ostracode surface sample","water chemistry","ostracode surface sample","water chemistry","ostracode surface sample","water chemistry","water chemistry","ostracode surface sample","water chemistry","ostracode surface sample","vertebrate fauna","ostracode surface sample","water chemistry","water chemistry","ostracode surface sample","ostracode surface sample","water chemistry","vertebrate fauna","geochronologic","geochronologic","vertebrate fauna","water chemistry","ostracode surface sample","ostracode surface sample","water chemistry","water chemistry","ostracode surface sample","geochronologic","vertebrate fauna","geochronologic","vertebrate fauna","vertebrate fauna","geochronologic","diatom","water chemistry","water chemistry","water chemistry","geochronologic","pollen","pollen","geochronologic","pollen","geochronologic","pollen","geochronologic","geochronologic","pollen","pollen surface sample","geochronologic","pollen","pollen surface sample","pollen surface sample","water chemistry","ostracode surface sample","pollen surface sample","water chemistry","ostracode surface sample","geochronologic","pollen","vertebrate fauna","vertebrate fauna","water chemistry","ostracode surface sample","water chemistry","ostracode surface sample","testate amoebae","pollen","geochronologic","testate amoebae surface sample","testate amoebae surface sample","testate amoebae surface sample","testate amoebae surface sample","testate amoebae surface sample","testate amoebae surface sample","testate amoebae surface sample","testate amoebae surface sample","testate amoebae surface sample","testate amoebae surface sample","testate amoebae surface sample","testate amoebae surface sample","testate amoebae surface sample","testate amoebae surface sample","testate amoebae surface sample","testate amoebae surface sample","testate amoebae surface sample","testate amoebae surface sample","testate amoebae surface sample","testate amoebae surface sample","testate amoebae surface sample","testate amoebae surface sample","testate amoebae surface sample","testate amoebae surface sample","testate amoebae surface sample","testate amoebae surface sample","testate amoebae surface sample","testate amoebae surface sample","testate amoebae surface sample","testate amoebae surface sample","testate amoebae surface sample","testate amoebae surface sample","testate amoebae surface sample","testate amoebae surface sample","testate amoebae surface sample","testate amoebae surface sample","testate amoebae surface sample","testate amoebae surface sample","testate amoebae surface sample","testate amoebae surface sample","testate amoebae surface sample","testate amoebae surface sample","testate amoebae surface sample","testate amoebae surface sample","testate amoebae surface sample","testate amoebae surface sample","testate amoebae surface sample","testate amoebae surface sample","testate amoebae surface sample","testate amoebae surface sample","testate amoebae surface sample","testate amoebae surface sample","testate amoebae surface sample","water chemistry","ostracode surface sample","ostracode surface sample","water chemistry","diatom surface sample","water chemistry","diatom surface sample","water chemistry","diatom","water chemistry","diatom","water chemistry","diatom","diatom","water chemistry","diatom","diatom","water chemistry","diatom","diatom","water chemistry","diatom","diatom","water chemistry","diatom","diatom","water chemistry","diatom","diatom","water chemistry","diatom","diatom","diatom","diatom","water chemistry","diatom","diatom","water chemistry","diatom","diatom","water chemistry","diatom","diatom","water chemistry","diatom","diatom","diatom","diatom","water chemistry","water chemistry","diatom","diatom surface sample","diatom surface sample","diatom surface sample","water chemistry","diatom","diatom surface sample","diatom surface sample","diatom surface sample","water chemistry","diatom surface sample","diatom surface sample","diatom surface sample","water chemistry","diatom","diatom surface sample","diatom surface sample","diatom surface sample","water chemistry","diatom surface sample","diatom surface sample","diatom surface sample","water chemistry","diatom surface sample","water chemistry","diatom surface sample","water chemistry","diatom","diatom surface sample","diatom surface sample","diatom surface sample","water chemistry","diatom surface sample","water chemistry","diatom","diatom","water chemistry","diatom","diatom","water chemistry","diatom","water chemistry","diatom surface sample","diatom bottom sample","water chemistry","diatom bottom sample","diatom surface sample","water chemistry","diatom surface sample","diatom bottom sample","water chemistry","diatom surface sample","diatom bottom sample","water chemistry","diatom surface sample","diatom bottom sample","water chemistry","diatom surface sample","diatom bottom sample","water chemistry","diatom surface sample","diatom bottom sample","water chemistry","diatom surface sample","diatom bottom sample","water chemistry","diatom surface sample","diatom bottom sample","water chemistry","diatom surface sample","diatom bottom sample","water chemistry","diatom bottom sample","diatom surface sample","water chemistry","diatom surface sample","diatom bottom sample","water chemistry","diatom surface sample","diatom bottom sample","water chemistry","diatom surface sample","diatom bottom sample","water chemistry","diatom surface sample","diatom bottom sample","water chemistry","diatom bottom sample","diatom surface sample","water chemistry","diatom surface sample","diatom bottom sample","water chemistry","diatom surface sample","diatom bottom sample","water chemistry","diatom bottom sample","diatom surface sample","water chemistry","diatom bottom sample","diatom surface sample","water chemistry","diatom bottom sample","diatom surface sample","water chemistry","diatom surface sample","diatom bottom sample","water chemistry","diatom bottom sample","diatom surface sample","water chemistry","diatom bottom sample","diatom surface sample","water chemistry","diatom bottom sample","diatom surface sample","water chemistry","diatom surface sample","diatom bottom sample"],[null,13240,null,12593,15014,null,null,10645,15783,null,null,15444,null,16295,null,14116,null,13732,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,15937,null,13930,null,2490,null,1125,null,235,null,950,1650,10000,10000,1650,5000,3000,1250,1150,null,730,null,13200,1150,null,3510,1350,3000,960,null,5000,1555,null,3930,null,1650,3850,5000,null,112,null,1650,2200,2200,null,650,null,1840,1650,1050,850,279,1350,850,1650,500,1450,3000,3000,null,950,null,770,24000,null,15000,15000,15000,15000,15000,15000,15000,15000,10200,null,15000,15000,null,12845,15000,15000,15000,15000,15000,15000,15000,15000,null,15000,null,15000,null,13385,15000,110000,15000,null,11400,9600,null,15000,15000,15000,15000,15000,15000,15000,11220,null,15000,null,5840,35000,35000,35000,null,11000,null,13200,35000,35000,35000,11160,null,12320,null,15000,null,11280,35000,35000,110000,110000,110000,110000,110000,110000,110000,110000,110000,null,10750,110000,110000,11850,null,110000,35000,110000,null,11200,110000,11320,14666,null,505,null,35000,35000,35000,11150,110000,1650,309,null,null,null,null,7829,null,null,null,null,-45,-45,-45,-45,-45,-45,-45,-45,5108,null,18066,null,20635,16446,null,-45,-45,16080,null,38301,null,null,null,3336,null,3610,2908,null,-45,-45,-45,-45,null,15536,-44,-44,-45,-45,-45,-45,-45,-45,-45,-45,-45,-45,-45,-45,-45,-45,-45,-45,-45,-45,-45,-45,-45,-45,-45,-45,-45,-45,-45,-45,-41,-41,150000,null,4676,null,16460,10140,null,-25,-25,-25,-25,-25,-25,-25,-25,-25,-25,2580000,-25,-25,-25,-25,-25,-25,885,null,null,49100,-24,-24,-24,-24,-24,-24,null,1500,null,15075,14695,null,null,-57,-57,-57,null,12347,12100,null,14572,null,11262,null,null,33542,null,null,3263,null,null,-45,-45,null,-45,-45,null,20200,null,125,-45,-45,-45,-45,null,3382,null,-51,-51,-51,-51,-51,-51,-51,-51,-51,-51,-51,-51,-51,-51,-51,-54,-54,-54,-54,-54,-54,-54,-54,-54,-54,-54,-54,-54,-54,-54,-54,-54,-54,-54,-54,-54,-54,-54,-54,-54,-54,-54,-54,-54,-54,-54,-54,-54,-54,-54,-54,-54,-54,-25,-25,-25,-25,-57,-57,-57,-57,null,-57,null,-57,null,null,-57,null,null,-57,null,null,-57,null,null,-57,null,null,-57,null,null,-57,null,null,null,null,-57,null,null,-57,null,null,-57,null,null,-57,null,null,null,null,-57,-57,null,-59,-64,-55,-57,null,-59,-64,-55,-57,-59,-64,-55,-57,null,-59,-64,-55,-57,-59,-64,-55,-57,-58,-57,-55,-57,null,-59,-64,-55,-57,-55,-57,null,null,-57,null,null,-57,null,-57,-57,null,-57,null,-57,-57,-57,null,-57,-57,null,-57,-57,null,-57,-57,null,-57,-57,null,-57,-57,null,-57,-57,null,-57,-57,null,-57,null,-57,-57,-57,null,-57,-57,null,-57,-57,null,-57,-57,null,-57,null,-57,-57,-57,null,-57,-57,null,-57,null,-57,-57,null,-57,-57,null,-57,-57,-57,null,-57,null,-57,-57,null,-57,-57,null,-57,-57,-57,null],[null,13131,null,-241,0,null,null,-21,-43,null,null,-11,null,14,null,42,null,1171,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,-76,null,0,null,1050,null,630,null,169,null,550,1150,1,1,1150,3000,100,1050,550,null,730,null,13200,550,null,390,650,1,960,null,1150,1555,null,3930,null,1150,950,1250,null,80,null,1150,1,70,null,300,null,600,1150,750,1,249,550,450,1150,1,200,1150,1150,null,600,null,1,24000,null,10000,10000,10000,10000,10000,10000,10000,10000,10200,null,10000,10000,null,10395,10000,10000,10000,10000,10000,10000,10000,10000,null,10000,null,10790,null,12983,10000,11500,10000,null,11400,9600,null,10000,10000,10000,10000,10000,10000,10000,11220,null,10000,null,5840,10000,10000,10000,null,11000,null,13200,10000,10000,10000,11160,null,9640,null,10000,null,11280,10000,10000,11500,11500,11500,11500,11500,11500,11500,11500,11500,null,10750,11500,11500,10840,null,11500,10000,11500,null,9910,11500,10920,13735,null,500,null,10000,10000,10000,11150,11500,1150,180,null,null,null,null,-88,null,null,null,null,-45,-45,-45,-45,-45,-45,-45,-45,4027,null,7633,null,7594,15267,null,-45,-45,15384,null,-35,null,null,null,-36,null,-94,-91,null,-45,-45,-45,-45,null,-58,-44,-44,-45,-45,-45,-45,-45,-45,-45,-45,-45,-45,-45,-45,-45,-45,-45,-45,-45,-45,-45,-45,-45,-45,-45,-45,-45,-45,-45,-45,-41,-41,14000,null,8,null,15820,9860,null,-25,-25,-25,-25,-25,-25,-25,-25,-25,-25,11800,-25,-25,-25,-25,-25,-25,735,null,null,37000,-24,-24,-24,-24,-24,-24,null,1260,null,14072,13745,null,null,-57,-57,-57,null,-3,-52,null,184,null,-53,null,null,-59,null,null,100,null,null,-45,-45,null,-45,-45,null,0,null,100,-45,-45,-45,-45,null,-46,null,-51,-51,-51,-51,-51,-51,-51,-51,-51,-51,-51,-51,-51,-51,-51,-54,-54,-54,-54,-54,-54,-54,-54,-54,-54,-54,-54,-54,-54,-54,-54,-54,-54,-54,-54,-54,-54,-54,-54,-54,-54,-54,-54,-54,-54,-54,-54,-54,-54,-54,-54,-54,-54,-25,-25,-25,-25,-57,-57,-57,-57,null,-57,null,-57,null,null,-57,null,null,-57,null,null,-57,null,null,-57,null,null,-57,null,null,-57,null,null,null,null,-57,null,null,-57,null,null,-57,null,null,-57,null,null,null,null,-57,-57,null,-59,-64,-55,-57,null,-59,-64,-55,-57,-59,-64,-55,-57,null,-59,-64,-55,-57,-59,-64,-55,-57,-58,-57,-55,-57,null,-59,-64,-55,-57,-55,-57,null,null,-57,null,null,-57,null,-57,-57,null,-57,null,-57,-57,-57,null,-57,-57,null,-57,-57,null,-57,-57,null,-57,-57,null,-57,-57,null,-57,-57,null,-57,-57,null,-57,null,-57,-57,-57,null,-57,-57,null,-57,-57,null,-57,-57,null,-57,null,-57,-57,-57,null,-57,-57,null,-57,null,-57,-57,null,-57,-57,null,-57,-57,-57,null,-57,null,-57,-57,null,-57,-57,null,-57,-57,-57,null],[null,null,null,"Bailey taxon code numbers 22, 68, 69, 70, 75 were assigned to our taxon code 157 (unknown).",null,null,null,"Dropped counts COHMAP's «DUMMY» variables (1992-12-28).","Corylus at 875 cm had a count of 211. The raw data table in Ahearn 1976 shows no Corylus grains at that level.",null,null,null,null,"Acer at 80 cm was changed from 64 to 34 based on table .",null,null,null,"Don't believe DUMMY1-DUMMY4. Mesic herbs, wetland shrubs etc. Dropped counts COHMAPS's «DUMMY» variables (1992-12-28).",null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,"Synonymization made from original publication:\r\nVulpes fulva -&gt; Vulpes vulpes","Synonymization made from original publication:\r\nVulpes fulva -&gt; Vulpes vulpes",null,null,"Synonymization made from original publication:\r\nVulpes fulva -&gt; Vulpes vulpes",null,null,null,null,null,null,"Synonymization made from original publication:\r\nVulpes fulva -&gt; Vulpes vulpes",null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,"Synonymization made from original publication:\r\nPitymys pinetorum -&gt; Microtus pinetorum",null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,"Synonymization made from original publication:\r\nBootherium sargenti -&gt; Bootherium bombifrons",null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,"Synonymization made from original publication:\r\nRangifer arcticus -&gt; Rangifer tarandus","Synonymization made from original publication:\r\nRangifer arcticus -&gt; Rangifer tarandus",null,null,null,null,"Synonymization made from original publication:\r\nVulpes fulva -&gt; Vulpes vulpes",null,null,"See Nichols, 2010 for methodological information.","See Nichols, 2010 for methodological information.","Age model was fit using a Bayesian approach developed by Blaauw and Christen which uses realistic estimates of sediment accumulation and the autocorrelation of this accumulation to relate the depth of any given pollen sample to calibrated AMS dates.",null,null,null,null,null,"Biota present at time of sampling: algae (greens), aquatic insects, molluscs, fish, birds (great blue heron), submerged macrophytes (nuphar), and emergent plants (nymphea). A bloom of green algae was also present in the water column. Thermocline was at 4 m, probably anoxic in hypolimnion.","Biota present at time of sampling: algae (greens), aquatic insects, molluscs, fish, birds (great blue heron), submerged macrophytes, and emergent plants.",null,null,null,null,null,null,null,null,null,null,"Molluscs, Ostracods, Plants, and Vertebrate fossils are discussed in the paper along with insect fossils.",null,"Biota present at time of sampling: algae, aquatic insects, chara, molluscs, fish, birds (swans), frogs, snakes, submerged macrophytes, and emergent plants (cattails and bullrushes).",null,"Western exposure within Bondi Site is 400 m long and 14 m high.",null,null,null,"There are no dates associated with this core",null,null,null,null,null,null,null,null,null,null,"All dates described in Wang et al. 2015 Holocene.","Note that Gill included Sporormiella in identifications, but Wang did not.  So absence of Sporormiella above 736.5cm is absence of search, not necessarily true absence.",null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,"Initially coded by Bart Weis on 26 Feb 2002.",null,null,null,"Initially coded by Bart Weis on 23 Jan 2001.","Initially coded by Bart Weis on 23 Jan 2001.\r\nFound were two sets of molars and other smaller remains.","McAndrews and Jackson (1988:167) state that the C14  dates are 'younger (by 300-3300 years) than those of the associated geochroniloccally assignable features...' and that 'Pollen analysis shows that they can all be assigned to zone 1 and thus are older than 10,000 years B.P.'",null,null,null,null,null,null,null,null,null,null,"Initially coded by Bart Weis on 11 Jan 2002.",null,null,null,null,null,null,"Initially coded by Bart Weis on 24 Jan 2001.",null,null,"Initially coded by Bart Weis on 13 Apr 1999.",null,null,null,null,null,null,null,"Initially coded by Bart Weis on 23 Jan 2001.",null,null,null,null,null,"“Water chemistry samples were collected from the epilimnion using a 2 m PVC tube.  Sampling depth was 2 m unless euphotic zone (Secchi disk depth x 2) was &lt;2m, in which case, water sampling depth was equal to the euphotic zone.  All methods are described in EPA NLA publications.  Basic measurement data were added to Neotoma; flags and tags were not.  Data and documentation are available from the EPA website https://www.epa.gov/national-aquatic-resource-surveys/data-national-aquatic-resource-surveys (as of April 2019).”","“Water chemistry samples were collected from the epilimnion using a 2 m PVC tube.  Sampling depth was 2 m unless euphotic zone (Secchi disk depth x 2) was &lt;2m, in which case, water sampling depth was equal to the euphotic zone.  All methods are described in EPA NLA publications.  Basic measurement data were added to Neotoma; flags and tags were not.  Data and documentation are available from the EPA website https://www.epa.gov/national-aquatic-resource-surveys/data-national-aquatic-resource-surveys (as of April 2019).”","“Water chemistry samples were collected from the epilimnion using a 2 m PVC tube.  Sampling depth was 2 m unless euphotic zone (Secchi disk depth x 2) was &lt;2m, in which case, water sampling depth was equal to the euphotic zone.  All methods are described in EPA NLA publications.  Basic measurement data were added to Neotoma; flags and tags were not.  Data and documentation are available from the EPA website https://www.epa.gov/national-aquatic-resource-surveys/data-national-aquatic-resource-surveys (as of April 2019).”",null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,"\r\n","Biota present at time of sampling: aquatic insects, molluscs, birds (canada geese), and emergent plants (watercress, forget-me-nots, and iris). ",null,null,null,"For Pinus differentiations Minckley did not roll grains and Booth did. Therefore, if complete dataset is used it is recommended to lump all pine types.",null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,"Taxonomic nomenclature is that origninally used by the analysts.  Names have not been lumped.","Water chemistry samples were collected from the epilimnion using a 2 m PVC tube.  Sampling depth was 2 m unless euphotic zone (Secchi disk depth x 2) was &lt;2m, in which case, water sampling depth was equal to the euphotic zone.  All methods are described in EPA NLA publications.  Basic measurement data were added to Neotoma; flags and tags were not.  Data and documentation are available from the EPA website https://www.epa.gov/national-aquatic-resource-surveys/data-national-aquatic-resource-surveys (as of April 2019).","Taxonomic nomenclature is that origninally used by the analysts.  Names have not been lumped.","Taxonomic nomenclature is that origninally used by the analysts.  Names have not been lumped.\r\n“Water chemistry samples were collected from the epilimnion using a 2 m PVC tube.  Sampling depth was 2 m unless euphotic zone (Secchi disk depth x 2) was &lt;2m, in which case, water sampling depth was equal to the euphotic zone.  All methods are described in EPA NLA publications.  Basic measurement data were added to Neotoma; flags and tags were not.  Data and documentation are available from the EPA website https://www.epa.gov/national-aquatic-resource-surveys/data-national-aquatic-resource-surveys (as of April 2019).”","Taxonomic nomenclature is that origninally used by the analysts.  Names have not been lumped.","Taxonomic nomenclature is that origninally used by the analysts.  Names have not been lumped.\r\n“Water chemistry samples were collected from the epilimnion using a 2 m PVC tube.  Sampling depth was 2 m unless euphotic zone (Secchi disk depth x 2) was &lt;2m, in which case, water sampling depth was equal to the euphotic zone.  All methods are described in EPA NLA publications.  Basic measurement data were added to Neotoma; flags and tags were not.  Data and documentation are available from the EPA website https://www.epa.gov/national-aquatic-resource-surveys/data-national-aquatic-resource-surveys (as of April 2019).”","Taxonomic nomenclature is that origninally used by the analysts.  Names have not been lumped.","Taxonomic nomenclature is that origninally used by the analysts.  Names have not been lumped.\r\n“Water chemistry samples were collected from the epilimnion using a 2 m PVC tube.  Sampling depth was 2 m unless euphotic zone (Secchi disk depth x 2) was &lt;2m, in which case, water sampling depth was equal to the euphotic zone.  All methods are described in EPA NLA publications.  Basic measurement data were added to Neotoma; flags and tags were not.  Data and documentation are available from the EPA website https://www.epa.gov/national-aquatic-resource-surveys/data-national-aquatic-resource-surveys (as of April 2019).”",null,null,"“Water chemistry samples were collected from the epilimnion using a 2 m PVC tube.  Sampling depth was 2 m unless euphotic zone (Secchi disk depth x 2) was &lt;2m, in which case, water sampling depth was equal to the euphotic zone.  All methods are described in EPA NLA publications.  Basic measurement data were added to Neotoma; flags and tags were not.  Data and documentation are available from the EPA website https://www.epa.gov/national-aquatic-resource-surveys/data-national-aquatic-resource-surveys (as of April 2019).”",null,null,"“Water chemistry samples were collected from the epilimnion using a 2 m PVC tube.  Sampling depth was 2 m unless euphotic zone (Secchi disk depth x 2) was &lt;2m, in which case, water sampling depth was equal to the euphotic zone.  All methods are described in EPA NLA publications.  Basic measurement data were added to Neotoma; flags and tags were not.  Data and documentation are available from the EPA website https://www.epa.gov/national-aquatic-resource-surveys/data-national-aquatic-resource-surveys (as of April 2019).”",null,null,"“Water chemistry samples were collected from the epilimnion using a 2 m PVC tube.  Sampling depth was 2 m unless euphotic zone (Secchi disk depth x 2) was &lt;2m, in which case, water sampling depth was equal to the euphotic zone.  All methods are described in EPA NLA publications.  Basic measurement data were added to Neotoma; flags and tags were not.  Data and documentation are available from the EPA website https://www.epa.gov/national-aquatic-resource-surveys/data-national-aquatic-resource-surveys (as of April 2019).”",null,null,"“Water chemistry samples were collected from the epilimnion using a 2 m PVC tube.  Sampling depth was 2 m unless euphotic zone (Secchi disk depth x 2) was &lt;2m, in which case, water sampling depth was equal to the euphotic zone.  All methods are described in EPA NLA publications.  Basic measurement data were added to Neotoma; flags and tags were not.  Data and documentation are available from the EPA website https://www.epa.gov/national-aquatic-resource-surveys/data-national-aquatic-resource-surveys (as of April 2019).”",null,null,"“Water chemistry samples were collected from the epilimnion using a 2 m PVC tube.  Sampling depth was 2 m unless euphotic zone (Secchi disk depth x 2) was &lt;2m, in which case, water sampling depth was equal to the euphotic zone.  All methods are described in EPA NLA publications.  Basic measurement data were added to Neotoma; flags and tags were not.  Data and documentation are available from the EPA website https://www.epa.gov/national-aquatic-resource-surveys/data-national-aquatic-resource-surveys (as of April 2019).”",null,null,"“Water chemistry samples were collected from the epilimnion using a 2 m PVC tube.  Sampling depth was 2 m unless euphotic zone (Secchi disk depth x 2) was &lt;2m, in which case, water sampling depth was equal to the euphotic zone.  All methods are described in EPA NLA publications.  Basic measurement data were added to Neotoma; flags and tags were not.  Data and documentation are available from the EPA website https://www.epa.gov/national-aquatic-resource-surveys/data-national-aquatic-resource-surveys (as of April 2019).”",null,null,null,null,"“Water chemistry samples were collected from the epilimnion using a 2 m PVC tube.  Sampling depth was 2 m unless euphotic zone (Secchi disk depth x 2) was &lt;2m, in which case, water sampling depth was equal to the euphotic zone.  All methods are described in EPA NLA publications.  Basic measurement data were added to Neotoma; flags and tags were not.  Data and documentation are available from the EPA website https://www.epa.gov/national-aquatic-resource-surveys/data-national-aquatic-resource-surveys (as of April 2019).”",null,null,"“Water chemistry samples were collected from the epilimnion using a 2 m PVC tube.  Sampling depth was 2 m unless euphotic zone (Secchi disk depth x 2) was &lt;2m, in which case, water sampling depth was equal to the euphotic zone.  All methods are described in EPA NLA publications.  Basic measurement data were added to Neotoma; flags and tags were not.  Data and documentation are available from the EPA website https://www.epa.gov/national-aquatic-resource-surveys/data-national-aquatic-resource-surveys (as of April 2019).”",null,null,"“Water chemistry samples were collected from the epilimnion using a 2 m PVC tube.  Sampling depth was 2 m unless euphotic zone (Secchi disk depth x 2) was &lt;2m, in which case, water sampling depth was equal to the euphotic zone.  All methods are described in EPA NLA publications.  Basic measurement data were added to Neotoma; flags and tags were not.  Data and documentation are available from the EPA website https://www.epa.gov/national-aquatic-resource-surveys/data-national-aquatic-resource-surveys (as of April 2019).”",null,null,"“Water chemistry samples were collected from the epilimnion using a 2 m PVC tube.  Sampling depth was 2 m unless euphotic zone (Secchi disk depth x 2) was &lt;2m, in which case, water sampling depth was equal to the euphotic zone.  All methods are described in EPA NLA publications.  Basic measurement data were added to Neotoma; flags and tags were not.  Data and documentation are available from the EPA website https://www.epa.gov/national-aquatic-resource-surveys/data-national-aquatic-resource-surveys (as of April 2019).”",null,null,null,null,"“Water chemistry samples were collected from the epilimnion using a 2 m PVC tube.  Sampling depth was 2 m unless euphotic zone (Secchi disk depth x 2) was &lt;2m, in which case, water sampling depth was equal to the euphotic zone.  All methods are described in EPA NLA publications.  Basic measurement data were added to Neotoma; flags and tags were not.  Data and documentation are available from the EPA website https://www.epa.gov/national-aquatic-resource-surveys/data-national-aquatic-resource-surveys (as of April 2019).”","Microscope slides are at the St. Croix Watershed Research Station of the Science Museum of Minnesota.\r\n\r\nSediment cores were collected by NPS personnel.","Diatom slides are archived at the St. Croix Watershed Research Station, Science Museum of Minnesota.","Microscope slides are at the St. Croix Watershed Research Station of the Science Museum of Minnesota.\r\n\r\nSediment samples were collected by NPS personnel.","Microscope slides are at the St. Croix Watershed Research Station of the Science Museum of Minnesota.\r\n\r\nSediment samples were collected by NPS personnel.","Microscope slides are at the St. Croix Watershed Research Station of the Science Museum of Minnesota.\r\n\r\nSediment cores were collected by NPS personnel.","Microscope slides are at the St. Croix Watershed Research Station of the Science Museum of Minnesota.\r\n\r\nSediment cores were collected by NPS personnel.","Diatom slides are archived at the St. Croix Watershed Research Station, Science Museum of Minnesota.","Microscope slides are at the St. Croix Watershed Research Station of the Science Museum of Minnesota.\r\n\r\nSediment samples were collected by NPS personnel.","Microscope slides are at the St. Croix Watershed Research Station of the Science Museum of Minnesota.\r\n\r\nSediment samples were collected by NPS personnel.","Microscope slides are at the St. Croix Watershed Research Station of the Science Museum of Minnesota.\r\n\r\nSediment cores were collected by NPS personnel.","Microscope slides are at the St. Croix Watershed Research Station of the Science Museum of Minnesota.\r\n\r\nSediment cores were collected by NPS personnel.","Microscope slides are at the St. Croix Watershed Research Station of the Science Museum of Minnesota.\r\n\r\nSediment samples were collected by NPS personnel.","Microscope slides are at the St. Croix Watershed Research Station of the Science Museum of Minnesota.\r\n\r\nSediment samples were collected by NPS personnel.","Microscope slides are at the St. Croix Watershed Research Station of the Science Museum of Minnesota.\r\n\r\nSediment cores were collected by NPS personnel.","Microscope slides are at the St. Croix Watershed Research Station of the Science Museum of Minnesota.\r\n\r\nSediment cores were collected by NPS personnel.","Diatom slides are archived at the St. Croix Watershed Research Station, Science Museum of Minnesota.","Microscope slides are at the St. Croix Watershed Research Station of the Science Museum of Minnesota.\r\n\r\nSediment samples were collected by NPS personnel.","Microscope slides are at the St. Croix Watershed Research Station of the Science Museum of Minnesota.\r\n\r\nSediment samples were collected by NPS personnel.","Microscope slides are at the St. Croix Watershed Research Station of the Science Museum of Minnesota.\r\n\r\nSediment cores were collected by NPS personnel.","Microscope slides are at the St. Croix Watershed Research Station of the Science Museum of Minnesota.\r\n\r\nSediment cores were collected by NPS personnel.","Microscope slides are at the St. Croix Watershed Research Station of the Science Museum of Minnesota.\r\n\r\nSediment samples were collected by NPS personnel.","Microscope slides are at the St. Croix Watershed Research Station of the Science Museum of Minnesota.\r\n\r\nSediment samples were collected by NPS personnel.","Microscope slides are at the St. Croix Watershed Research Station of the Science Museum of Minnesota.\r\n\r\nSediment cores were collected by NPS personnel.","Microscope slides are at the St. Croix Watershed Research Station of the Science Museum of Minnesota.\r\n\r\nSediment cores were collected by NPS personnel.","Microscope slides are at the St. Croix Watershed Research Station of the Science Museum of Minnesota.\r\n\r\nSediment cores were collected by NPS personnel.","Microscope slides are at the St. Croix Watershed Research Station of the Science Museum of Minnesota.\r\n\r\nSediment cores were collected by NPS personnel.","Microscope slides are at the St. Croix Watershed Research Station of the Science Museum of Minnesota.\r\n\r\nSediment cores were collected by NPS personnel.","Microscope slides are at the St. Croix Watershed Research Station of the Science Museum of Minnesota.\r\n\r\nSediment cores were collected by NPS personnel.","Diatom slides are archived at the St. Croix Watershed Research Station, Science Museum of Minnesota.","Microscope slides are at the St. Croix Watershed Research Station of the Science Museum of Minnesota.\r\n\r\nSediment samples were collected by NPS personnel.","Microscope slides are at the St. Croix Watershed Research Station of the Science Museum of Minnesota.\r\n\r\nSediment samples were collected by NPS personnel.","Microscope slides are at the St. Croix Watershed Research Station of the Science Museum of Minnesota.\r\n\r\nSediment cores were collected by NPS personnel.","Microscope slides are at the St. Croix Watershed Research Station of the Science Museum of Minnesota.\r\n\r\nSediment cores were collected by NPS personnel.","Microscope slides are at the St. Croix Watershed Research Station of the Science Museum of Minnesota.\r\n\r\nSediment cores were collected by NPS personnel.","“Water chemistry samples were collected from the epilimnion using a 2 m PVC tube.  Sampling depth was 2 m unless euphotic zone (Secchi disk depth x 2) was &lt;2m, in which case, water sampling depth was equal to the euphotic zone.  All methods are described in EPA NLA publications.  Basic measurement data were added to Neotoma; flags and tags were not.  Data and documentation are available from the EPA website https://www.epa.gov/national-aquatic-resource-surveys/data-national-aquatic-resource-surveys (as of April 2019).”","The Academy of Natural Sciences of Philadelphia Diatom Herbarium has slides from the eastern and midwestern parts of the US.","The Academy of Natural Sciences of Philadelphia Diatom Herbarium has slides from the eastern and midwestern parts of the US.","“Water chemistry samples were collected from the epilimnion using a 2 m PVC tube.  Sampling depth was 2 m unless euphotic zone (Secchi disk depth x 2) was &lt;2m, in which case, water sampling depth was equal to the euphotic zone.  All methods are described in EPA NLA publications.  Basic measurement data were added to Neotoma; flags and tags were not.  Data and documentation are available from the EPA website https://www.epa.gov/national-aquatic-resource-surveys/data-national-aquatic-resource-surveys (as of April 2019).”","The Academy of Natural Sciences of Philadelphia Diatom Herbarium has slides from the eastern and midwestern parts of the US.","The Academy of Natural Sciences of Philadelphia Diatom Herbarium has slides from the eastern and midwestern parts of the US.","“Water chemistry samples were collected from the epilimnion using a 2 m PVC tube.  Sampling depth was 2 m unless euphotic zone (Secchi disk depth x 2) was &lt;2m, in which case, water sampling depth was equal to the euphotic zone.  All methods are described in EPA NLA publications.  Basic measurement data were added to Neotoma; flags and tags were not.  Data and documentation are available from the EPA website https://www.epa.gov/national-aquatic-resource-surveys/data-national-aquatic-resource-surveys (as of April 2019).”","The Academy of Natural Sciences of Philadelphia Diatom Herbarium has slides from the eastern and midwestern parts of the US.","“Water chemistry samples were collected from the epilimnion using a 2 m PVC tube.  Sampling depth was 2 m unless euphotic zone (Secchi disk depth x 2) was &lt;2m, in which case, water sampling depth was equal to the euphotic zone.  All methods are described in EPA NLA publications.  Basic measurement data were added to Neotoma; flags and tags were not.  Data and documentation are available from the EPA website https://www.epa.gov/national-aquatic-resource-surveys/data-national-aquatic-resource-surveys (as of April 2019).”","The Academy of Natural Sciences of Philadelphia Diatom Herbarium has slides from the eastern and midwestern parts of the US.","The Academy of Natural Sciences of Philadelphia Diatom Herbarium has slides from the eastern and midwestern parts of the US.","“Water chemistry samples were collected from the epilimnion using a 2 m PVC tube.  Sampling depth was 2 m unless euphotic zone (Secchi disk depth x 2) was &lt;2m, in which case, water sampling depth was equal to the euphotic zone.  All methods are described in EPA NLA publications.  Basic measurement data were added to Neotoma; flags and tags were not.  Data and documentation are available from the EPA website https://www.epa.gov/national-aquatic-resource-surveys/data-national-aquatic-resource-surveys (as of April 2019).”","The Academy of Natural Sciences of Philadelphia Diatom Herbarium has slides from the eastern and midwestern parts of the US.","The Academy of Natural Sciences of Philadelphia Diatom Herbarium has slides from the eastern and midwestern parts of the US.","“Water chemistry samples were collected from the epilimnion using a 2 m PVC tube.  Sampling depth was 2 m unless euphotic zone (Secchi disk depth x 2) was &lt;2m, in which case, water sampling depth was equal to the euphotic zone.  All methods are described in EPA NLA publications.  Basic measurement data were added to Neotoma; flags and tags were not.  Data and documentation are available from the EPA website https://www.epa.gov/national-aquatic-resource-surveys/data-national-aquatic-resource-surveys (as of April 2019).”","“Water chemistry samples were collected from the epilimnion using a 2 m PVC tube.  Sampling depth was 2 m unless euphotic zone (Secchi disk depth x 2) was &lt;2m, in which case, water sampling depth was equal to the euphotic zone.  All methods are described in EPA NLA publications.  Basic measurement data were added to Neotoma; flags and tags were not.  Data and documentation are available from the EPA website https://www.epa.gov/national-aquatic-resource-surveys/data-national-aquatic-resource-surveys (as of April 2019).”","Diatom slides for lakes in the Eastern US are archived in the Diatom Herbarium at the Academy of Natural Sciences in Philadelphia.","“Water chemistry samples were collected from the epilimnion using a 2 m PVC tube.  Sampling depth was 2 m unless euphotic zone (Secchi disk depth x 2) was &lt;2m, in which case, water sampling depth was equal to the euphotic zone.  All methods are described in EPA NLA publications.  Basic measurement data were added to Neotoma; flags and tags were not.  Data and documentation are available from the EPA website https://www.epa.gov/national-aquatic-resource-surveys/data-national-aquatic-resource-surveys (as of April 2019).”","“Water chemistry samples were collected from the epilimnion using a 2 m PVC tube.  Sampling depth was 2 m unless euphotic zone (Secchi disk depth x 2) was &lt;2m, in which case, water sampling depth was equal to the euphotic zone.  All methods are described in EPA NLA publications.  Basic measurement data were added to Neotoma; flags and tags were not.  Data and documentation are available from the EPA website https://www.epa.gov/national-aquatic-resource-surveys/data-national-aquatic-resource-surveys (as of April 2019).”","Diatom slides for lakes in the Eastern US are archived in the Diatom Herbarium at the Academy of Natural Sciences in Philadelphia.","“Water chemistry samples were collected from the epilimnion using a 2 m PVC tube.  Sampling depth was 2 m unless euphotic zone (Secchi disk depth x 2) was &lt;2m, in which case, water sampling depth was equal to the euphotic zone.  All methods are described in EPA NLA publications.  Basic measurement data were added to Neotoma; flags and tags were not.  Data and documentation are available from the EPA website https://www.epa.gov/national-aquatic-resource-surveys/data-national-aquatic-resource-surveys (as of April 2019).”","“Water chemistry samples were collected from the epilimnion using a 2 m PVC tube.  Sampling depth was 2 m unless euphotic zone (Secchi disk depth x 2) was &lt;2m, in which case, water sampling depth was equal to the euphotic zone.  All methods are described in EPA NLA publications.  Basic measurement data were added to Neotoma; flags and tags were not.  Data and documentation are available from the EPA website https://www.epa.gov/national-aquatic-resource-surveys/data-national-aquatic-resource-surveys (as of April 2019).”","Diatom microscope slides for sites in the eastern USA are in the Diatom Herbarium of the Academy of Natural Sciences of Philadelphia.","“Water chemistry samples were collected from the epilimnion using a 2 m PVC tube.  Sampling depth was 2 m unless euphotic zone (Secchi disk depth x 2) was &lt;2m, in which case, water sampling depth was equal to the euphotic zone.  All methods are described in EPA NLA publications.  Basic measurement data were added to Neotoma; flags and tags were not.  Data and documentation are available from the EPA website https://www.epa.gov/national-aquatic-resource-surveys/data-national-aquatic-resource-surveys (as of April 2019).”","“Water chemistry samples were collected from the epilimnion using a 2 m PVC tube.  Sampling depth was 2 m unless euphotic zone (Secchi disk depth x 2) was &lt;2m, in which case, water sampling depth was equal to the euphotic zone.  All methods are described in EPA NLA publications.  Basic measurement data were added to Neotoma; flags and tags were not.  Data and documentation are available from the EPA website https://www.epa.gov/national-aquatic-resource-surveys/data-national-aquatic-resource-surveys (as of April 2019).”","Diatom microscope slides for sites in the eastern USA are in the Diatom Herbarium of the Academy of Natural Sciences of Philadelphia.","“Water chemistry samples were collected from the epilimnion using a 2 m PVC tube.  Sampling depth was 2 m unless euphotic zone (Secchi disk depth x 2) was &lt;2m, in which case, water sampling depth was equal to the euphotic zone.  All methods are described in EPA NLA publications.  Basic measurement data were added to Neotoma; flags and tags were not.  Data and documentation are available from the EPA website https://www.epa.gov/national-aquatic-resource-surveys/data-national-aquatic-resource-surveys (as of April 2019).”","“Water chemistry samples were collected from the epilimnion using a 2 m PVC tube.  Sampling depth was 2 m unless euphotic zone (Secchi disk depth x 2) was &lt;2m, in which case, water sampling depth was equal to the euphotic zone.  All methods are described in EPA NLA publications.  Basic measurement data were added to Neotoma; flags and tags were not.  Data and documentation are available from the EPA website https://www.epa.gov/national-aquatic-resource-surveys/data-national-aquatic-resource-surveys (as of April 2019).”","Diatom microscope slides for sites in the eastern USA are in the Diatom Herbarium of the Academy of Natural Sciences of Philadelphia.","“Water chemistry samples were collected from the epilimnion using a 2 m PVC tube.  Sampling depth was 2 m unless euphotic zone (Secchi disk depth x 2) was &lt;2m, in which case, water sampling depth was equal to the euphotic zone.  All methods are described in EPA NLA publications.  Basic measurement data were added to Neotoma; flags and tags were not.  Data and documentation are available from the EPA website https://www.epa.gov/national-aquatic-resource-surveys/data-national-aquatic-resource-surveys (as of April 2019).”","“Water chemistry samples were collected from the epilimnion using a 2 m PVC tube.  Sampling depth was 2 m unless euphotic zone (Secchi disk depth x 2) was &lt;2m, in which case, water sampling depth was equal to the euphotic zone.  All methods are described in EPA NLA publications.  Basic measurement data were added to Neotoma; flags and tags were not.  Data and documentation are available from the EPA website https://www.epa.gov/national-aquatic-resource-surveys/data-national-aquatic-resource-surveys (as of April 2019).”","Diatom microscope slides for sites in the eastern USA are in the Diatom Herbarium of the Academy of Natural Sciences of Philadelphia.","“Water chemistry samples were collected from the epilimnion using a 2 m PVC tube.  Sampling depth was 2 m unless euphotic zone (Secchi disk depth x 2) was &lt;2m, in which case, water sampling depth was equal to the euphotic zone.  All methods are described in EPA NLA publications.  Basic measurement data were added to Neotoma; flags and tags were not.  Data and documentation are available from the EPA website https://www.epa.gov/national-aquatic-resource-surveys/data-national-aquatic-resource-surveys (as of April 2019).”","“Water chemistry samples were collected from the epilimnion using a 2 m PVC tube.  Sampling depth was 2 m unless euphotic zone (Secchi disk depth x 2) was &lt;2m, in which case, water sampling depth was equal to the euphotic zone.  All methods are described in EPA NLA publications.  Basic measurement data were added to Neotoma; flags and tags were not.  Data and documentation are available from the EPA website https://www.epa.gov/national-aquatic-resource-surveys/data-national-aquatic-resource-surveys (as of April 2019).”","Diatom microscope slides for sites in the eastern USA are in the Diatom Herbarium of the Academy of Natural Sciences of Philadelphia.","“Water chemistry samples were collected from the epilimnion using a 2 m PVC tube.  Sampling depth was 2 m unless euphotic zone (Secchi disk depth x 2) was &lt;2m, in which case, water sampling depth was equal to the euphotic zone.  All methods are described in EPA NLA publications.  Basic measurement data were added to Neotoma; flags and tags were not.  Data and documentation are available from the EPA website https://www.epa.gov/national-aquatic-resource-surveys/data-national-aquatic-resource-surveys (as of April 2019).”","“Water chemistry samples were collected from the epilimnion using a 2 m PVC tube.  Sampling depth was 2 m unless euphotic zone (Secchi disk depth x 2) was &lt;2m, in which case, water sampling depth was equal to the euphotic zone.  All methods are described in EPA NLA publications.  Basic measurement data were added to Neotoma; flags and tags were not.  Data and documentation are available from the EPA website https://www.epa.gov/national-aquatic-resource-surveys/data-national-aquatic-resource-surveys (as of April 2019).”","Diatom microscope slides are in the Diatom Herbarium at the Academy of Natural Sciences of Philadelphia.","“Water chemistry samples were collected from the epilimnion using a 2 m PVC tube.  Sampling depth was 2 m unless euphotic zone (Secchi disk depth x 2) was &lt;2m, in which case, water sampling depth was equal to the euphotic zone.  All methods are described in EPA NLA publications.  Basic measurement data were added to Neotoma; flags and tags were not.  Data and documentation are available from the EPA website https://www.epa.gov/national-aquatic-resource-surveys/data-national-aquatic-resource-surveys (as of April 2019).”","Diatom microscope slides are in the Diatom Herbarium at the Academy of Natural Sciences of Philadelphia.","“Water chemistry samples were collected from the epilimnion using a 2 m PVC tube.  Sampling depth was 2 m unless euphotic zone (Secchi disk depth x 2) was &lt;2m, in which case, water sampling depth was equal to the euphotic zone.  All methods are described in EPA NLA publications.  Basic measurement data were added to Neotoma; flags and tags were not.  Data and documentation are available from the EPA website https://www.epa.gov/national-aquatic-resource-surveys/data-national-aquatic-resource-surveys (as of April 2019).”","“Water chemistry samples were collected from the epilimnion using a 2 m PVC tube.  Sampling depth was 2 m unless euphotic zone (Secchi disk depth x 2) was &lt;2m, in which case, water sampling depth was equal to the euphotic zone.  All methods are described in EPA NLA publications.  Basic measurement data were added to Neotoma; flags and tags were not.  Data and documentation are available from the EPA website https://www.epa.gov/national-aquatic-resource-surveys/data-national-aquatic-resource-surveys (as of April 2019).”","“Water chemistry samples were collected from the epilimnion using a 2 m PVC tube.  Sampling depth was 2 m unless euphotic zone (Secchi disk depth x 2) was &lt;2m, in which case, water sampling depth was equal to the euphotic zone.  All methods are described in EPA NLA publications.  Basic measurement data were added to Neotoma; flags and tags were not.  Data and documentation are available from the EPA website https://www.epa.gov/national-aquatic-resource-surveys/data-national-aquatic-resource-surveys (as of April 2019).”","Diatom microscope slides are in the Diatom Herbarium at the Academy of Natural Sciences of Philadelphia.","“Water chemistry samples were collected from the epilimnion using a 2 m PVC tube.  Sampling depth was 2 m unless euphotic zone (Secchi disk depth x 2) was &lt;2m, in which case, water sampling depth was equal to the euphotic zone.  All methods are described in EPA NLA publications.  Basic measurement data were added to Neotoma; flags and tags were not.  Data and documentation are available from the EPA website https://www.epa.gov/national-aquatic-resource-surveys/data-national-aquatic-resource-surveys (as of April 2019).”","“Water chemistry samples were collected from the epilimnion using a 2 m PVC tube.  Sampling depth was 2 m unless euphotic zone (Secchi disk depth x 2) was &lt;2m, in which case, water sampling depth was equal to the euphotic zone.  All methods are described in EPA NLA publications.  Basic measurement data were added to Neotoma; flags and tags were not.  Data and documentation are available from the EPA website https://www.epa.gov/national-aquatic-resource-surveys/data-national-aquatic-resource-surveys (as of April 2019).”","Diatom microscope slides are in the Diatom Herbarium at the Academy of Natural Sciences of Philadelphia.","“Water chemistry samples were collected from the epilimnion using a 2 m PVC tube.  Sampling depth was 2 m unless euphotic zone (Secchi disk depth x 2) was &lt;2m, in which case, water sampling depth was equal to the euphotic zone.  All methods are described in EPA NLA publications.  Basic measurement data were added to Neotoma; flags and tags were not.  Data and documentation are available from the EPA website https://www.epa.gov/national-aquatic-resource-surveys/data-national-aquatic-resource-surveys (as of April 2019).”","“Water chemistry samples were collected from the epilimnion using a 2 m PVC tube.  Sampling depth was 2 m unless euphotic zone (Secchi disk depth x 2) was &lt;2m, in which case, water sampling depth was equal to the euphotic zone.  All methods are described in EPA NLA publications.  Basic measurement data were added to Neotoma; flags and tags were not.  Data and documentation are available from the EPA website https://www.epa.gov/national-aquatic-resource-surveys/data-national-aquatic-resource-surveys (as of April 2019).”","Diatom microscope slides are in the Diatom Herbarium at the Academy of Natural Sciences of Philadelphia.","“Water chemistry samples were collected from the epilimnion using a 2 m PVC tube.  Sampling depth was 2 m unless euphotic zone (Secchi disk depth x 2) was &lt;2m, in which case, water sampling depth was equal to the euphotic zone.  All methods are described in EPA NLA publications.  Basic measurement data were added to Neotoma; flags and tags were not.  Data and documentation are available from the EPA website https://www.epa.gov/national-aquatic-resource-surveys/data-national-aquatic-resource-surveys (as of April 2019).”","“Water chemistry samples were collected from the epilimnion using a 2 m PVC tube.  Sampling depth was 2 m unless euphotic zone (Secchi disk depth x 2) was &lt;2m, in which case, water sampling depth was equal to the euphotic zone.  All methods are described in EPA NLA publications.  Basic measurement data were added to Neotoma; flags and tags were not.  Data and documentation are available from the EPA website https://www.epa.gov/national-aquatic-resource-surveys/data-national-aquatic-resource-surveys (as of April 2019).”","Diatom microscope slides are in the Diatom Herbarium at the Academy of Natural Sciences of Philadelphia.","“Water chemistry samples were collected from the epilimnion using a 2 m PVC tube.  Sampling depth was 2 m unless euphotic zone (Secchi disk depth x 2) was &lt;2m, in which case, water sampling depth was equal to the euphotic zone.  All methods are described in EPA NLA publications.  Basic measurement data were added to Neotoma; flags and tags were not.  Data and documentation are available from the EPA website https://www.epa.gov/national-aquatic-resource-surveys/data-national-aquatic-resource-surveys (as of April 2019).”","Diatom slides for sites in the eastern USA are in the Diatom Herbarium at the Academy of Natural Sciences in Philadelphia.","“Water chemistry samples were collected from the epilimnion using a 2 m PVC tube.  Sampling depth was 2 m unless euphotic zone (Secchi disk depth x 2) was &lt;2m, in which case, water sampling depth was equal to the euphotic zone.  All methods are described in EPA NLA publications.  Basic measurement data were added to Neotoma; flags and tags were not.  Data and documentation are available from the EPA website https://www.epa.gov/national-aquatic-resource-surveys/data-national-aquatic-resource-surveys (as of April 2019).”","“Water chemistry samples were collected from the epilimnion using a 2 m PVC tube.  Sampling depth was 2 m unless euphotic zone (Secchi disk depth x 2) was &lt;2m, in which case, water sampling depth was equal to the euphotic zone.  All methods are described in EPA NLA publications.  Basic measurement data were added to Neotoma; flags and tags were not.  Data and documentation are available from the EPA website https://www.epa.gov/national-aquatic-resource-surveys/data-national-aquatic-resource-surveys (as of April 2019).”","“Water chemistry samples were collected from the epilimnion using a 2 m PVC tube.  Sampling depth was 2 m unless euphotic zone (Secchi disk depth x 2) was &lt;2m, in which case, water sampling depth was equal to the euphotic zone.  All methods are described in EPA NLA publications.  Basic measurement data were added to Neotoma; flags and tags were not.  Data and documentation are available from the EPA website https://www.epa.gov/national-aquatic-resource-surveys/data-national-aquatic-resource-surveys (as of April 2019).”","Diatom slides for sites in the eastern USA are in the Diatom Herbarium at the Academy of Natural Sciences in Philadelphia.","“Water chemistry samples were collected from the epilimnion using a 2 m PVC tube.  Sampling depth was 2 m unless euphotic zone (Secchi disk depth x 2) was &lt;2m, in which case, water sampling depth was equal to the euphotic zone.  All methods are described in EPA NLA publications.  Basic measurement data were added to Neotoma; flags and tags were not.  Data and documentation are available from the EPA website https://www.epa.gov/national-aquatic-resource-surveys/data-national-aquatic-resource-surveys (as of April 2019).”","“Water chemistry samples were collected from the epilimnion using a 2 m PVC tube.  Sampling depth was 2 m unless euphotic zone (Secchi disk depth x 2) was &lt;2m, in which case, water sampling depth was equal to the euphotic zone.  All methods are described in EPA NLA publications.  Basic measurement data were added to Neotoma; flags and tags were not.  Data and documentation are available from the EPA website https://www.epa.gov/national-aquatic-resource-surveys/data-national-aquatic-resource-surveys (as of April 2019).”","Diatom slides for sites in the eastern USA are in the Diatom Herbarium at the Academy of Natural Sciences in Philadelphia.","“Water chemistry samples were collected from the epilimnion using a 2 m PVC tube.  Sampling depth was 2 m unless euphotic zone (Secchi disk depth x 2) was &lt;2m, in which case, water sampling depth was equal to the euphotic zone.  All methods are described in EPA NLA publications.  Basic measurement data were added to Neotoma; flags and tags were not.  Data and documentation are available from the EPA website https://www.epa.gov/national-aquatic-resource-surveys/data-national-aquatic-resource-surveys (as of April 2019).”","Diatom slides for sites in the eastern USA are in the Diatom Herbarium at the Academy of Natural Sciences in Philadelphia.","“Water chemistry samples were collected from the epilimnion using a 2 m PVC tube.  Sampling depth was 2 m unless euphotic zone (Secchi disk depth x 2) was &lt;2m, in which case, water sampling depth was equal to the euphotic zone.  All methods are described in EPA NLA publications.  Basic measurement data were added to Neotoma; flags and tags were not.  Data and documentation are available from the EPA website https://www.epa.gov/national-aquatic-resource-surveys/data-national-aquatic-resource-surveys (as of April 2019).”","“Water chemistry samples were collected from the epilimnion using a 2 m PVC tube.  Sampling depth was 2 m unless euphotic zone (Secchi disk depth x 2) was &lt;2m, in which case, water sampling depth was equal to the euphotic zone.  All methods are described in EPA NLA publications.  Basic measurement data were added to Neotoma; flags and tags were not.  Data and documentation are available from the EPA website https://www.epa.gov/national-aquatic-resource-surveys/data-national-aquatic-resource-surveys (as of April 2019).”","Diatom slides for sites in the eastern USA are in the Diatom Herbarium at the Academy of Natural Sciences in Philadelphia.","“Water chemistry samples were collected from the epilimnion using a 2 m PVC tube.  Sampling depth was 2 m unless euphotic zone (Secchi disk depth x 2) was &lt;2m, in which case, water sampling depth was equal to the euphotic zone.  All methods are described in EPA NLA publications.  Basic measurement data were added to Neotoma; flags and tags were not.  Data and documentation are available from the EPA website https://www.epa.gov/national-aquatic-resource-surveys/data-national-aquatic-resource-surveys (as of April 2019).”","“Water chemistry samples were collected from the epilimnion using a 2 m PVC tube.  Sampling depth was 2 m unless euphotic zone (Secchi disk depth x 2) was &lt;2m, in which case, water sampling depth was equal to the euphotic zone.  All methods are described in EPA NLA publications.  Basic measurement data were added to Neotoma; flags and tags were not.  Data and documentation are available from the EPA website https://www.epa.gov/national-aquatic-resource-surveys/data-national-aquatic-resource-surveys (as of April 2019).”","Diatom slides for sites in the eastern USA are in the Diatom Herbarium at the Academy of Natural Sciences in Philadelphia.","“Water chemistry samples were collected from the epilimnion using a 2 m PVC tube.  Sampling depth was 2 m unless euphotic zone (Secchi disk depth x 2) was &lt;2m, in which case, water sampling depth was equal to the euphotic zone.  All methods are described in EPA NLA publications.  Basic measurement data were added to Neotoma; flags and tags were not.  Data and documentation are available from the EPA website https://www.epa.gov/national-aquatic-resource-surveys/data-national-aquatic-resource-surveys (as of April 2019).”","“Water chemistry samples were collected from the epilimnion using a 2 m PVC tube.  Sampling depth was 2 m unless euphotic zone (Secchi disk depth x 2) was &lt;2m, in which case, water sampling depth was equal to the euphotic zone.  All methods are described in EPA NLA publications.  Basic measurement data were added to Neotoma; flags and tags were not.  Data and documentation are available from the EPA website https://www.epa.gov/national-aquatic-resource-surveys/data-national-aquatic-resource-surveys (as of April 2019).”","“Water chemistry samples were collected from the epilimnion using a 2 m PVC tube.  Sampling depth was 2 m unless euphotic zone (Secchi disk depth x 2) was &lt;2m, in which case, water sampling depth was equal to the euphotic zone.  All methods are described in EPA NLA publications.  Basic measurement data were added to Neotoma; flags and tags were not.  Data and documentation are available from the EPA website https://www.epa.gov/national-aquatic-resource-surveys/data-national-aquatic-resource-surveys (as of April 2019).”","Diatom microscope slides are in the Diatom Herbarium of the Academy of Natural Sciences in Philadelphia.","“Water chemistry samples were collected from the epilimnion using a 2 m PVC tube.  Sampling depth was 2 m unless euphotic zone (Secchi disk depth x 2) was &lt;2m, in which case, water sampling depth was equal to the euphotic zone.  All methods are described in EPA NLA publications.  Basic measurement data were added to Neotoma; flags and tags were not.  Data and documentation are available from the EPA website https://www.epa.gov/national-aquatic-resource-surveys/data-national-aquatic-resource-surveys (as of April 2019).”","Diatom microscope slides are in the Diatom Herbarium of the Academy of Natural Sciences in Philadelphia.","“Water chemistry samples were collected from the epilimnion using a 2 m PVC tube.  Sampling depth was 2 m unless euphotic zone (Secchi disk depth x 2) was &lt;2m, in which case, water sampling depth was equal to the euphotic zone.  All methods are described in EPA NLA publications.  Basic measurement data were added to Neotoma; flags and tags were not.  Data and documentation are available from the EPA website https://www.epa.gov/national-aquatic-resource-surveys/data-national-aquatic-resource-surveys (as of April 2019).”","“Water chemistry samples were collected from the epilimnion using a 2 m PVC tube.  Sampling depth was 2 m unless euphotic zone (Secchi disk depth x 2) was &lt;2m, in which case, water sampling depth was equal to the euphotic zone.  All methods are described in EPA NLA publications.  Basic measurement data were added to Neotoma; flags and tags were not.  Data and documentation are available from the EPA website https://www.epa.gov/national-aquatic-resource-surveys/data-national-aquatic-resource-surveys (as of April 2019).”","Diatom microscope slides are in the Diatom Herbarium of the Academy of Natural Sciences in Philadelphia.","“Water chemistry samples were collected from the epilimnion using a 2 m PVC tube.  Sampling depth was 2 m unless euphotic zone (Secchi disk depth x 2) was &lt;2m, in which case, water sampling depth was equal to the euphotic zone.  All methods are described in EPA NLA publications.  Basic measurement data were added to Neotoma; flags and tags were not.  Data and documentation are available from the EPA website https://www.epa.gov/national-aquatic-resource-surveys/data-national-aquatic-resource-surveys (as of April 2019).”","“Water chemistry samples were collected from the epilimnion using a 2 m PVC tube.  Sampling depth was 2 m unless euphotic zone (Secchi disk depth x 2) was &lt;2m, in which case, water sampling depth was equal to the euphotic zone.  All methods are described in EPA NLA publications.  Basic measurement data were added to Neotoma; flags and tags were not.  Data and documentation are available from the EPA website https://www.epa.gov/national-aquatic-resource-surveys/data-national-aquatic-resource-surveys (as of April 2019).”","Diatom microscope slides are in the Diatom Herbarium of the Academy of Natural Sciences in Philadelphia.","“Water chemistry samples were collected from the epilimnion using a 2 m PVC tube.  Sampling depth was 2 m unless euphotic zone (Secchi disk depth x 2) was &lt;2m, in which case, water sampling depth was equal to the euphotic zone.  All methods are described in EPA NLA publications.  Basic measurement data were added to Neotoma; flags and tags were not.  Data and documentation are available from the EPA website https://www.epa.gov/national-aquatic-resource-surveys/data-national-aquatic-resource-surveys (as of April 2019).”","“Water chemistry samples were collected from the epilimnion using a 2 m PVC tube.  Sampling depth was 2 m unless euphotic zone (Secchi disk depth x 2) was &lt;2m, in which case, water sampling depth was equal to the euphotic zone.  All methods are described in EPA NLA publications.  Basic measurement data were added to Neotoma; flags and tags were not.  Data and documentation are available from the EPA website https://www.epa.gov/national-aquatic-resource-surveys/data-national-aquatic-resource-surveys (as of April 2019).”","“Water chemistry samples were collected from the epilimnion using a 2 m PVC tube.  Sampling depth was 2 m unless euphotic zone (Secchi disk depth x 2) was &lt;2m, in which case, water sampling depth was equal to the euphotic zone.  All methods are described in EPA NLA publications.  Basic measurement data were added to Neotoma; flags and tags were not.  Data and documentation are available from the EPA website https://www.epa.gov/national-aquatic-resource-surveys/data-national-aquatic-resource-surveys (as of April 2019).”","Diatom microscope slides are in the Diatom Herbarium of the Academy of Natural Sciences in Philadelphia."]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>datasetid<\/th>\n      <th>database<\/th>\n      <th>datasettype<\/th>\n      <th>age_range_old<\/th>\n      <th>age_range_young<\/th>\n      <th>notes<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"scrollX":"100%","dom":"t","columnDefs":[{"className":"dt-right","targets":[1,4,5]},{"orderable":false,"targets":0}],"order":[],"autoWidth":false,"orderClasses":false},"selection":{"mode":"multiple","selected":null,"target":"row","selectable":null}},"evals":[],"jsHooks":[]}</script>
</div>
</div>
</section>
</section>
<section id="section-3" class="level3" data-number="5.4.5">
<h3 data-number="5.4.5" class="anchored" data-anchor-id="section-3"><span class="header-section-number">5.4.5</span> </h3>
<p><strong>Question 4</strong>: How many different kinds of datasets are available at Devil’s Lake, WI? Show both code and answer. Ensure that your code just retrieves datasets for just this single site.</p>
</section>
<section id="filter-records" class="level3 tabset" data-number="5.4.6">
<h3 class="tabset anchored" data-number="5.4.6" data-anchor-id="filter-records"><span class="header-section-number">5.4.6</span> Filter Records</h3>
<p>If we choose to pull in information about only a single dataset type, or if there is additional filtering we want to do before we download the data, we can use the <code>filter()</code> function. For example, if we only want pollen records, and want records with known chronologies, we can filter:</p>
<section id="code-4" class="level4" data-number="5.4.6.1">
<h4 data-number="5.4.6.1" class="anchored" data-anchor-id="code-4"><span class="header-section-number">5.4.6.1</span> Code</h4>
<div class="cell" data-hash="04-neotoma2_cache/html/downloads_16a8e88ca75cacf4c61d7b24d68fef5e">
<div class="sourceCode cell-code" id="cb8"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb8-1"><a href="#cb8-1" aria-hidden="true" tabindex="-1"></a>mich_pollen <span class="ot">&lt;-</span> mich_datasets <span class="sc">%&gt;%</span></span>
<span id="cb8-2"><a href="#cb8-2" aria-hidden="true" tabindex="-1"></a>  neotoma2<span class="sc">::</span><span class="fu">filter</span>(datasettype <span class="sc">==</span> <span class="st">"pollen"</span> <span class="sc">&amp;</span> <span class="sc">!</span><span class="fu">is.na</span>(age_range_young))</span>
<span id="cb8-3"><a href="#cb8-3" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb8-4"><a href="#cb8-4" aria-hidden="true" tabindex="-1"></a>neotoma2<span class="sc">::</span><span class="fu">summary</span>(mich_pollen)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
</section>
<section id="result-4" class="level4" data-number="5.4.6.2">
<h4 data-number="5.4.6.2" class="anchored" data-anchor-id="result-4"><span class="header-section-number">5.4.6.2</span> Result</h4>
<div class="cell" data-hash="04-neotoma2_cache/html/downloadsCode_557c7df89a45b1badefa1091886b0adb">
<div class="cell-output-display">

<div class="datatables html-widget html-fill-item-overflow-hidden html-fill-item" id="htmlwidget-8a9f420dede42e5d61bf" style="width:100%;height:auto;"></div>
<script type="application/json" data-for="htmlwidget-8a9f420dede42e5d61bf">{"x":{"filter":"none","vertical":false,"data":[["1","2","3","4","5","6","7","8","9","10","11","12","13","14","15","16","17","18","19","20","21","22","23","24","25","26","27"],[294,360,479,518,661,785,952,1100,1540,2864,2954,8559,10003,10129,10239,10242,10251,13350,16313,28059,28100,28101,28113,821,1541,2814,13582],["Bondi Section","Chippewa Bog","Clear Lake","Cub Lake","Demont Lake","Lake Erie","Green Lake","Hudson Lake","Lake Sixteen","Wintergreen Lake","Pretty Lake","Irwin Smith Bog","Appleman Lake","Pinhook Bog","Cowden Lake","Hicks Lake","Morrison Lake","Spicer Lake","Otter Lake","Huffman Lake","Clifford Lake","O'Brien Lake","Horseshoe Lake","Frains Lake","Lake 27","Vestaburg Bog","Minden Bog"],["BONDI","CHIPPEWA","CLEARLK","CUB2","DEMONTLK","ERIE","GREENLK","HUDSONLK","LAKE16","WINTERGR","PRETTYLK","IRWINSMT","APPLEMAN","PINHOOK","COWDEN","HICKSMI","MORRIMI","SPIC06A1","OTTERMI","HUFFMI","CLIFFMI","OBRIENMI","HORSEMI","FRAINSLK","LAKE27","VESTABRG","MINDEN"],[0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0],[1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1],["pollen","pollen","pollen","pollen","pollen","pollen","pollen","pollen","pollen","pollen","pollen","pollen","pollen","pollen","pollen","pollen","pollen","pollen","pollen","pollen","pollen","pollen","pollen","pollen","pollen","pollen","pollen"]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>siteid<\/th>\n      <th>sitename<\/th>\n      <th>collectionunit<\/th>\n      <th>chronolgies<\/th>\n      <th>datasets<\/th>\n      <th>types<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"scrollX":"100%","dom":"t","columnDefs":[{"className":"dt-right","targets":[1,4,5]},{"orderable":false,"targets":0}],"order":[],"autoWidth":false,"orderClasses":false},"selection":{"mode":"multiple","selected":null,"target":"row","selectable":null}},"evals":[],"jsHooks":[]}</script>
</div>
</div>
</section>
</section>
<section id="section-4" class="level3" data-number="5.4.7">
<h3 data-number="5.4.7" class="anchored" data-anchor-id="section-4"><span class="header-section-number">5.4.7</span> </h3>
<p>Note, that we are filtering on two conditions. (You may want to look up the operators being used in the above code: <code>==</code>, <code>&amp;</code>, and <code>!</code> to understand what they accomplish in the code.) We can see now that the data table looks different, and there are fewer total sites.</p>
</section>
<section id="retrieving-sample-data." class="level3" data-number="5.4.8">
<h3 data-number="5.4.8" class="anchored" data-anchor-id="retrieving-sample-data."><span class="header-section-number">5.4.8</span> Retrieving <code>sample()</code> data.</h3>
<p>The sample data are the actual data that scientists usually want - counts of pollen grains, lists of vertebrate fossil occurrences, etc. Because sample data can have fairly large data volumes (each dataset may contain many samples), which can strain server bandwidth and local computing memory, we try to call <code>get_downloads()</code> <em>after</em> we’ve done our preliminary filtering. After <code>get_datasets()</code>, you have enough information to filter based on location, time bounds, and dataset type. When we move to <code>get_downloads()</code> we can do more fine-tuned filtering at the analysis unit or taxon level.</p>
<p>The following command may take a few moments to run. (If it takes too long, we have stored an already-downloaded version of the function output as an RDS data file that you can load directly into R.)</p>
<div class="cell" data-hash="04-neotoma2_cache/html/taxa_40914575e2cb3d33c8f24195b1ec7247">
<div class="sourceCode cell-code" id="cb9"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb9-1"><a href="#cb9-1" aria-hidden="true" tabindex="-1"></a>mich_dl <span class="ot">&lt;-</span> mich_pollen <span class="sc">%&gt;%</span> <span class="fu">get_downloads</span>(<span class="at">all_data =</span> <span class="cn">TRUE</span>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stdout">
<pre><code>...........................</code></pre>
</div>
<div class="sourceCode cell-code" id="cb11"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb11-1"><a href="#cb11-1" aria-hidden="true" tabindex="-1"></a><span class="co"># mich_dl &lt;- readRDS('data/mich_dl.rds')</span></span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
<p>Once we’ve downloaded the sample data, we now have information for each site about all the associated collection units, the datasets, and, for each dataset, all the samples associated with the datasets. To extract all the samples we can call:</p>
<div class="cell" data-hash="04-neotoma2_cache/html/allSamples_08104614278673efe0c15b00a3a8c236">
<div class="sourceCode cell-code" id="cb12"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb12-1"><a href="#cb12-1" aria-hidden="true" tabindex="-1"></a>allSamp <span class="ot">&lt;-</span> <span class="fu">samples</span>(mich_dl)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
<p>When we’ve done this, we get a <code>data.frame</code> that is 38062 rows long and 37 columns wide. The reason the table is so wide is that we are returning data in a <strong>long</strong> format. Each row contains all the information you should need to properly interpret it:</p>
<div class="cell" data-hash="04-neotoma2_cache/html/colNamesAllSamp_71b52459f9930be69f3ddf500f345d86">
<div class="cell-output cell-output-stdout">
<pre><code> [1] "age"             "agetype"         "ageolder"        "ageyounger"     
 [5] "chronologyid"    "chronologyname"  "units"           "value"          
 [9] "context"         "element"         "taxonid"         "symmetry"       
[13] "taxongroup"      "elementtype"     "variablename"    "ecologicalgroup"
[17] "analysisunitid"  "sampleanalyst"   "sampleid"        "depth"          
[21] "thickness"       "samplename"      "datasetid"       "siteid"         
[25] "sitename"        "lat"             "long"            "area"           
[29] "sitenotes"       "description"     "elev"            "collunitid"     
[33] "database"        "datasettype"     "age_range_old"   "age_range_young"
[37] "datasetnotes"   </code></pre>
</div>
</div>
<p>For some dataset types, or analyses some of these columns may not be needed, however, for other dataset types they may be critically important. To allow the <code>neotoma2</code> package to be as useful as possible for the community we’ve included as many as we can.</p>
<section id="extracting-taxa" class="level4 tabset" data-number="5.4.8.1">
<h4 class="tabset anchored" data-number="5.4.8.1" data-anchor-id="extracting-taxa"><span class="header-section-number">5.4.8.1</span> Extracting Taxa</h4>
<p>If you want to know what taxa we have in a dataset, you can use the helper function <code>taxa()</code> on the sites object. The <code>taxa()</code> function gives us, not only the unique taxa, but two additional columns, <code>sites</code> and <code>samples</code>, that tell us how many sites the taxa appear in, and how many samples the taxa appear in, to help us better understand how common individual taxa are.</p>
<section id="code-5" class="level5" data-number="5.4.8.1.1">
<h5 data-number="5.4.8.1.1" class="anchored" data-anchor-id="code-5"><span class="header-section-number">5.4.8.1.1</span> Code</h5>
<div class="cell" data-hash="04-neotoma2_cache/html/taxa2_3cdf0af50564c779f20ef6bb5f8b636c">
<div class="sourceCode cell-code" id="cb14"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb14-1"><a href="#cb14-1" aria-hidden="true" tabindex="-1"></a>neotomatx <span class="ot">&lt;-</span> neotoma2<span class="sc">::</span><span class="fu">taxa</span>(mich_dl)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
</section>
<section id="results" class="level5" data-number="5.4.8.1.2">
<h5 data-number="5.4.8.1.2" class="anchored" data-anchor-id="results"><span class="header-section-number">5.4.8.1.2</span> Results</h5>
<div class="cell" data-hash="04-neotoma2_cache/html/taxaprint_0b3c925518d1bcc07ac46c099cc485eb">
<div class="cell-output-display">

<div class="datatables html-widget html-fill-item-overflow-hidden html-fill-item" id="htmlwidget-086eeea0d697e445d9db" style="width:100%;height:auto;"></div>
<script type="application/json" data-for="htmlwidget-086eeea0d697e445d9db">{"x":{"filter":"none","vertical":false,"data":[["g","grains/ml","grains/tablet","ml","NISP","NISP","NISP","NISP","NISP","NISP","NISP","NISP","NISP","NISP","NISP","NISP","NISP","NISP","NISP","NISP"],[null,null,null,null,"anachronic",null,null,null,null,null,null,null,null,null,null,null,null,null,null,null],["dry mass examined","counted","concentration","volume","pollen/spore","colony","pollen","pollen","pollen","pollen","pollen","pollen","pollen","pollen","pollen","pollen","pollen","pollen","pollen","pollen"],[276,324,930,276,313,202,1,3,4,5,6,7,9,16,20,21,23,25,29,34],[null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null],["Laboratory analyses","Laboratory analyses","Laboratory analyses","Laboratory analyses","Unidentified palynomorphs","Algae","Vascular plants","Vascular plants","Vascular plants","Vascular plants","Vascular plants","Vascular plants","Vascular plants","Vascular plants","Vascular plants","Vascular plants","Vascular plants","Vascular plants","Vascular plants","Vascular plants"],["dry mass examined","counted","concentration","volume","pollen/spore","colony","pollen","pollen","pollen","pollen","pollen","pollen","pollen","pollen","pollen","pollen","pollen","pollen","pollen","pollen"],["Sample quantity","Microspheres","Lycopodium tablets","Sample quantity","Unknown","Pediastrum","Abies","Acer saccharum-type","Acer negundo","Acer pensylvanicum","Acer rubrum","Acer saccharinum-type","Acer undiff.","Ambrosia-type","Spermatophyta undiff. (aquatics)","Aquifoliaceae","Arceuthobium","Artemisia","Betula","Brasenia schreberi"],["LABO","LABO","LABO","LABO","UNID","ALGA","TRSH","TRSH","TRSH","TRSH","TRSH","TRSH","TRSH","UPHE","AQVP","TRSH","TRSH","UPHE","TRSH","AQVP"],[1,1,2,3,1,2,24,1,2,1,14,2,6,12,1,1,2,27,27,2],[15,144,116,261,15,52,355,20,75,7,214,41,180,567,24,2,40,939,1415,14]],"container":"<table class=\"c(&quot;g&quot;, &quot;grains/ml&quot;, &quot;grains/tablet&quot;, &quot;ml&quot;, &quot;NISP&quot;, &quot;NISP&quot;, &quot;NISP&quot;, &quot;NISP&quot;, &quot;NISP&quot;, &quot;NISP&quot;, &quot;NISP&quot;, &quot;NISP&quot;, &quot;NISP&quot;, &quot;NISP&quot;, &quot;NISP&quot;, &quot;NISP&quot;, &quot;NISP&quot;, &quot;NISP&quot;, &quot;NISP&quot;, &quot;NISP&quot;, &quot;NISP&quot;, &quot;NISP&quot;, &quot;NISP&quot;, &quot;NISP&quot;, &quot;NISP&quot;, &quot;NISP&quot;, &quot;NISP&quot;, &quot;NISP&quot;, &quot;NISP&quot;, &quot;NISP&quot;, &quot;NISP&quot;, &quot;NISP&quot;, &quot;NISP&quot;, &quot;NISP&quot;, &quot;NISP&quot;, &quot;NISP&quot;, &quot;NISP&quot;, &quot;NISP&quot;, &quot;NISP&quot;, &quot;NISP&quot;, &quot;NISP&quot;, &quot;NISP&quot;, &quot;NISP&quot;, &quot;NISP&quot;, &quot;NISP&quot;, &quot;NISP&quot;, &quot;NISP&quot;, &quot;NISP&quot;, &quot;NISP&quot;, &quot;NISP&quot;, &quot;NISP&quot;, &quot;NISP&quot;, &quot;NISP&quot;, &quot;NISP&quot;, &quot;NISP&quot;, &quot;NISP&quot;, &quot;NISP&quot;, &quot;NISP&quot;, &quot;NISP&quot;, &quot;NISP&quot;, &quot;NISP&quot;, &quot;NISP&quot;, &#10;&quot;NISP&quot;, &quot;NISP&quot;, &quot;NISP&quot;, &quot;NISP&quot;, &quot;NISP&quot;, &quot;NISP&quot;, &quot;NISP&quot;, &quot;NISP&quot;, &quot;NISP&quot;, &quot;NISP&quot;, &quot;NISP&quot;, &quot;NISP&quot;, &quot;NISP&quot;, &quot;NISP&quot;, &quot;NISP&quot;, &quot;NISP&quot;, &quot;NISP&quot;, &quot;NISP&quot;, &quot;NISP&quot;, &quot;NISP&quot;, &quot;NISP&quot;, &quot;NISP&quot;, &quot;NISP&quot;, &quot;NISP&quot;, &quot;NISP&quot;, &quot;NISP&quot;, &quot;NISP&quot;, &quot;NISP&quot;, &quot;NISP&quot;, &quot;NISP&quot;, &quot;NISP&quot;, &quot;NISP&quot;, &quot;NISP&quot;, &quot;NISP&quot;, &quot;NISP&quot;, &quot;NISP&quot;, &quot;NISP&quot;, &quot;NISP&quot;, &quot;NISP&quot;, &quot;NISP&quot;, &quot;NISP&quot;, &quot;NISP&quot;, &quot;NISP&quot;, &quot;NISP&quot;, &quot;NISP&quot;, &quot;NISP&quot;, &quot;NISP&quot;, &quot;NISP&quot;, &quot;NISP&quot;, &quot;NISP&quot;, &quot;NISP&quot;, &quot;NISP&quot;, &quot;NISP&quot;, &quot;NISP&quot;, &quot;NISP&quot;, &quot;NISP&quot;, &quot;NISP&quot;, &quot;NISP&quot;, &quot;NISP&quot;, &quot;NISP&quot;, &quot;NISP&quot;, &quot;NISP&quot;, &quot;NISP&quot;, &#10;&quot;NISP&quot;, &quot;NISP&quot;, &quot;NISP&quot;, &quot;NISP&quot;, &quot;NISP&quot;, &quot;NISP&quot;, &quot;NISP&quot;, &quot;NISP&quot;, &quot;NISP&quot;, &quot;NISP&quot;, &quot;NISP&quot;, &quot;NISP&quot;, &quot;NISP&quot;, &quot;NISP&quot;, &quot;NISP&quot;, &quot;NISP&quot;, &quot;NISP&quot;, &quot;NISP&quot;, &quot;NISP&quot;, &quot;NISP&quot;, &quot;NISP&quot;, &quot;NISP&quot;, &quot;NISP&quot;, &quot;NISP&quot;, &quot;NISP&quot;, &quot;NISP&quot;, &quot;NISP&quot;, &quot;NISP&quot;, &quot;NISP&quot;, &quot;NISP&quot;, &quot;NISP&quot;, &quot;NISP&quot;, &quot;NISP&quot;, &quot;NISP&quot;, &quot;NISP&quot;, &quot;NISP&quot;, &quot;NISP&quot;, &quot;NISP&quot;, &quot;NISP&quot;, &quot;NISP&quot;, &quot;NISP&quot;, &quot;NISP&quot;, &quot;NISP&quot;, &quot;NISP&quot;, &quot;NISP&quot;, &quot;NISP&quot;, &quot;NISP&quot;, &quot;NISP&quot;, &quot;NISP&quot;, &quot;NISP&quot;, &quot;NISP&quot;, &quot;NISP&quot;, &quot;NISP&quot;, &quot;NISP&quot;, &quot;NISP&quot;, &quot;NISP&quot;, &quot;NISP&quot;, &quot;NISP&quot;, &quot;NISP&quot;, &quot;NISP&quot;, &quot;NISP&quot;, &quot;NISP&quot;, &quot;NISP&quot;, &#10;&quot;NISP&quot;, &quot;NISP&quot;, &quot;NISP&quot;, &quot;NISP&quot;, &quot;NISP&quot;, &quot;NISP&quot;, &quot;NISP&quot;, &quot;NISP&quot;, &quot;NISP&quot;, &quot;NISP&quot;, &quot;NISP&quot;, &quot;NISP&quot;, &quot;NISP&quot;, &quot;NISP&quot;, &quot;NISP&quot;, &quot;NISP&quot;, &quot;NISP&quot;, &quot;NISP&quot;, &quot;NISP&quot;, &quot;NISP&quot;, &quot;NISP&quot;, &quot;NISP&quot;, &quot;NISP&quot;, &quot;NISP&quot;, &quot;NISP&quot;, &quot;NISP&quot;, &quot;NISP&quot;, &quot;NISP&quot;, &quot;NISP&quot;, &quot;NISP&quot;, &quot;NISP&quot;, &quot;NISP&quot;, &quot;NISP&quot;, &quot;NISP&quot;, &quot;NISP&quot;, &quot;NISP&quot;, &quot;NISP&quot;, &quot;NISP&quot;, &quot;NISP&quot;, &quot;NISP&quot;, &quot;NISP&quot;, &quot;NISP&quot;, &quot;NISP&quot;, &quot;NISP&quot;, &quot;NISP&quot;, &quot;NISP&quot;, &quot;NISP&quot;, &quot;NISP&quot;, &quot;NISP&quot;, &quot;NISP&quot;, &quot;NISP&quot;, &quot;NISP&quot;, &quot;NISP&quot;, &quot;NISP&quot;, &quot;NISP&quot;, &quot;NISP&quot;, &quot;NISP&quot;, &quot;NISP&quot;, &quot;NISP&quot;, &quot;NISP&quot;, &quot;NISP&quot;, &quot;number&quot;, &quot;number&quot;, &#10;&quot;number&quot;, &quot;number&quot;, &quot;number&quot;) c(NA, NA, NA, NA, &quot;anachronic&quot;, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, &#10;NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, &#10;NA, NA, NA, NA, NA) c(&quot;dry mass examined&quot;, &quot;counted&quot;, &quot;concentration&quot;, &quot;volume&quot;, &quot;pollen/spore&quot;, &quot;colony&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &#10;&quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &#10;&quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &#10;&quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &#10;&quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen/spore&quot;, &quot;pollen/spore&quot;, &quot;spore&quot;, &quot;spore&quot;, &quot;spore&quot;, &quot;spore&quot;, &quot;spore&quot;, &quot;spore&quot;, &quot;spore&quot;, &quot;spore&quot;, &quot;spore&quot;, &quot;spore&quot;, &quot;spore&quot;, &quot;spore&quot;, &quot;spore&quot;, &quot;spore&quot;, &quot;spore&quot;, &quot;spore&quot;, &quot;spore&quot;, &quot;spore&quot;, &quot;spore&quot;, &quot;spore&quot;, &quot;spore&quot;, &quot;spore&quot;, &quot;spore&quot;, &quot;spore&quot;, &quot;spore&quot;, &quot;spore&quot;, &quot;spore&quot;, &quot;spore&quot;, &quot;spore&quot;, &quot;spore&quot;, &quot;spore&quot;, &quot;spore&quot;, &quot;counted&quot;, &quot;counted&quot;, &quot;counted&quot;, &quot;fragment&quot;, &#10;&quot;quantity added&quot;) c(276, 324, 930, 276, 313, 202, 1, 3, 4, 5, 6, 7, 9, 16, 20, 21, 23, 25, 29, 34, 40, 43, 44, 45, 47, 49, 50, 54, 61, 67, 72, 74, 81, 84, 85, 88, 92, 107, 112, 113, 114, 116, 125, 127, 129, 135, 144, 145, 146, 153, 157, 160, 162, 163, 182, 185, 186, 188, 189, 190, 192, 193, 197, 210, 212, 213, 216, 218, 222, 225, 230, 237, 251, 252, 260, 263, 267, 268, 270, 271, 274, 275, 277, 279, 280, 282, 289, 290, 293, 298, 300, 302, 306, 307, 308, 309, 310, 311, 312, 314, 316, 317, 320, 322, 323, 330, 338, 341, &#10;344, 345, 346, 351, 355, 361, 364, 366, 367, 369, 371, 372, 378, 381, 383, 385, 389, 391, 397, 402, 416, 417, 418, 419, 420, 425, 431, 432, 433, 445, 457, 460, 464, 480, 483, 498, 515, 516, 517, 534, 540, 542, 549, 550, 554, 555, 661, 672, 697, 705, 709, 726, 751, 758, 785, 803, 809, 825, 836, 848, 875, 876, 877, 885, 896, 908, 909, 936, 960, 961, 963, 968, 988, 1082, 1083, 1102, 1108, 1155, 1177, 1237, 1246, 1358, 1571, 1634, 2286, 2317, 2318, 2572, 2588, 2894, 3010, 3171, 3460, 3468, 3606, 3762, &#10;5070, 5218, 5758, 7651, 7655, 9115, 9505, 9509, 9520, 20932, 46115, 312, 313, 33, 83, 91, 108, 139, 168, 170, 171, 173, 234, 247, 286, 294, 339, 348, 505, 508, 509, 613, 648, 659, 677, 750, 807, 835, 1091, 1433, 1489, 2027, 3717, 5011, 29212, 95, 932, 1305, 1062, 930) c(NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, &#10;NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, &#10;NA, NA, NA) c(&quot;Laboratory analyses&quot;, &quot;Laboratory analyses&quot;, &quot;Laboratory analyses&quot;, &quot;Laboratory analyses&quot;, &quot;Unidentified palynomorphs&quot;, &quot;Algae&quot;, &quot;Vascular plants&quot;, &quot;Vascular plants&quot;, &quot;Vascular plants&quot;, &quot;Vascular plants&quot;, &quot;Vascular plants&quot;, &quot;Vascular plants&quot;, &quot;Vascular plants&quot;, &quot;Vascular plants&quot;, &quot;Vascular plants&quot;, &quot;Vascular plants&quot;, &quot;Vascular plants&quot;, &quot;Vascular plants&quot;, &quot;Vascular plants&quot;, &quot;Vascular plants&quot;, &quot;Vascular plants&quot;, &quot;Vascular plants&quot;, &quot;Vascular plants&quot;, &quot;Vascular plants&quot;, &quot;Vascular plants&quot;, &quot;Vascular plants&quot;, &#10;&quot;Vascular plants&quot;, &quot;Vascular plants&quot;, &quot;Vascular plants&quot;, &quot;Vascular plants&quot;, &quot;Vascular plants&quot;, &quot;Vascular plants&quot;, &quot;Vascular plants&quot;, &quot;Vascular plants&quot;, &quot;Vascular plants&quot;, &quot;Vascular plants&quot;, &quot;Vascular plants&quot;, &quot;Vascular plants&quot;, &quot;Vascular plants&quot;, &quot;Vascular plants&quot;, &quot;Vascular plants&quot;, &quot;Vascular plants&quot;, &quot;Vascular plants&quot;, &quot;Vascular plants&quot;, &quot;Vascular plants&quot;, &quot;Vascular plants&quot;, &quot;Vascular plants&quot;, &quot;Vascular plants&quot;, &quot;Vascular plants&quot;, &quot;Vascular plants&quot;, &quot;Vascular plants&quot;, &quot;Vascular plants&quot;, &quot;Vascular plants&quot;, &#10;&quot;Vascular plants&quot;, &quot;Vascular plants&quot;, &quot;Vascular plants&quot;, &quot;Vascular plants&quot;, &quot;Vascular plants&quot;, &quot;Vascular plants&quot;, &quot;Vascular plants&quot;, &quot;Vascular plants&quot;, &quot;Vascular plants&quot;, &quot;Vascular plants&quot;, &quot;Vascular plants&quot;, &quot;Vascular plants&quot;, &quot;Vascular plants&quot;, &quot;Vascular plants&quot;, &quot;Vascular plants&quot;, &quot;Vascular plants&quot;, &quot;Vascular plants&quot;, &quot;Vascular plants&quot;, &quot;Vascular plants&quot;, &quot;Vascular plants&quot;, &quot;Vascular plants&quot;, &quot;Vascular plants&quot;, &quot;Vascular plants&quot;, &quot;Vascular plants&quot;, &quot;Vascular plants&quot;, &quot;Vascular plants&quot;, &quot;Vascular plants&quot;, &#10;&quot;Vascular plants&quot;, &quot;Vascular plants&quot;, &quot;Vascular plants&quot;, &quot;Vascular plants&quot;, &quot;Vascular plants&quot;, &quot;Vascular plants&quot;, &quot;Vascular plants&quot;, &quot;Vascular plants&quot;, &quot;Vascular plants&quot;, &quot;Vascular plants&quot;, &quot;Vascular plants&quot;, &quot;Vascular plants&quot;, &quot;Vascular plants&quot;, &quot;Vascular plants&quot;, &quot;Vascular plants&quot;, &quot;Vascular plants&quot;, &quot;Vascular plants&quot;, &quot;Vascular plants&quot;, &quot;Unidentified palynomorphs&quot;, &quot;Vascular plants&quot;, &quot;Vascular plants&quot;, &quot;Vascular plants&quot;, &quot;Vascular plants&quot;, &quot;Vascular plants&quot;, &quot;Vascular plants&quot;, &quot;Vascular plants&quot;, &#10;&quot;Vascular plants&quot;, &quot;Vascular plants&quot;, &quot;Vascular plants&quot;, &quot;Vascular plants&quot;, &quot;Vascular plants&quot;, &quot;Vascular plants&quot;, &quot;Vascular plants&quot;, &quot;Vascular plants&quot;, &quot;Vascular plants&quot;, &quot;Vascular plants&quot;, &quot;Vascular plants&quot;, &quot;Vascular plants&quot;, &quot;Vascular plants&quot;, &quot;Vascular plants&quot;, &quot;Vascular plants&quot;, &quot;Vascular plants&quot;, &quot;Vascular plants&quot;, &quot;Vascular plants&quot;, &quot;Vascular plants&quot;, &quot;Vascular plants&quot;, &quot;Vascular plants&quot;, &quot;Vascular plants&quot;, &quot;Vascular plants&quot;, &quot;Vascular plants&quot;, &quot;Vascular plants&quot;, &quot;Vascular plants&quot;, &quot;Vascular plants&quot;, &#10;&quot;Vascular plants&quot;, &quot;Vascular plants&quot;, &quot;Vascular plants&quot;, &quot;Vascular plants&quot;, &quot;Vascular plants&quot;, &quot;Vascular plants&quot;, &quot;Vascular plants&quot;, &quot;Vascular plants&quot;, &quot;Vascular plants&quot;, &quot;Vascular plants&quot;, &quot;Vascular plants&quot;, &quot;Vascular plants&quot;, &quot;Vascular plants&quot;, &quot;Vascular plants&quot;, &quot;Vascular plants&quot;, &quot;Vascular plants&quot;, &quot;Vascular plants&quot;, &quot;Vascular plants&quot;, &quot;Vascular plants&quot;, &quot;Vascular plants&quot;, &quot;Vascular plants&quot;, &quot;Vascular plants&quot;, &quot;Vascular plants&quot;, &quot;Vascular plants&quot;, &quot;Vascular plants&quot;, &quot;Vascular plants&quot;, &quot;Vascular plants&quot;, &#10;&quot;Vascular plants&quot;, &quot;Vascular plants&quot;, &quot;Vascular plants&quot;, &quot;Vascular plants&quot;, &quot;Vascular plants&quot;, &quot;Vascular plants&quot;, &quot;Vascular plants&quot;, &quot;Vascular plants&quot;, &quot;Vascular plants&quot;, &quot;Vascular plants&quot;, &quot;Vascular plants&quot;, &quot;Vascular plants&quot;, &quot;Vascular plants&quot;, &quot;Vascular plants&quot;, &quot;Vascular plants&quot;, &quot;Vascular plants&quot;, &quot;Vascular plants&quot;, &quot;Vascular plants&quot;, &quot;Vascular plants&quot;, &quot;Vascular plants&quot;, &quot;Vascular plants&quot;, &quot;Vascular plants&quot;, &quot;Vascular plants&quot;, &quot;Vascular plants&quot;, &quot;Vascular plants&quot;, &quot;Vascular plants&quot;, &quot;Vascular plants&quot;, &#10;&quot;Vascular plants&quot;, &quot;Vascular plants&quot;, &quot;Vascular plants&quot;, &quot;Vascular plants&quot;, &quot;Vascular plants&quot;, &quot;Vascular plants&quot;, &quot;Vascular plants&quot;, &quot;Vascular plants&quot;, &quot;Vascular plants&quot;, &quot;Vascular plants&quot;, &quot;Vascular plants&quot;, &quot;Vascular plants&quot;, &quot;Vascular plants&quot;, &quot;Vascular plants&quot;, &quot;Vascular plants&quot;, &quot;Vascular plants&quot;, &quot;Vascular plants&quot;, &quot;Vascular plants&quot;, &quot;Vascular plants&quot;, &quot;Vascular plants&quot;, &quot;Vascular plants&quot;, &quot;Vascular plants&quot;, &quot;Vascular plants&quot;, &quot;Vascular plants&quot;, &quot;Vascular plants&quot;, &quot;Vascular plants&quot;, &quot;Vascular plants&quot;, &#10;&quot;Vascular plants&quot;, &quot;Unidentified palynomorphs&quot;, &quot;Unidentified palynomorphs&quot;, &quot;Vascular plants&quot;, &quot;Vascular plants&quot;, &quot;Vascular plants&quot;, &quot;Vascular plants&quot;, &quot;Vascular plants&quot;, &quot;Vascular plants&quot;, &quot;Vascular plants&quot;, &quot;Vascular plants&quot;, &quot;Vascular plants&quot;, &quot;Vascular plants&quot;, &quot;Vascular plants&quot;, &quot;Vascular plants&quot;, &quot;Bryophytes&quot;, &quot;Unidentified palynomorphs&quot;, &quot;Unidentified palynomorphs&quot;, &quot;Vascular plants&quot;, &quot;Vascular plants&quot;, &quot;Vascular plants&quot;, &quot;Vascular plants&quot;, &quot;Vascular plants&quot;, &quot;Bryophytes&quot;, &quot;Vascular plants&quot;, &#10;&quot;Vascular plants&quot;, &quot;Vascular plants&quot;, &quot;Vascular plants&quot;, &quot;Vascular plants&quot;, &quot;Fungi&quot;, &quot;Fungi&quot;, &quot;Vascular plants&quot;, &quot;Vascular plants&quot;, &quot;Vascular plants&quot;, &quot;Vascular plants&quot;, &quot;Laboratory analyses&quot;, &quot;Laboratory analyses&quot;, &quot;Laboratory analyses&quot;, &quot;Charcoal&quot;, &quot;Laboratory analyses&quot;) c(&quot;dry mass examined&quot;, &quot;counted&quot;, &quot;concentration&quot;, &quot;volume&quot;, &quot;pollen/spore&quot;, &quot;colony&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &#10;&quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &#10;&quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &#10;&quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &#10;&quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen&quot;, &quot;pollen/spore&quot;, &quot;pollen/spore&quot;, &quot;spore&quot;, &quot;spore&quot;, &quot;spore&quot;, &quot;spore&quot;, &quot;spore&quot;, &quot;spore&quot;, &quot;spore&quot;, &quot;spore&quot;, &quot;spore&quot;, &quot;spore&quot;, &quot;spore&quot;, &quot;spore&quot;, &quot;spore&quot;, &quot;spore&quot;, &quot;spore&quot;, &quot;spore&quot;, &quot;spore&quot;, &quot;spore&quot;, &quot;spore&quot;, &quot;spore&quot;, &quot;spore&quot;, &quot;spore&quot;, &quot;spore&quot;, &quot;spore&quot;, &quot;spore&quot;, &quot;spore&quot;, &quot;spore&quot;, &quot;spore&quot;, &quot;spore&quot;, &quot;spore&quot;, &quot;spore&quot;, &quot;spore&quot;, &quot;counted&quot;, &quot;counted&quot;, &quot;counted&quot;, &quot;fragment&quot;, &#10;&quot;quantity added&quot;) c(&quot;Sample quantity&quot;, &quot;Microspheres&quot;, &quot;Lycopodium tablets&quot;, &quot;Sample quantity&quot;, &quot;Unknown&quot;, &quot;Pediastrum&quot;, &quot;Abies&quot;, &quot;Acer saccharum-type&quot;, &quot;Acer negundo&quot;, &quot;Acer pensylvanicum&quot;, &quot;Acer rubrum&quot;, &quot;Acer saccharinum-type&quot;, &quot;Acer undiff.&quot;, &quot;Ambrosia-type&quot;, &quot;Spermatophyta undiff. (aquatics)&quot;, &quot;Aquifoliaceae&quot;, &quot;Arceuthobium&quot;, &quot;Artemisia&quot;, &quot;Betula&quot;, &quot;Brasenia schreberi&quot;, &quot;Caltha&quot;, &quot;Caprifoliaceae undiff.&quot;, &quot;Carya&quot;, &quot;Caryophyllaceae&quot;, &quot;Castanea&quot;, &quot;Celtis&quot;, &quot;Cephalanthus&quot;, &quot;Amaranthaceae undiff.&quot;, &quot;Asteraceae undiff.&quot;, &#10;&quot;Corylus&quot;, &quot;Cupressaceae&quot;, &quot;Cyperaceae&quot;, &quot;Drosera&quot;, &quot;Elaeagnus&quot;, &quot;Elaeagnaceae&quot;, &quot;Ephedra viridis-type&quot;, &quot;Ericaceae&quot;, &quot;Fagus&quot;, &quot;Fraxinus nigra-type&quot;, &quot;Fraxinus pennsylvanica-type&quot;, &quot;Fraxinus undiff.&quot;, &quot;Galium&quot;, &quot;Poaceae undiff.&quot;, &quot;Angiospermae undiff. (herbs)&quot;, &quot;Hippuris vulgaris&quot;, &quot;Ilex&quot;, &quot;Juglans cinerea&quot;, &quot;Juglans nigra&quot;, &quot;Juglans undiff.&quot;, &quot;Larix&quot;, &quot;Fabaceae undiff.&quot;, &quot;Cichorioideae&quot;, &quot;Liquidambar&quot;, &quot;Liriodendron&quot;, &quot;Menyanthes trifoliata&quot;, &quot;Moraceae&quot;, &quot;Morus&quot;, &quot;Myriophyllum&quot;, &quot;Nemopanthus&quot;, &quot;Nuphar&quot;, &#10;&quot;Nymphaea&quot;, &quot;Nyssa&quot;, &quot;Ostrya/Carpinus&quot;, &quot;Picea&quot;, &quot;Pinus subg. Pinus&quot;, &quot;Pinus subg. Strobus&quot;, &quot;Pinus undiff.&quot;, &quot;Plantago undiff.&quot;, &quot;Platanus&quot;, &quot;Polygonaceae undiff.&quot;, &quot;Persicaria amphibia&quot;, &quot;Populus undiff.&quot;, &quot;Quercus&quot;, &quot;Ranunculaceae undiff.&quot;, &quot;Rhus&quot;, &quot;Robinia&quot;, &quot;Rubus&quot;, &quot;Rumex&quot;, &quot;Sagittaria&quot;, &quot;Salix&quot;, &quot;Salsola&quot;, &quot;Sambucus&quot;, &quot;Sanguisorba&quot;, &quot;Sarcobatus vermiculatus&quot;, &quot;Sarracenia&quot;, &quot;Saxifragaceae&quot;, &quot;Shepherdia argentea&quot;, &quot;Shepherdia canadensis&quot;, &quot;Sparganium-type&quot;, &quot;Taxodium&quot;, &quot;Thalictrum&quot;, &quot;Tilia&quot;, &#10;&quot;Tsuga&quot;, &quot;Asteroideae undiff.&quot;, &quot;Typha/Sparganium&quot;, &quot;Typha latifolia&quot;, &quot;Ulmus&quot;, &quot;Apiaceae&quot;, &quot;Indeterminable&quot;, &quot;Urtica-type&quot;, &quot;Utricularia&quot;, &quot;Vaccinium&quot;, &quot;Vitis&quot;, &quot;Xanthium&quot;, &quot;Zea mays&quot;, &quot;Alnus&quot;, &quot;Acer&quot;, &quot;Liliaceae&quot;, &quot;Alnus incana&quot;, &quot;Ambrosia&quot;, &quot;Fabaceae&quot;, &quot;Campanulaceae&quot;, &quot;Rosaceae undiff.&quot;, &quot;Cornus&quot;, &quot;Juniperus/Thuja&quot;, &quot;Ephedra&quot;, &quot;Ephedra torreyana-type&quot;, &quot;Fraxinus&quot;, &quot;Fraxinus americana-type&quot;, &quot;Juglans&quot;, &quot;Iva annua&quot;, &quot;Myricaceae&quot;, &quot;Myrica&quot;, &quot;Pinus&quot;, &quot;Populus&quot;, &quot;Amaranthaceae&quot;, &quot;Viburnum&quot;, &quot;Amorpha&quot;, &#10;&quot;Poaceae (Cerealia)&quot;, &quot;Poaceae&quot;, &quot;Asteroideae&quot;, &quot;Brassica&quot;, &quot;Brassicaceae&quot;, &quot;Humulus&quot;, &quot;Dalea&quot;, &quot;Dalea candida&quot;, &quot;Dalea purpurea&quot;, &quot;Onagraceae&quot;, &quot;Cyclachaena xanthiifolia&quot;, &quot;Lamiaceae&quot;, &quot;Urticaceae&quot;, &quot;Plantago&quot;, &quot;Rosaceae&quot;, &quot;Urtica&quot;, &quot;Lemna&quot;, &quot;Stuckenia&quot;, &quot;Potamogeton&quot;, &quot;Cupressaceae undiff.&quot;, &quot;Polygonum&quot;, &quot;Shepherdia&quot;, &quot;Ranunculus&quot;, &quot;Abies balsamea&quot;, &quot;Acer saccharinum&quot;, &quot;Acer saccharum&quot;, &quot;Monotropa&quot;, &quot;Ilex/Nemopanthus&quot;, &quot;Pinus strobus&quot;, &quot;Bistorta vivipara&quot;, &quot;Populus deltoides-type&quot;, &quot;Rhamnaceae/Vitaceae&quot;, &#10;&quot;Typha&quot;, &quot;Tilia americana&quot;, &quot;Rhamnus&quot;, &quot;Acer negundo-type&quot;, &quot;Zanthoxylum&quot;, &quot;Potentilla&quot;, &quot;Pinus banksiana/P. resinosa&quot;, &quot;Ericales&quot;, &quot;Myricaceae undiff.&quot;, &quot;Asteraceae (high spine)&quot;, &quot;Asteraceae (low spine)&quot;, &quot;Pontederia&quot;, &quot;Ranunculaceae (tricolpate) undiff.&quot;, &quot;Sparganium&quot;, &quot;Ericaceae undiff.&quot;, &quot;Oxalis&quot;, &quot;Fraxinus quadrangulata/F. nigra-type&quot;, &quot;Fraxinus pennsylvanica/F. americana-type&quot;, &quot;Iva&quot;, &quot;Triglochin&quot;, &quot;Persicaria maculosa&quot;, &quot;Fraxinus nigra&quot;, &quot;Fraxinus pennsylvanica&quot;, &quot;Fraxinus americana&quot;, &quot;Vitaceae&quot;, &#10;&quot;Populus tremuloides&quot;, &quot;Alismataceae&quot;, &quot;Ostrya&quot;, &quot;Heteranthera&quot;, &quot;Cupressaceae/Taxaceae&quot;, &quot;Eupatorium&quot;, &quot;Oxalidaceae&quot;, &quot;Mentha&quot;, &quot;Picea glauca&quot;, &quot;Picea mariana&quot;, &quot;Typha angustifolia/Sparganium&quot;, &quot;Fagus grandifolia&quot;, &quot;Stachys&quot;, &quot;Spermatophyta undiff. (trees)&quot;, &quot;Persicaria lapathifolia&quot;, &quot;Balsaminaceae&quot;, &quot;Potamogetonaceae&quot;, &quot;Alnus viridis subsp. crispa&quot;, &quot;Alisma-type&quot;, &quot;Chenopodium&quot;, &quot;Suaeda&quot;, &quot;Tradescantia&quot;, &quot;Chenopodiaceae sensu stricto&quot;, &quot;Sarcobatus&quot;, &quot;Cannabaceae&quot;, &quot;Menyanthes&quot;, &quot;Hippuris&quot;, &quot;Polygonum sensu lato&quot;, &#10;&quot;Maclura&quot;, &quot;Ilex verticillata&quot;, &quot;Indeterminable&quot;, &quot;Unknown&quot;, &quot;Botrychium&quot;, &quot;Dryopteris-type&quot;, &quot;Equisetum&quot;, &quot;Polypodiophyta undiff.&quot;, &quot;Isoëtes&quot;, &quot;Lycopodium annotinum&quot;, &quot;Lycopodium clavatum&quot;, &quot;Diphasiastrum complanatum-type&quot;, &quot;Huperzia lucidula&quot;, &quot;Polypodiaceae&quot;, &quot;Pteridium-type&quot;, &quot;Selaginella&quot;, &quot;Sphagnum&quot;, &quot;Unknown (trilete)&quot;, &quot;Unknown (monolete)&quot;, &quot;Lycopodium-type&quot;, &quot;Ophioglossum&quot;, &quot;Osmunda&quot;, &quot;Dryopteris&quot;, &quot;Diphasiastrum complanatum&quot;, &quot;Bryopsida&quot;, &quot;Pteridophyta&quot;, &quot;Selaginella selaginoides&quot;, &quot;Pteridium&quot;, &#10;&quot;Polypodium&quot;, &quot;Osmunda regalis/O. claytoniana&quot;, &quot;Tilletia&quot;, &quot;Sporormiella&quot;, &quot;Pteridophyta undiff.&quot;, &quot;Equisetum-type&quot;, &quot;Lycopodium&quot;, &quot;Osmundastrum cinnamomeum&quot;, &quot;Eucalyptus spike&quot;, &quot;Lycopodium spike&quot;, &quot;Spike&quot;, &quot;Charcoal&quot;, &quot;Lycopodium tablets&quot;) c(&quot;LABO&quot;, &quot;LABO&quot;, &quot;LABO&quot;, &quot;LABO&quot;, &quot;UNID&quot;, &quot;ALGA&quot;, &quot;TRSH&quot;, &quot;TRSH&quot;, &quot;TRSH&quot;, &quot;TRSH&quot;, &quot;TRSH&quot;, &quot;TRSH&quot;, &quot;TRSH&quot;, &quot;UPHE&quot;, &quot;AQVP&quot;, &quot;TRSH&quot;, &quot;TRSH&quot;, &quot;UPHE&quot;, &quot;TRSH&quot;, &quot;AQVP&quot;, &quot;AQVP&quot;, &quot;TRSH&quot;, &quot;TRSH&quot;, &quot;UPHE&quot;, &quot;TRSH&quot;, &quot;TRSH&quot;, &quot;TRSH&quot;, &quot;UPHE&quot;, &quot;UPHE&quot;, &quot;TRSH&quot;, &quot;TRSH&quot;, &quot;UPHE&quot;, &quot;AQVP&quot;, &quot;TRSH&quot;, &quot;TRSH&quot;, &quot;TRSH&quot;, &quot;TRSH&quot;, &quot;TRSH&quot;, &quot;TRSH&quot;, &quot;TRSH&quot;, &quot;TRSH&quot;, &quot;UPHE&quot;, &quot;UPHE&quot;, &quot;UPHE&quot;, &quot;AQVP&quot;, &quot;TRSH&quot;, &quot;TRSH&quot;, &quot;TRSH&quot;, &quot;TRSH&quot;, &quot;TRSH&quot;, &quot;UPHE&quot;, &quot;UPHE&quot;, &quot;TRSH&quot;, &quot;TRSH&quot;, &quot;AQVP&quot;, &quot;TRSH&quot;, &quot;TRSH&quot;, &quot;AQVP&quot;, &quot;TRSH&quot;, &quot;AQVP&quot;, &quot;AQVP&quot;, &quot;TRSH&quot;, &quot;TRSH&quot;, &#10;&quot;TRSH&quot;, &quot;TRSH&quot;, &quot;TRSH&quot;, &quot;TRSH&quot;, &quot;UPHE&quot;, &quot;TRSH&quot;, &quot;UPHE&quot;, &quot;UPHE&quot;, &quot;TRSH&quot;, &quot;TRSH&quot;, &quot;UPHE&quot;, &quot;TRSH&quot;, &quot;TRSH&quot;, &quot;TRSH&quot;, &quot;UPHE&quot;, &quot;AQVP&quot;, &quot;TRSH&quot;, &quot;UPHE&quot;, &quot;TRSH&quot;, &quot;UPHE&quot;, &quot;TRSH&quot;, &quot;AQVP&quot;, &quot;UPHE&quot;, &quot;TRSH&quot;, &quot;TRSH&quot;, &quot;AQVP&quot;, &quot;TRSH&quot;, &quot;UPHE&quot;, &quot;TRSH&quot;, &quot;TRSH&quot;, &quot;UPHE&quot;, &quot;AQVP&quot;, &quot;AQVP&quot;, &quot;TRSH&quot;, &quot;UPHE&quot;, &quot;UNID&quot;, &quot;UPHE&quot;, &quot;AQVP&quot;, &quot;TRSH&quot;, &quot;TRSH&quot;, &quot;UPHE&quot;, &quot;UPHE&quot;, &quot;TRSH&quot;, &quot;TRSH&quot;, &quot;UPHE&quot;, &quot;TRSH&quot;, &quot;UPHE&quot;, &quot;UPHE&quot;, &quot;UPHE&quot;, &quot;UPHE&quot;, &quot;TRSH&quot;, &quot;TRSH&quot;, &quot;TRSH&quot;, &quot;TRSH&quot;, &quot;TRSH&quot;, &quot;TRSH&quot;, &quot;TRSH&quot;, &quot;UPHE&quot;, &quot;TRSH&quot;, &quot;TRSH&quot;, &quot;TRSH&quot;, &quot;TRSH&quot;, &quot;UPHE&quot;, &#10;&quot;TRSH&quot;, &quot;UPHE&quot;, &quot;UPHE&quot;, &quot;UPHE&quot;, &quot;UPHE&quot;, &quot;UPHE&quot;, &quot;UPHE&quot;, &quot;UPHE&quot;, &quot;UPHE&quot;, &quot;UPHE&quot;, &quot;UPHE&quot;, &quot;UPHE&quot;, &quot;UPHE&quot;, &quot;UPHE&quot;, &quot;UPHE&quot;, &quot;UPHE&quot;, &quot;UPHE&quot;, &quot;UPHE&quot;, &quot;AQVP&quot;, &quot;AQVP&quot;, &quot;AQVP&quot;, &quot;TRSH&quot;, &quot;UPHE&quot;, &quot;TRSH&quot;, &quot;UPHE&quot;, &quot;TRSH&quot;, &quot;TRSH&quot;, &quot;TRSH&quot;, &quot;UPHE&quot;, &quot;TRSH&quot;, &quot;TRSH&quot;, &quot;UPHE&quot;, &quot;TRSH&quot;, &quot;TRSH&quot;, &quot;AQVP&quot;, &quot;TRSH&quot;, &quot;TRSH&quot;, &quot;TRSH&quot;, &quot;TRSH&quot;, &quot;UPHE&quot;, &quot;TRSH&quot;, &quot;TRSH&quot;, &quot;TRSH&quot;, &quot;UPHE&quot;, &quot;UPHE&quot;, &quot;AQVP&quot;, &quot;UPHE&quot;, &quot;AQVP&quot;, &quot;TRSH&quot;, &quot;UPHE&quot;, &quot;TRSH&quot;, &quot;TRSH&quot;, &quot;UPHE&quot;, &quot;AQVP&quot;, &quot;UPHE&quot;, &quot;TRSH&quot;, &quot;TRSH&quot;, &quot;TRSH&quot;, &quot;TRSH&quot;, &quot;TRSH&quot;, &quot;AQVP&quot;, &quot;TRSH&quot;, &quot;AQVP&quot;, &#10;&quot;TRSH&quot;, &quot;UPHE&quot;, &quot;UPHE&quot;, &quot;UPHE&quot;, &quot;TRSH&quot;, &quot;TRSH&quot;, &quot;AQVP&quot;, &quot;TRSH&quot;, &quot;UPHE&quot;, &quot;TRSH&quot;, &quot;UPHE&quot;, &quot;UPHE&quot;, &quot;AQVP&quot;, &quot;TRSH&quot;, &quot;AQVP&quot;, &quot;UPHE&quot;, &quot;UPHE&quot;, &quot;UPHE&quot;, &quot;UPHE&quot;, &quot;TRSH&quot;, &quot;TRSH&quot;, &quot;AQVP&quot;, &quot;AQVP&quot;, &quot;UPHE&quot;, &quot;TRSH&quot;, &quot;TRSH&quot;, &quot;UNID&quot;, &quot;UNID&quot;, &quot;VACR&quot;, &quot;VACR&quot;, &quot;VACR&quot;, &quot;VACR&quot;, &quot;AQVP&quot;, &quot;VACR&quot;, &quot;VACR&quot;, &quot;VACR&quot;, &quot;VACR&quot;, &quot;VACR&quot;, &quot;VACR&quot;, &quot;VACR&quot;, &quot;AQBR&quot;, &quot;UNID&quot;, &quot;UNID&quot;, &quot;VACR&quot;, &quot;VACR&quot;, &quot;VACR&quot;, &quot;VACR&quot;, &quot;VACR&quot;, &quot;UPBR&quot;, &quot;VACR&quot;, &quot;VACR&quot;, &quot;VACR&quot;, &quot;VACR&quot;, &quot;VACR&quot;, &quot;FUNG&quot;, &quot;FUNG&quot;, &quot;VACR&quot;, &quot;VACR&quot;, &quot;VACR&quot;, &quot;VACR&quot;, &quot;LABO&quot;, &quot;LABO&quot;, &quot;LABO&quot;, &#10;&quot;CHAR&quot;, &quot;LABO&quot;) c(1, 1, 2, 3, 1, 2, 24, 1, 2, 1, 14, 2, 6, 12, 1, 1, 2, 27, 27, 2, 1, 1, 27, 7, 6, 14, 3, 5, 19, 25, 20, 23, 1, 1, 1, 1, 9, 24, 6, 2, 7, 6, 6, 8, 3, 4, 11, 11, 5, 25, 1, 3, 6, 2, 2, 1, 5, 5, 2, 11, 10, 7, 24, 26, 12, 12, 15, 1, 16, 2, 1, 4, 27, 3, 1, 1, 1, 8, 7, 27, 6, 2, 2, 9, 1, 4, 1, 2, 3, 1, 14, 25, 27, 4, 6, 13, 27, 4, 1, 1, 2, 1, 1, 2, 1, 19, 11, 1, 8, 15, 3, 3, 5, 11, 2, 4, 1, 10, 4, 10, 1, 2, 8, 12, 20, 18, 3, 1, 1, 21, 1, 3, 4, 2, 1, 1, 1, 1, 1, 6, 1, 9, 10, 3, 3, 1, 3, 1, 2, 2, 3, 2, 3, &#10;10, 1, 1, 3, 1, 1, 3, 2, 1, 2, 1, 1, 1, 3, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 3, 5, 2, 1, 10, 4, 5, 1, 1, 1, 3, 1, 2, 2, 1, 1, 1, 1, 1, 2, 2, 5, 1, 2, 3, 6, 1, 3, 2, 1, 1, 1, 1, 1, 1, 1, 1, 1, 19, 17, 3, 2, 8, 1, 3, 2, 3, 1, 3, 1, 1, 1, 8, 8, 10, 4, 1, 5, 2, 1, 1, 1, 1, 6, 2, 1, 1, 2, 1, 1, 2, 1, 3, 2, 5, 1, 2) c(15, 144, 116, 261, 15, 52, 355, 20, 75, 7, 214, 41, 180, 567, 24, 2, 40, 939, 1415, 14, 1, 1, 1229, 22, 68, 281, 60, 185, 684, 681, 526, 804, 11, 4, 5, 3, 289, 1063, 121, 20, 143, 72, 198, 148, 3, 24, 314, 270, 117, 640, 1, 5, 48, 3, 10, 1, 72, 44, 26, 129, 76, 93, 1191, 1038, 612, 598, 792, 2, 316, 57, 3, 64, 1531, 47, 4, 1, 1, 191, 27, 876, 10, 56, 2, 26, 1, 16, 1, 3, 14, 1, 111, 925, 886, 82, 102, 124, 1482, 14, 47, 1, 2, 6, 12, 15, 1, 666, 644, 2, 332, 686, 27, 5, 31, 126, 69, 14, 2, 528, 109, &#10;274, 4, 22, 64, 638, 621, 500, 4, 2, 1, 907, 32, 7, 15, 13, 2, 1, 2, 1, 2, 26, 27, 196, 193, 41, 6, 1, 21, 50, 4, 8, 11, 185, 92, 515, 2, 6, 100, 1, 40, 10, 84, 44, 20, 60, 1, 3, 146, 10, 5, 10, 2, 3, 2, 1, 3, 1, 145, 202, 54, 15, 1, 343, 99, 201, 17, 57, 1, 91, 1, 184, 32, 2, 14, 2, 52, 9, 128, 28, 51, 1, 3, 36, 115, 1, 51, 6, 1, 32, 21, 25, 1, 6, 1, 4, 1, 598, 393, 31, 41, 251, 24, 42, 6, 13, 1, 21, 28, 36, 1, 300, 156, 231, 16, 1, 67, 54, 2, 10, 27, 1, 202, 14, 11, 6, 99, 36, 31, 32, 3, 161, 127, &#10;238, 36, 116)\">\n  <thead>\n    <tr>\n      <th>units<\/th>\n      <th>context<\/th>\n      <th>element<\/th>\n      <th>taxonid<\/th>\n      <th>symmetry<\/th>\n      <th>taxongroup<\/th>\n      <th>elementtype<\/th>\n      <th>variablename<\/th>\n      <th>ecologicalgroup<\/th>\n      <th>sites<\/th>\n      <th>samples<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"scrollX":"100%","dom":"t","columnDefs":[{"className":"dt-right","targets":[3,9,10]}],"order":[],"autoWidth":false,"orderClasses":false},"selection":{"mode":"multiple","selected":null,"target":"row","selectable":null}},"evals":[],"jsHooks":[]}</script>
</div>
</div>
</section>
</section>
<section id="section-5" class="level4 unnumbered">
<h4 class="unnumbered anchored" data-anchor-id="section-5"></h4>
<p>The <code>taxonid</code> values can be linked to the <code>taxonid</code> column in the <code>samples()</code>. This allows us to build taxon harmonization tables if we choose to. Note also that the <code>taxonname</code> is in the field <code>variablename</code>. Individual sample counts are reported in Neotoma as <a href="https://open.neotomadb.org/manual/taxonomy-related-tables-1.html#Variables"><code>variables</code></a>. A “variable” may be either a species for which we have presence or count data, a geochemical measurement, or any other proxy, such as charcoal counts. Each stored entry for a variable includes the units of measurement and the value.</p>
</section>
<section id="taxonomic-harmonization-simple" class="level4 tabset" data-number="5.4.8.2">
<h4 class="tabset anchored" data-number="5.4.8.2" data-anchor-id="taxonomic-harmonization-simple"><span class="header-section-number">5.4.8.2</span> Taxonomic Harmonization (Simple)</h4>
<p>A standard challenge in Neotoma (and in biodiversity research more generally) is that different scientists use different names for taxonomic entities such as species. Even if everyone agrees on a common taxonomy, it’s quite possible that a given fossil might be only partially identifiable, perhaps just to genus or even family. Hence, when working with data from Neotoma, a common intermediary step is to ‘harmonize’ all the taxa names stored in Neotoma into some standard names of interest to you.</p>
<p>Let’s say we want to know the past distribution of <em>Pinus</em>. We want all the various pollen morphotypes that are associated with <em>Pinus</em> (e.g.&nbsp;<em>Pinus strobus</em>, <em>Pinus strobus</em>-type, <em>Pinus undif.</em>, <em>Pinus banksiana/resinosa</em>) to be grouped together into one aggregated taxon names called <em>Pinus</em>. There are several ways of doing this, either directly by exporting the file and editing each individual cell, or by creating an external “harmonization” table.</p>
<p>Programmatically, we can harmonize all the taxon names using matching and transformation. We’re using <code>dplyr</code> type coding here to <code>mutate()</code> the column <code>variablename</code> so that any time we detect (<code>str_detect()</code>) a <code>variablename</code> that starts with <code>Pinus</code> (the <code>.*</code> represents a wildcard for any character [<code>.</code>], zero or more times [<code>*</code>]) we <code>replace()</code> it with the character string <code>"Pinus"</code>. Note that this changes <em>Pinus</em> in the <code>allSamp</code> object, but if we were to call <code>samples()</code> again, the taxonomy would return to its original form.</p>
<p>As a first step, we’re going to filter the ecological groups to include only <em>UPHE</em> (upleand/heath) and <em>TRSH</em> (trees and shrubs). (More information about ecological groups is available from the <a href="https://open.neotomadb.org/manual">Neotoma Online Manual</a>.) After converting all _Pinus.<em>_ records to </em>Pinus* we then sum the counts of the <em>Pinus</em> records.</p>
<div class="cell" data-hash="04-neotoma2_cache/html/simpleTaxonChange_c30af9e4c8f7a8aeeea91a75fbcae26c">
<div class="sourceCode cell-code" id="cb15"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb15-1"><a href="#cb15-1" aria-hidden="true" tabindex="-1"></a>allSamp <span class="ot">&lt;-</span> allSamp <span class="sc">%&gt;%</span></span>
<span id="cb15-2"><a href="#cb15-2" aria-hidden="true" tabindex="-1"></a>  dplyr<span class="sc">::</span><span class="fu">filter</span>(ecologicalgroup <span class="sc">%in%</span> <span class="fu">c</span>(<span class="st">"UPHE"</span>, <span class="st">"TRSH"</span>)) <span class="sc">%&gt;%</span></span>
<span id="cb15-3"><a href="#cb15-3" aria-hidden="true" tabindex="-1"></a>  <span class="fu">mutate</span>(<span class="at">variablename =</span> <span class="fu">replace</span>(variablename,</span>
<span id="cb15-4"><a href="#cb15-4" aria-hidden="true" tabindex="-1"></a>                                stringr<span class="sc">::</span><span class="fu">str_detect</span>(variablename, <span class="st">"Pinus.*"</span>),</span>
<span id="cb15-5"><a href="#cb15-5" aria-hidden="true" tabindex="-1"></a>                                <span class="st">"Pinus"</span>),</span>
<span id="cb15-6"><a href="#cb15-6" aria-hidden="true" tabindex="-1"></a>         <span class="at">variablename =</span> <span class="fu">replace</span>(variablename,</span>
<span id="cb15-7"><a href="#cb15-7" aria-hidden="true" tabindex="-1"></a>                                stringr<span class="sc">::</span><span class="fu">str_detect</span>(variablename, <span class="st">"Picea.*"</span>),</span>
<span id="cb15-8"><a href="#cb15-8" aria-hidden="true" tabindex="-1"></a>                                <span class="st">"Picea"</span>)) <span class="sc">%&gt;%</span></span>
<span id="cb15-9"><a href="#cb15-9" aria-hidden="true" tabindex="-1"></a>  <span class="fu">group_by</span>(siteid, sitename,</span>
<span id="cb15-10"><a href="#cb15-10" aria-hidden="true" tabindex="-1"></a>           sampleid, variablename, units, age,</span>
<span id="cb15-11"><a href="#cb15-11" aria-hidden="true" tabindex="-1"></a>           agetype, depth, datasetid,</span>
<span id="cb15-12"><a href="#cb15-12" aria-hidden="true" tabindex="-1"></a>           long, lat) <span class="sc">%&gt;%</span></span>
<span id="cb15-13"><a href="#cb15-13" aria-hidden="true" tabindex="-1"></a>  <span class="fu">summarise</span>(<span class="at">value =</span> <span class="fu">sum</span>(value), <span class="at">.groups=</span><span class="st">'keep'</span>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
<p>There were originally 6 different taxa identified as being within the genus <em>Pinus</em> (including <em>Pinus</em>, <em>Pinus subg. Pinus</em>, and <em>Pinus undiff.</em>). The above code reduces them all to a single taxonomic group <em>Pinus</em>. We can check out the unique names by using:</p>
<section id="code-6" class="level5" data-number="5.4.8.2.1">
<h5 data-number="5.4.8.2.1" class="anchored" data-anchor-id="code-6"><span class="header-section-number">5.4.8.2.1</span> Code</h5>
<div class="cell" data-hash="04-neotoma2_cache/html/UniqueNames_0dbc455cd3211477ceac7b7d1ca5eb87">
<div class="sourceCode cell-code" id="cb16"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb16-1"><a href="#cb16-1" aria-hidden="true" tabindex="-1"></a>neotomatx <span class="sc">%&gt;%</span></span>
<span id="cb16-2"><a href="#cb16-2" aria-hidden="true" tabindex="-1"></a>  <span class="fu">ungroup</span>() <span class="sc">%&gt;%</span></span>
<span id="cb16-3"><a href="#cb16-3" aria-hidden="true" tabindex="-1"></a>  <span class="fu">filter</span>(stringr<span class="sc">::</span><span class="fu">str_detect</span>(variablename, <span class="st">"Pinus"</span>)) <span class="sc">%&gt;%</span></span>
<span id="cb16-4"><a href="#cb16-4" aria-hidden="true" tabindex="-1"></a>  <span class="fu">summarise</span>(<span class="at">pinus_spp =</span> <span class="fu">unique</span>(variablename))</span>
<span id="cb16-5"><a href="#cb16-5" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb16-6"><a href="#cb16-6" aria-hidden="true" tabindex="-1"></a><span class="co"># I actually like Base here for the one-liner:</span></span>
<span id="cb16-7"><a href="#cb16-7" aria-hidden="true" tabindex="-1"></a><span class="co"># unique(grep("Pinus", neotomatx$variablename, value = TRUE))</span></span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
</section>
<section id="result-5" class="level5" data-number="5.4.8.2.2">
<h5 data-number="5.4.8.2.2" class="anchored" data-anchor-id="result-5"><span class="header-section-number">5.4.8.2.2</span> Result</h5>
<div class="cell" data-hash="04-neotoma2_cache/html/UniqueNamesTab_46d82501ea6be8ebf333fc6b043981be">
<div class="cell-output-display">

<div class="datatables html-widget html-fill-item-overflow-hidden html-fill-item" id="htmlwidget-e2fae7cc89214b2ab25f" style="width:100%;height:auto;"></div>
<script type="application/json" data-for="htmlwidget-e2fae7cc89214b2ab25f">{"x":{"filter":"none","vertical":false,"data":[["1","2","3","4","5","6"],["Pinus subg. Pinus","Pinus subg. Strobus","Pinus undiff.","Pinus","Pinus strobus","Pinus banksiana/P. resinosa"]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>pinus_spp<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"scrollX":"100%","dom":"t","columnDefs":[{"orderable":false,"targets":0}],"order":[],"autoWidth":false,"orderClasses":false},"selection":{"mode":"multiple","selected":null,"target":"row","selectable":null}},"evals":[],"jsHooks":[]}</script>
</div>
</div>
</section>
</section>
<section id="section-6" class="level4 unnumbered tabset">
<h4 class="unnumbered tabset anchored" data-anchor-id="section-6"></h4>
<p><strong>Question 5</strong>: Follow the <em>Pinus</em> example above, but now for <em>Picea.</em> How many taxon names were aggregated into your <em>Picea</em> name?</p>
<p>If we want to store a record of our choices outside of R, we can use an external table. For example, a table of pairs (what we want changed, and the name we want it replaced with) can be generated, and it can include regular expressions (if we choose):</p>
<table class="table">
<thead>
<tr class="header">
<th>original</th>
<th>replacement</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>Abies.*</td>
<td>Abies</td>
</tr>
<tr class="even">
<td>Vaccinium.*</td>
<td>Ericaceae</td>
</tr>
<tr class="odd">
<td>Typha.*</td>
<td>Aquatic</td>
</tr>
<tr class="even">
<td>Nymphaea</td>
<td>Aquatic</td>
</tr>
<tr class="odd">
<td>…</td>
<td>…</td>
</tr>
</tbody>
</table>
<p>We can get the list of original names directly from the <code>taxa()</code> call, applied to a <code>sites</code> object, and then export it using <code>write.csv()</code>. We can also do some exploratory plots of the data:</p>
<section id="code-7" class="level5" data-number="5.4.8.2.3">
<h5 data-number="5.4.8.2.3" class="anchored" data-anchor-id="code-7"><span class="header-section-number">5.4.8.2.3</span> Code</h5>
<div class="cell" data-hash="04-neotoma2_cache/html/countbySitesSamples_af6af2667e3460f2f890989a7269697e">
<div class="sourceCode cell-code" id="cb17"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb17-1"><a href="#cb17-1" aria-hidden="true" tabindex="-1"></a>taxaplots <span class="ot">&lt;-</span> <span class="fu">taxa</span>(mich_dl)</span>
<span id="cb17-2"><a href="#cb17-2" aria-hidden="true" tabindex="-1"></a><span class="co"># Save the taxon list to file so we can edit it subsequently.</span></span>
<span id="cb17-3"><a href="#cb17-3" aria-hidden="true" tabindex="-1"></a>readr<span class="sc">::</span><span class="fu">write_csv</span>(taxaplots, <span class="st">"data/mytaxontable.csv"</span>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
</section>
<section id="result-6" class="level5" data-number="5.4.8.2.4">
<h5 data-number="5.4.8.2.4" class="anchored" data-anchor-id="result-6"><span class="header-section-number">5.4.8.2.4</span> Result</h5>
<div class="cell" data-hash="04-neotoma2_cache/html/PlotTaxonCounts_561f2326fe67cf390ab8733d24611daf">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="04-neotoma2_files/figure-html/PlotTaxonCounts-1.png" class="img-fluid figure-img" width="672"></p>
<p></p><figcaption class="figure-caption"><strong>Figure</strong>. A plot of the number of sites a taxon appears in, against the number of samples a taxon appears in.</figcaption><p></p>
</figure>
</div>
</div>
</div>
</section>
</section>
<section id="section-7" class="level4 unnumbered">
<h4 class="unnumbered anchored" data-anchor-id="section-7"></h4>
<p>The plot is mostly for illustration, but we can see, as a sanity check, that the relationship is as we’d expect.</p>
</section>
</section>
</section>
<section id="simple-analytics" class="level2" data-number="5.5">
<h2 data-number="5.5" class="anchored" data-anchor-id="simple-analytics"><span class="header-section-number">5.5</span> Simple Analytics</h2>
<section id="stratigraphic-plotting-building-a-pollen-diagram" class="level3" data-number="5.5.1">
<h3 data-number="5.5.1" class="anchored" data-anchor-id="stratigraphic-plotting-building-a-pollen-diagram"><span class="header-section-number">5.5.1</span> Stratigraphic Plotting: Building a Pollen Diagram</h3>
<p>As you’ve seen already, stratigraphic diagrams are a very common way of viewing geological data, in which time is represented vertically and with older materials at bottom, just like in the sediment record. Palynologists use a particular form of a stratigraphic diagram called a pollen diagram.</p>
<p>We can use packages like <code>rioja</code> to do stratigraphic plotting for a single dataset. Here, we’ll take a few key species at a single site and plot them.</p>
<div class="cell" data-result="false" data-hash="04-neotoma2_cache/html/stratiplot2_924edbae7a79c66043a549ddb0956e78">
<div class="sourceCode cell-code" id="cb18"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb18-1"><a href="#cb18-1" aria-hidden="true" tabindex="-1"></a><span class="co"># Get a particular site, select only taxa identified from pollen (and only trees/shrubs)</span></span>
<span id="cb18-2"><a href="#cb18-2" aria-hidden="true" tabindex="-1"></a><span class="co"># Transform to proportion values.</span></span>
<span id="cb18-3"><a href="#cb18-3" aria-hidden="true" tabindex="-1"></a>devils_samples <span class="ot">&lt;-</span> <span class="fu">get_sites</span>(<span class="at">siteid =</span> <span class="dv">666</span>) <span class="sc">%&gt;%</span></span>
<span id="cb18-4"><a href="#cb18-4" aria-hidden="true" tabindex="-1"></a>  <span class="fu">get_downloads</span>() <span class="sc">%&gt;%</span></span>
<span id="cb18-5"><a href="#cb18-5" aria-hidden="true" tabindex="-1"></a>  <span class="fu">samples</span>()</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stdout">
<pre><code>.....</code></pre>
</div>
<div class="sourceCode cell-code" id="cb20"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb20-1"><a href="#cb20-1" aria-hidden="true" tabindex="-1"></a>devils_samples <span class="ot">&lt;-</span> devils_samples <span class="sc">%&gt;%</span></span>
<span id="cb20-2"><a href="#cb20-2" aria-hidden="true" tabindex="-1"></a>  <span class="fu">mutate</span>(<span class="at">variablename =</span> <span class="fu">replace</span>(variablename,</span>
<span id="cb20-3"><a href="#cb20-3" aria-hidden="true" tabindex="-1"></a>                                stringr<span class="sc">::</span><span class="fu">str_detect</span>(variablename, <span class="st">"Pinus.*"</span>),</span>
<span id="cb20-4"><a href="#cb20-4" aria-hidden="true" tabindex="-1"></a>                                <span class="st">"Pinus"</span>)) <span class="sc">%&gt;%</span></span>
<span id="cb20-5"><a href="#cb20-5" aria-hidden="true" tabindex="-1"></a>  <span class="fu">group_by</span>(siteid, sitename,</span>
<span id="cb20-6"><a href="#cb20-6" aria-hidden="true" tabindex="-1"></a>           sampleid, variablename, units, age,</span>
<span id="cb20-7"><a href="#cb20-7" aria-hidden="true" tabindex="-1"></a>           agetype, depth, datasetid,</span>
<span id="cb20-8"><a href="#cb20-8" aria-hidden="true" tabindex="-1"></a>           long, lat) <span class="sc">%&gt;%</span></span>
<span id="cb20-9"><a href="#cb20-9" aria-hidden="true" tabindex="-1"></a>  <span class="fu">summarise</span>(<span class="at">value =</span> <span class="fu">sum</span>(value), <span class="at">.groups=</span><span class="st">'keep'</span>)</span>
<span id="cb20-10"><a href="#cb20-10" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb20-11"><a href="#cb20-11" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb20-12"><a href="#cb20-12" aria-hidden="true" tabindex="-1"></a>onesite <span class="ot">&lt;-</span> devils_samples <span class="sc">%&gt;%</span></span>
<span id="cb20-13"><a href="#cb20-13" aria-hidden="true" tabindex="-1"></a>  <span class="fu">group_by</span>(age) <span class="sc">%&gt;%</span></span>
<span id="cb20-14"><a href="#cb20-14" aria-hidden="true" tabindex="-1"></a>  <span class="fu">mutate</span>(<span class="at">pollencount =</span> <span class="fu">sum</span>(value, <span class="at">na.rm =</span> <span class="cn">TRUE</span>)) <span class="sc">%&gt;%</span></span>
<span id="cb20-15"><a href="#cb20-15" aria-hidden="true" tabindex="-1"></a>  <span class="fu">group_by</span>(variablename) <span class="sc">%&gt;%</span></span>
<span id="cb20-16"><a href="#cb20-16" aria-hidden="true" tabindex="-1"></a>  <span class="fu">mutate</span>(<span class="at">prop =</span> value <span class="sc">/</span> pollencount) <span class="sc">%&gt;%</span></span>
<span id="cb20-17"><a href="#cb20-17" aria-hidden="true" tabindex="-1"></a>  <span class="fu">arrange</span>(<span class="fu">desc</span>(age))</span>
<span id="cb20-18"><a href="#cb20-18" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb20-19"><a href="#cb20-19" aria-hidden="true" tabindex="-1"></a><span class="co"># Spread the data to a "wide" table, with taxa as column headings.</span></span>
<span id="cb20-20"><a href="#cb20-20" aria-hidden="true" tabindex="-1"></a>widetable <span class="ot">&lt;-</span> onesite <span class="sc">%&gt;%</span></span>
<span id="cb20-21"><a href="#cb20-21" aria-hidden="true" tabindex="-1"></a>  dplyr<span class="sc">::</span><span class="fu">select</span>(age, variablename, prop) <span class="sc">%&gt;%</span></span>
<span id="cb20-22"><a href="#cb20-22" aria-hidden="true" tabindex="-1"></a>  <span class="fu">mutate</span>(<span class="at">prop =</span> <span class="fu">as.numeric</span>(prop))  <span class="sc">%&gt;%</span></span>
<span id="cb20-23"><a href="#cb20-23" aria-hidden="true" tabindex="-1"></a>  <span class="fu">filter</span>(variablename <span class="sc">%in%</span> <span class="fu">c</span>(<span class="st">"Pinus"</span>, <span class="st">"Betula"</span>, <span class="st">"Quercus"</span>,</span>
<span id="cb20-24"><a href="#cb20-24" aria-hidden="true" tabindex="-1"></a>                             <span class="st">"Tsuga"</span>, <span class="st">"Ulmus"</span>, <span class="st">"Picea"</span>))</span>
<span id="cb20-25"><a href="#cb20-25" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb20-26"><a href="#cb20-26" aria-hidden="true" tabindex="-1"></a>counts <span class="ot">&lt;-</span> tidyr<span class="sc">::</span><span class="fu">pivot_wider</span>(widetable,</span>
<span id="cb20-27"><a href="#cb20-27" aria-hidden="true" tabindex="-1"></a>                             <span class="at">id_cols =</span> age,</span>
<span id="cb20-28"><a href="#cb20-28" aria-hidden="true" tabindex="-1"></a>                             <span class="at">names_from =</span> variablename,</span>
<span id="cb20-29"><a href="#cb20-29" aria-hidden="true" tabindex="-1"></a>                             <span class="at">values_from =</span> prop,</span>
<span id="cb20-30"><a href="#cb20-30" aria-hidden="true" tabindex="-1"></a>                             <span class="at">values_fill =</span> <span class="dv">0</span>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
<p>This appears to be a fairly long set of commands, but the code is pretty straightforward, and it provides you with significant control over the taxa for display, units pf measurement, and other elements of your data before you get them into the wide matrix (<code>depth</code> by <code>taxon</code>) that most statistical tools such as the <code>vegan</code> package or <code>rioja</code> use. To plot we can use <code>rioja</code>’s <code>strat.plot()</code>, sorting the taxa using weighted averaging scores (<code>wa.order</code>). We’ve also added a CONISS plot to the edge of the plot, to show how the new <em>wide</em> data frame works with distance metric functions. (We’ll talk more about distance and dissimilarity metrics in upcoming labs.)</p>
<div class="cell" data-hash="04-neotoma2_cache/html/plotStrigraph_1cc9b8b9d85fd9d4ba04b10e1b8b7c93">
<div class="sourceCode cell-code" id="cb21"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb21-1"><a href="#cb21-1" aria-hidden="true" tabindex="-1"></a>clust <span class="ot">&lt;-</span> rioja<span class="sc">::</span><span class="fu">chclust</span>(<span class="fu">dist</span>(<span class="fu">sqrt</span>(counts)),</span>
<span id="cb21-2"><a href="#cb21-2" aria-hidden="true" tabindex="-1"></a>                        <span class="at">method =</span> <span class="st">"coniss"</span>)</span>
<span id="cb21-3"><a href="#cb21-3" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb21-4"><a href="#cb21-4" aria-hidden="true" tabindex="-1"></a>plot <span class="ot">&lt;-</span> rioja<span class="sc">::</span><span class="fu">strat.plot</span>(counts[,<span class="sc">-</span><span class="dv">1</span>] <span class="sc">*</span> <span class="dv">100</span>, <span class="at">yvar =</span> counts<span class="sc">$</span>age,</span>
<span id="cb21-5"><a href="#cb21-5" aria-hidden="true" tabindex="-1"></a>                  <span class="at">title =</span> devils_samples<span class="sc">$</span>sitename[<span class="dv">1</span>],</span>
<span id="cb21-6"><a href="#cb21-6" aria-hidden="true" tabindex="-1"></a>                  <span class="at">ylabel =</span> <span class="st">"Calibrated Years BP"</span>,</span>
<span id="cb21-7"><a href="#cb21-7" aria-hidden="true" tabindex="-1"></a>                  <span class="at">xlabel =</span> <span class="st">"Pollen (%)"</span>,</span>
<span id="cb21-8"><a href="#cb21-8" aria-hidden="true" tabindex="-1"></a>                  <span class="at">y.rev =</span> <span class="cn">TRUE</span>,</span>
<span id="cb21-9"><a href="#cb21-9" aria-hidden="true" tabindex="-1"></a>                  <span class="at">clust =</span> clust,</span>
<span id="cb21-10"><a href="#cb21-10" aria-hidden="true" tabindex="-1"></a>                  <span class="at">wa.order =</span> <span class="st">"topleft"</span>, <span class="at">scale.percent =</span> <span class="cn">TRUE</span>)</span>
<span id="cb21-11"><a href="#cb21-11" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb21-12"><a href="#cb21-12" aria-hidden="true" tabindex="-1"></a>rioja<span class="sc">::</span><span class="fu">addClustZone</span>(plot, clust, <span class="dv">4</span>, <span class="at">col =</span> <span class="st">"red"</span>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output-display">
<p><img src="04-neotoma2_files/figure-html/plotStrigraph-1.png" class="img-fluid" width="672"></p>
</div>
</div>
<p><strong>Question 5</strong>: Make a stratigraphic pollen diagram in <code>rioja</code>, for a site of your choice (not Devils Lake) and taxa of your choice. Show code and resulting diagram.</p>
</section>
<section id="change-taxon-distributions-across-space-and-time" class="level3" data-number="5.5.2">
<h3 data-number="5.5.2" class="anchored" data-anchor-id="change-taxon-distributions-across-space-and-time"><span class="header-section-number">5.5.2</span> Change Taxon Distributions Across Space and Time</h3>
<p>The true power of Neotoma is its ability to support large-scale analyses across many sites, many s time periods within sites, many proxies, and many taxa. As a first dipping of our toes in the water, lets look at temporal trends in abundance when averaged across ites. We now have site information across Michigan, with samples, and with taxon names. Let’s say we are interested in looking at the distributions of the selected taxa across time, their presence/absence:</p>
<div class="cell" data-hash="04-neotoma2_cache/html/summarizeByTime_8372e94d9abc0bdb357fc93a6c5f7dc1">
<div class="sourceCode cell-code" id="cb22"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb22-1"><a href="#cb22-1" aria-hidden="true" tabindex="-1"></a>taxabyage <span class="ot">&lt;-</span> allSamp <span class="sc">%&gt;%</span></span>
<span id="cb22-2"><a href="#cb22-2" aria-hidden="true" tabindex="-1"></a>  <span class="fu">filter</span>(variablename <span class="sc">%in%</span> <span class="fu">c</span>(<span class="st">"Pinus"</span>, <span class="st">"Betula"</span>, <span class="st">"Quercus"</span>,</span>
<span id="cb22-3"><a href="#cb22-3" aria-hidden="true" tabindex="-1"></a>                             <span class="st">"Tsuga"</span>, <span class="st">"Ulmus"</span>, <span class="st">"Picea"</span>),</span>
<span id="cb22-4"><a href="#cb22-4" aria-hidden="true" tabindex="-1"></a>                             age <span class="sc">&lt;</span> <span class="dv">11000</span>) <span class="sc">%&gt;%</span></span>
<span id="cb22-5"><a href="#cb22-5" aria-hidden="true" tabindex="-1"></a>  <span class="fu">group_by</span>(variablename, <span class="st">"age"</span> <span class="ot">=</span> <span class="fu">round</span>(age <span class="sc">*</span> <span class="dv">2</span>, <span class="sc">-</span><span class="dv">3</span>) <span class="sc">/</span> <span class="dv">2</span>) <span class="sc">%&gt;%</span></span>
<span id="cb22-6"><a href="#cb22-6" aria-hidden="true" tabindex="-1"></a>  <span class="fu">summarise</span>(<span class="at">n =</span> <span class="fu">length</span>(<span class="fu">unique</span>(siteid)), <span class="at">.groups =</span> <span class="st">'keep'</span>)</span>
<span id="cb22-7"><a href="#cb22-7" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb22-8"><a href="#cb22-8" aria-hidden="true" tabindex="-1"></a>samplesbyage <span class="ot">&lt;-</span> allSamp <span class="sc">%&gt;%</span></span>
<span id="cb22-9"><a href="#cb22-9" aria-hidden="true" tabindex="-1"></a>  <span class="fu">filter</span>(variablename <span class="sc">%in%</span> <span class="fu">c</span>(<span class="st">"Pinus"</span>, <span class="st">"Betula"</span>, <span class="st">"Quercus"</span>,</span>
<span id="cb22-10"><a href="#cb22-10" aria-hidden="true" tabindex="-1"></a>                             <span class="st">"Tsuga"</span>, <span class="st">"Ulmus"</span>, <span class="st">"Picea"</span>)) <span class="sc">%&gt;%</span></span>
<span id="cb22-11"><a href="#cb22-11" aria-hidden="true" tabindex="-1"></a>  <span class="fu">group_by</span>(<span class="st">"age"</span> <span class="ot">=</span> <span class="fu">round</span>(age <span class="sc">*</span> <span class="dv">2</span>, <span class="sc">-</span><span class="dv">3</span>) <span class="sc">/</span> <span class="dv">2</span>) <span class="sc">%&gt;%</span></span>
<span id="cb22-12"><a href="#cb22-12" aria-hidden="true" tabindex="-1"></a>  <span class="fu">summarise</span>(<span class="at">samples =</span> <span class="fu">length</span>(<span class="fu">unique</span>(siteid)), <span class="at">.groups =</span> <span class="st">'keep'</span>)</span>
<span id="cb22-13"><a href="#cb22-13" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb22-14"><a href="#cb22-14" aria-hidden="true" tabindex="-1"></a>groupbyage <span class="ot">&lt;-</span> taxabyage <span class="sc">%&gt;%</span></span>
<span id="cb22-15"><a href="#cb22-15" aria-hidden="true" tabindex="-1"></a>  <span class="fu">inner_join</span>(samplesbyage, <span class="at">by =</span> <span class="st">"age"</span>) <span class="sc">%&gt;%</span></span>
<span id="cb22-16"><a href="#cb22-16" aria-hidden="true" tabindex="-1"></a>  <span class="fu">mutate</span>(<span class="at">proportion =</span> n <span class="sc">/</span> samples)</span>
<span id="cb22-17"><a href="#cb22-17" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb22-18"><a href="#cb22-18" aria-hidden="true" tabindex="-1"></a><span class="fu">ggplot</span>(groupbyage, <span class="fu">aes</span>(<span class="at">x =</span> age, <span class="at">y =</span> proportion)) <span class="sc">+</span></span>
<span id="cb22-19"><a href="#cb22-19" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_point</span>() <span class="sc">+</span></span>
<span id="cb22-20"><a href="#cb22-20" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_smooth</span>(<span class="at">method =</span> <span class="st">'gam'</span>,</span>
<span id="cb22-21"><a href="#cb22-21" aria-hidden="true" tabindex="-1"></a>              <span class="at">method.args =</span> <span class="fu">list</span>(<span class="at">family =</span> <span class="st">'binomial'</span>)) <span class="sc">+</span></span>
<span id="cb22-22"><a href="#cb22-22" aria-hidden="true" tabindex="-1"></a>  <span class="fu">facet_wrap</span>(<span class="sc">~</span>variablename) <span class="sc">+</span></span>
<span id="cb22-23"><a href="#cb22-23" aria-hidden="true" tabindex="-1"></a>  <span class="co">#coord_cartesian(xlim = c(22500, 0), ylim = c(0, 1)) +</span></span>
<span id="cb22-24"><a href="#cb22-24" aria-hidden="true" tabindex="-1"></a>  <span class="fu">scale_x_reverse</span>() <span class="sc">+</span></span>
<span id="cb22-25"><a href="#cb22-25" aria-hidden="true" tabindex="-1"></a>  <span class="fu">xlab</span>(<span class="st">"Proportion of Sites with Taxon"</span>) <span class="sc">+</span></span>
<span id="cb22-26"><a href="#cb22-26" aria-hidden="true" tabindex="-1"></a>  <span class="fu">theme_bw</span>()</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stderr">
<pre><code>Warning in eval(family$initialize): non-integer #successes in a binomial glm!

Warning in eval(family$initialize): non-integer #successes in a binomial glm!

Warning in eval(family$initialize): non-integer #successes in a binomial glm!

Warning in eval(family$initialize): non-integer #successes in a binomial glm!

Warning in eval(family$initialize): non-integer #successes in a binomial glm!

Warning in eval(family$initialize): non-integer #successes in a binomial glm!

Warning in eval(family$initialize): non-integer #successes in a binomial glm!

Warning in eval(family$initialize): non-integer #successes in a binomial glm!

Warning in eval(family$initialize): non-integer #successes in a binomial glm!

Warning in eval(family$initialize): non-integer #successes in a binomial glm!

Warning in eval(family$initialize): non-integer #successes in a binomial glm!

Warning in eval(family$initialize): non-integer #successes in a binomial glm!

Warning in eval(family$initialize): non-integer #successes in a binomial glm!

Warning in eval(family$initialize): non-integer #successes in a binomial glm!

Warning in eval(family$initialize): non-integer #successes in a binomial glm!

Warning in eval(family$initialize): non-integer #successes in a binomial glm!

Warning in eval(family$initialize): non-integer #successes in a binomial glm!

Warning in eval(family$initialize): non-integer #successes in a binomial glm!

Warning in eval(family$initialize): non-integer #successes in a binomial glm!

Warning in eval(family$initialize): non-integer #successes in a binomial glm!

Warning in eval(family$initialize): non-integer #successes in a binomial glm!

Warning in eval(family$initialize): non-integer #successes in a binomial glm!

Warning in eval(family$initialize): non-integer #successes in a binomial glm!

Warning in eval(family$initialize): non-integer #successes in a binomial glm!

Warning in eval(family$initialize): non-integer #successes in a binomial glm!

Warning in eval(family$initialize): non-integer #successes in a binomial glm!

Warning in eval(family$initialize): non-integer #successes in a binomial glm!

Warning in eval(family$initialize): non-integer #successes in a binomial glm!

Warning in eval(family$initialize): non-integer #successes in a binomial glm!

Warning in eval(family$initialize): non-integer #successes in a binomial glm!

Warning in eval(family$initialize): non-integer #successes in a binomial glm!

Warning in eval(family$initialize): non-integer #successes in a binomial glm!

Warning in eval(family$initialize): non-integer #successes in a binomial glm!

Warning in eval(family$initialize): non-integer #successes in a binomial glm!

Warning in eval(family$initialize): non-integer #successes in a binomial glm!

Warning in eval(family$initialize): non-integer #successes in a binomial glm!

Warning in eval(family$initialize): non-integer #successes in a binomial glm!

Warning in eval(family$initialize): non-integer #successes in a binomial glm!

Warning in eval(family$initialize): non-integer #successes in a binomial glm!

Warning in eval(family$initialize): non-integer #successes in a binomial glm!

Warning in eval(family$initialize): non-integer #successes in a binomial glm!

Warning in eval(family$initialize): non-integer #successes in a binomial glm!

Warning in eval(family$initialize): non-integer #successes in a binomial glm!

Warning in eval(family$initialize): non-integer #successes in a binomial glm!

Warning in eval(family$initialize): non-integer #successes in a binomial glm!

Warning in eval(family$initialize): non-integer #successes in a binomial glm!

Warning in eval(family$initialize): non-integer #successes in a binomial glm!

Warning in eval(family$initialize): non-integer #successes in a binomial glm!

Warning in eval(family$initialize): non-integer #successes in a binomial glm!

Warning in eval(family$initialize): non-integer #successes in a binomial glm!

Warning in eval(family$initialize): non-integer #successes in a binomial glm!

Warning in eval(family$initialize): non-integer #successes in a binomial glm!

Warning in eval(family$initialize): non-integer #successes in a binomial glm!

Warning in eval(family$initialize): non-integer #successes in a binomial glm!

Warning in eval(family$initialize): non-integer #successes in a binomial glm!

Warning in eval(family$initialize): non-integer #successes in a binomial glm!

Warning in eval(family$initialize): non-integer #successes in a binomial glm!

Warning in eval(family$initialize): non-integer #successes in a binomial glm!

Warning in eval(family$initialize): non-integer #successes in a binomial glm!

Warning in eval(family$initialize): non-integer #successes in a binomial glm!

Warning in eval(family$initialize): non-integer #successes in a binomial glm!

Warning in eval(family$initialize): non-integer #successes in a binomial glm!

Warning in eval(family$initialize): non-integer #successes in a binomial glm!

Warning in eval(family$initialize): non-integer #successes in a binomial glm!

Warning in eval(family$initialize): non-integer #successes in a binomial glm!

Warning in eval(family$initialize): non-integer #successes in a binomial glm!

Warning in eval(family$initialize): non-integer #successes in a binomial glm!

Warning in eval(family$initialize): non-integer #successes in a binomial glm!

Warning in eval(family$initialize): non-integer #successes in a binomial glm!

Warning in eval(family$initialize): non-integer #successes in a binomial glm!

Warning in eval(family$initialize): non-integer #successes in a binomial glm!

Warning in eval(family$initialize): non-integer #successes in a binomial glm!

Warning in eval(family$initialize): non-integer #successes in a binomial glm!

Warning in eval(family$initialize): non-integer #successes in a binomial glm!</code></pre>
</div>
<div class="cell-output-display">
<p><img src="04-neotoma2_files/figure-html/summarizeByTime-1.png" class="img-fluid" width="672"></p>
</div>
</div>
<p>We can see clear patterns of change for at least some taxa, and the smoothed surfaces are modeled using Generalized Additive Models (GAMs) in R, so we can have more or less control over the actual modeling using the <code>gam</code> or <code>mgcv</code> packages. Depending on how we divide the data we can also look at shifts in altitude, latitude or longitude to better understand how species distributions and abundances changed over time in this region.</p>
<p>Note that for some taxa, they always have a few pollen grains in all pollen samples, so this ‘proportion of sites with taxon’ isn’t very informative. Calculating a metric like average abundance might be more useful.</p>
<p><strong>Question 6</strong>: Repeat the above example, for a different state or other geographic region of your choice.</p>
</section>
</section>
<section id="conclusion" class="level2" data-number="5.6">
<h2 data-number="5.6" class="anchored" data-anchor-id="conclusion"><span class="header-section-number">5.6</span> Conclusion</h2>
<p>So, we’ve done a lot in this exercise. We’ve (1) learned how APIs work (2) searched for sites using site names and geographic parameters, (3) filtered results using temporal and spatial parameters, (4) built a pollen diagram, and (5) done a first-pass spatial mapping of taxa. We will build upon this methodological foundation in future lab exercises.</p>


<div id="refs" class="references csl-bib-body hanging-indent" role="doc-bibliography" style="display: none">
<div id="ref-goring2015" class="csl-entry" role="doc-biblioentry">
Goring, Simon, Andria Dawson, Gavin Simpson, Karthik Ram, Russ Graham, Eric Grimm, and John Williams. 2015. <span>“Neotoma: <span>A Programmatic Interface</span> to the <span>Neotoma Paleoecological Database</span>.”</span> <em>Open Quaternary</em> 1 (1): Art. 2. <a href="https://doi.org/10.5334/oq.ab">https://doi.org/10.5334/oq.ab</a>.
</div>
</div>
</section>

</main> <!-- /main -->
<script id="quarto-html-after-body" type="application/javascript">
window.document.addEventListener("DOMContentLoaded", function (event) {
  const toggleBodyColorMode = (bsSheetEl) => {
    const mode = bsSheetEl.getAttribute("data-mode");
    const bodyEl = window.document.querySelector("body");
    if (mode === "dark") {
      bodyEl.classList.add("quarto-dark");
      bodyEl.classList.remove("quarto-light");
    } else {
      bodyEl.classList.add("quarto-light");
      bodyEl.classList.remove("quarto-dark");
    }
  }
  const toggleBodyColorPrimary = () => {
    const bsSheetEl = window.document.querySelector("link#quarto-bootstrap");
    if (bsSheetEl) {
      toggleBodyColorMode(bsSheetEl);
    }
  }
  toggleBodyColorPrimary();  
  const icon = "";
  const anchorJS = new window.AnchorJS();
  anchorJS.options = {
    placement: 'right',
    icon: icon
  };
  anchorJS.add('.anchored');
  const clipboard = new window.ClipboardJS('.code-copy-button', {
    target: function(trigger) {
      return trigger.previousElementSibling;
    }
  });
  clipboard.on('success', function(e) {
    // button target
    const button = e.trigger;
    // don't keep focus
    button.blur();
    // flash "checked"
    button.classList.add('code-copy-button-checked');
    var currentTitle = button.getAttribute("title");
    button.setAttribute("title", "Copied!");
    let tooltip;
    if (window.bootstrap) {
      button.setAttribute("data-bs-toggle", "tooltip");
      button.setAttribute("data-bs-placement", "left");
      button.setAttribute("data-bs-title", "Copied!");
      tooltip = new bootstrap.Tooltip(button, 
        { trigger: "manual", 
          customClass: "code-copy-button-tooltip",
          offset: [0, -8]});
      tooltip.show();    
    }
    setTimeout(function() {
      if (tooltip) {
        tooltip.hide();
        button.removeAttribute("data-bs-title");
        button.removeAttribute("data-bs-toggle");
        button.removeAttribute("data-bs-placement");
      }
      button.setAttribute("title", currentTitle);
      button.classList.remove('code-copy-button-checked');
    }, 1000);
    // clear code selection
    e.clearSelection();
  });
  function tippyHover(el, contentFn) {
    const config = {
      allowHTML: true,
      content: contentFn,
      maxWidth: 500,
      delay: 100,
      arrow: false,
      appendTo: function(el) {
          return el.parentElement;
      },
      interactive: true,
      interactiveBorder: 10,
      theme: 'quarto',
      placement: 'bottom-start'
    };
    window.tippy(el, config); 
  }
  const noterefs = window.document.querySelectorAll('a[role="doc-noteref"]');
  for (var i=0; i<noterefs.length; i++) {
    const ref = noterefs[i];
    tippyHover(ref, function() {
      // use id or data attribute instead here
      let href = ref.getAttribute('data-footnote-href') || ref.getAttribute('href');
      try { href = new URL(href).hash; } catch {}
      const id = href.replace(/^#\/?/, "");
      const note = window.document.getElementById(id);
      return note.innerHTML;
    });
  }
  const findCites = (el) => {
    const parentEl = el.parentElement;
    if (parentEl) {
      const cites = parentEl.dataset.cites;
      if (cites) {
        return {
          el,
          cites: cites.split(' ')
        };
      } else {
        return findCites(el.parentElement)
      }
    } else {
      return undefined;
    }
  };
  var bibliorefs = window.document.querySelectorAll('a[role="doc-biblioref"]');
  for (var i=0; i<bibliorefs.length; i++) {
    const ref = bibliorefs[i];
    const citeInfo = findCites(ref);
    if (citeInfo) {
      tippyHover(citeInfo.el, function() {
        var popup = window.document.createElement('div');
        citeInfo.cites.forEach(function(cite) {
          var citeDiv = window.document.createElement('div');
          citeDiv.classList.add('hanging-indent');
          citeDiv.classList.add('csl-entry');
          var biblioDiv = window.document.getElementById('ref-' + cite);
          if (biblioDiv) {
            citeDiv.innerHTML = biblioDiv.innerHTML;
          }
          popup.appendChild(citeDiv);
        });
        return popup.innerHTML;
      });
    }
  }
});
</script>
<nav class="page-navigation">
  <div class="nav-page nav-page-previous">
      <a href="./03-age-depth-modelling.html" class="pagination-link">
        <i class="bi bi-arrow-left-short"></i> <span class="nav-page-text"><span class="chapter-number">4</span>&nbsp; <span class="chapter-title">Age-depth modelling</span></span>
      </a>          
  </div>
  <div class="nav-page nav-page-next">
      <a href="./05-wrangling-visualisation.html" class="pagination-link">
        <span class="nav-page-text"><span class="chapter-number">6</span>&nbsp; <span class="chapter-title">Data wrangling and visualisations</span></span> <i class="bi bi-arrow-right-short"></i>
      </a>
  </div>
</nav>
</div> <!-- /content -->



</body></html>